{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "683a8012-ad4a-4b4e-b0ee-4f789b71b055",
   "metadata": {},
   "source": [
    "## HOMERを使用したモチーフ解析の実行"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d93d8c87-e7ac-4c81-9b6f-a6e0293061f0",
   "metadata": {},
   "source": [
    "### パッケージやコマンドの使用方法/動作の確認"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "818b7ce9-3b8e-461d-9471-0b9581601113",
   "metadata": {},
   "source": [
    "このconda仮想環境に導入済みのパッケージの種類・バージョンを確認する。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "322b35fb-5e10-4ab6-adba-9a9df8326e44",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# packages in environment at /opt/conda:\n",
      "#\n",
      "# Name                    Version                   Build  Channel\n",
      "_anaconda_depends         2023.07                 py311_0  \n",
      "_libgcc_mutex             0.1                 conda_forge    conda-forge\n",
      "_openmp_mutex             4.5                  2_kmp_llvm    conda-forge\n",
      "_r-mutex                  1.0.0               anacondar_1    r\n",
      "abseil-cpp                20211102.0           hd4dd3e8_0  \n",
      "aiobotocore               2.7.0           py311h06a4308_0  \n",
      "aiohttp                   3.9.0           py311h5eee18b_0  \n",
      "aioitertools              0.7.1              pyhd3eb1b0_0  \n",
      "aiosignal                 1.2.0              pyhd3eb1b0_0  \n",
      "alabaster                 0.7.12             pyhd3eb1b0_0  \n",
      "altair                    5.2.0              pyhd8ed1ab_0    conda-forge\n",
      "anaconda-anon-usage       0.4.3           py311hfc0e8ea_100  \n",
      "anaconda-catalogs         0.2.0           py311h06a4308_0  \n",
      "anaconda-client           1.11.3          py311h06a4308_0  \n",
      "anaconda-cloud-auth       0.1.4           py311h06a4308_0  \n",
      "anaconda-navigator        2.5.2           py311h06a4308_0  \n",
      "anaconda-project          0.11.1          py311h06a4308_0  \n",
      "anyio                     3.5.0           py311h06a4308_0  \n",
      "aom                       3.6.0                h6a678d5_0  \n",
      "appdirs                   1.4.4              pyhd3eb1b0_0  \n",
      "archspec                  0.2.1              pyhd3eb1b0_0  \n",
      "argon2-cffi               21.3.0             pyhd3eb1b0_0  \n",
      "argon2-cffi-bindings      21.2.0          py311h5eee18b_0  \n",
      "arrow                     1.2.3           py311h06a4308_1  \n",
      "arrow-cpp                 11.0.0               h374c478_2  \n",
      "astroid                   2.14.2          py311h06a4308_0  \n",
      "astropy                   5.3.4           py311hf4808d0_0  \n",
      "asttokens                 2.0.5              pyhd3eb1b0_0  \n",
      "async-lru                 2.0.4           py311h06a4308_0  \n",
      "atomicwrites              1.4.0                      py_0  \n",
      "attrs                     23.1.0          py311h06a4308_0  \n",
      "automat                   20.2.0                     py_0  \n",
      "autopep8                  1.6.0              pyhd3eb1b0_1  \n",
      "aws-c-common              0.6.8                h5eee18b_1  \n",
      "aws-c-event-stream        0.1.6                h6a678d5_6  \n",
      "aws-checksums             0.1.11               h5eee18b_2  \n",
      "aws-sdk-cpp               1.8.185              h721c034_1  \n",
      "babel                     2.11.0          py311h06a4308_0  \n",
      "backports                 1.1                pyhd3eb1b0_0  \n",
      "backports.functools_lru_cache 1.6.4              pyhd3eb1b0_0  \n",
      "backports.tempfile        1.0                pyhd3eb1b0_1  \n",
      "backports.weakref         1.0.post1                  py_1  \n",
      "bcrypt                    3.2.0           py311h5eee18b_1  \n",
      "beautifulsoup4            4.12.2          py311h06a4308_0  \n",
      "binaryornot               0.4.4              pyhd3eb1b0_1  \n",
      "binutils_impl_linux-64    2.38                 h2a08ee3_1  \n",
      "binutils_linux-64         2.38.0               hc2dff05_0  \n",
      "black                     23.11.0         py311h06a4308_0  \n",
      "blas                      2.121                  openblas    conda-forge\n",
      "blas-devel                3.9.0           21_linux64_openblas    conda-forge\n",
      "bleach                    4.1.0              pyhd3eb1b0_0  \n",
      "blosc                     1.21.3               h6a678d5_0  \n",
      "bokeh                     3.3.0           py311h92b7b1e_0  \n",
      "boltons                   23.0.0          py311h06a4308_0  \n",
      "boost-cpp                 1.82.0               hdb19cb5_2  \n",
      "botocore                  1.31.64         py311h06a4308_0  \n",
      "bottleneck                1.3.5           py311hbed6279_0  \n",
      "bqplot                    0.12.42            pyhd8ed1ab_0    conda-forge\n",
      "brotli                    1.0.9                h5eee18b_7  \n",
      "brotli-bin                1.0.9                h5eee18b_7  \n",
      "brotli-python             1.0.9           py311h6a678d5_7  \n",
      "brunsli                   0.1                  h2531618_0  \n",
      "bwidget                   1.9.14               ha770c72_1    conda-forge\n",
      "bzip2                     1.0.8                h7b6447c_0  \n",
      "c-ares                    1.19.1               h5eee18b_0  \n",
      "c-blosc2                  2.12.0               h80c7b02_0  \n",
      "ca-certificates           2023.12.12           h06a4308_0  \n",
      "cached-property           1.5.2                hd8ed1ab_1    conda-forge\n",
      "cached_property           1.5.2              pyha770c72_1    conda-forge\n",
      "cairo                     1.16.0               hb05425b_5  \n",
      "certifi                   2023.11.17         pyhd8ed1ab_0    conda-forge\n",
      "cffi                      1.16.0          py311h5eee18b_0  \n",
      "cfitsio                   3.470                h5893167_7  \n",
      "chardet                   4.0.0           py311h06a4308_1003  \n",
      "charls                    2.2.0                h2531618_0  \n",
      "charset-normalizer        2.0.4              pyhd3eb1b0_0  \n",
      "click                     8.1.7           py311h06a4308_0  \n",
      "cloudpickle               2.2.1           py311h06a4308_0  \n",
      "clyent                    1.2.2           py311h06a4308_1  \n",
      "colorama                  0.4.6           py311h06a4308_0  \n",
      "colorcet                  3.0.1           py311h06a4308_0  \n",
      "comm                      0.2.1              pyhd8ed1ab_0    conda-forge\n",
      "conda                     23.11.0         py311h38be061_1    conda-forge\n",
      "conda-build               3.28.4          py311h06a4308_0  \n",
      "conda-content-trust       0.2.0           py311h06a4308_0  \n",
      "conda-index               0.3.0           py311h06a4308_0  \n",
      "conda-libmamba-solver     23.12.0            pyhd3eb1b0_1  \n",
      "conda-pack                0.6.0              pyhd3eb1b0_0  \n",
      "conda-package-handling    2.2.0           py311h06a4308_0  \n",
      "conda-package-streaming   0.9.0           py311h06a4308_0  \n",
      "conda-repo-cli            1.0.75          py311h06a4308_0  \n",
      "conda-token               0.4.0              pyhd3eb1b0_0  \n",
      "conda-verify              3.4.2                      py_1  \n",
      "constantly                23.10.4         py311h06a4308_0  \n",
      "contourpy                 1.2.0           py311hdb19cb5_0  \n",
      "cookiecutter              2.5.0           py311h06a4308_0  \n",
      "cryptography              41.0.7          py311hdda0065_0  \n",
      "cssselect                 1.1.0              pyhd3eb1b0_0  \n",
      "curl                      8.5.0                hdbd6064_0  \n",
      "cycler                    0.11.0             pyhd3eb1b0_0  \n",
      "cyrus-sasl                2.1.28               h52b45da_1  \n",
      "cytoolz                   0.12.2          py311h5eee18b_0  \n",
      "daal4py                   2023.1.1        py311h4cb112f_0  \n",
      "dal                       2023.1.1         hdb19cb5_48680  \n",
      "dask                      2023.11.0       py311h06a4308_0  \n",
      "dask-core                 2023.11.0       py311h06a4308_0  \n",
      "datasets                  2.12.0          py311h06a4308_0  \n",
      "datashader                0.16.0          py311h06a4308_0  \n",
      "dav1d                     1.2.1                h5eee18b_0  \n",
      "dbus                      1.13.18              hb2f20db_0  \n",
      "debugpy                   1.6.7           py311h6a678d5_0  \n",
      "decorator                 5.1.1              pyhd3eb1b0_0  \n",
      "defusedxml                0.7.1              pyhd3eb1b0_0  \n",
      "diff-match-patch          20200713           pyhd3eb1b0_0  \n",
      "dill                      0.3.6           py311h06a4308_0  \n",
      "distributed               2023.11.0       py311h06a4308_0  \n",
      "distro                    1.8.0           py311h06a4308_0  \n",
      "docstring-to-markdown     0.11            py311h06a4308_0  \n",
      "docutils                  0.18.1          py311h06a4308_3  \n",
      "entrypoints               0.4             py311h06a4308_0  \n",
      "et_xmlfile                1.1.0           py311h06a4308_0  \n",
      "executing                 0.8.3              pyhd3eb1b0_0  \n",
      "expat                     2.5.0                h6a678d5_0  \n",
      "filelock                  3.13.1          py311h06a4308_0  \n",
      "flake8                    6.0.0           py311h06a4308_0  \n",
      "flask                     2.2.5           py311h06a4308_0  \n",
      "fmt                       9.1.0                hdb19cb5_0  \n",
      "fontconfig                2.14.1               h4c34cd2_2  \n",
      "fonttools                 4.25.0             pyhd3eb1b0_0  \n",
      "fqdn                      1.5.1              pyhd8ed1ab_0    conda-forge\n",
      "freetype                  2.12.1               h4a9f257_0  \n",
      "fribidi                   1.0.10               h36c2ea0_0    conda-forge\n",
      "frozenlist                1.4.0           py311h5eee18b_0  \n",
      "fsspec                    2023.10.0       py311h06a4308_0  \n",
      "future                    0.18.3          py311h06a4308_0  \n",
      "gcc_impl_linux-64         11.2.0               h1234567_1  \n",
      "gcc_linux-64              11.2.0               h5c386dc_0  \n",
      "gensim                    4.3.0           py311hba01205_1  \n",
      "gflags                    2.2.2                he6710b0_0  \n",
      "gfortran_impl_linux-64    11.2.0              h7a446d4_16    conda-forge\n",
      "gfortran_linux-64         11.2.0               hc2dff05_0  \n",
      "giflib                    5.2.1                h5eee18b_3  \n",
      "glib                      2.69.1               he621ea3_2  \n",
      "glog                      0.5.0                h2531618_0  \n",
      "gmp                       6.2.1                h295c915_3  \n",
      "gmpy2                     2.1.2           py311hc9b5ff0_0  \n",
      "graphite2                 1.3.14               h295c915_1  \n",
      "greenlet                  3.0.1           py311h6a678d5_0  \n",
      "grpc-cpp                  1.48.2               he1ff14a_1  \n",
      "gst-plugins-base          1.14.1               h6a678d5_1  \n",
      "gstreamer                 1.14.1               h5eee18b_1  \n",
      "gxx_impl_linux-64         11.2.0               h1234567_1  \n",
      "gxx_linux-64              11.2.0               hc2dff05_0  \n",
      "h5py                      3.9.0           py311hdd6beaf_0  \n",
      "harfbuzz                  4.3.0                hf52aaf7_2  \n",
      "hdf5                      1.12.1               h2b7332f_3  \n",
      "heapdict                  1.0.1              pyhd3eb1b0_0  \n",
      "holoviews                 1.18.1          py311h06a4308_0  \n",
      "huggingface_hub           0.17.3          py311h06a4308_0  \n",
      "hvplot                    0.9.1           py311h06a4308_0  \n",
      "hyperlink                 21.0.0             pyhd3eb1b0_0  \n",
      "icu                       73.1                 h6a678d5_0  \n",
      "idna                      3.4             py311h06a4308_0  \n",
      "imagecodecs               2023.1.23       py311h8105a5c_0  \n",
      "imageio                   2.31.4          py311h06a4308_0  \n",
      "imagesize                 1.4.1           py311h06a4308_0  \n",
      "imbalanced-learn          0.11.0          py311h06a4308_1  \n",
      "importlib-metadata        7.0.1           py311h06a4308_0  \n",
      "importlib_metadata        7.0.1                hd3eb1b0_0  \n",
      "importlib_resources       6.1.1              pyhd8ed1ab_0    conda-forge\n",
      "incremental               21.3.0             pyhd3eb1b0_0  \n",
      "inflection                0.5.1           py311h06a4308_0  \n",
      "iniconfig                 1.1.1              pyhd3eb1b0_0  \n",
      "intake                    0.6.8           py311h06a4308_0  \n",
      "intel-openmp              2023.1.0         hdb19cb5_46306  \n",
      "intervaltree              3.1.0              pyhd3eb1b0_0  \n",
      "ipykernel                 6.28.0          py311h06a4308_0  \n",
      "ipython                   8.20.0          py311h06a4308_0  \n",
      "ipython_genutils          0.2.0              pyhd3eb1b0_1  \n",
      "ipywidgets                8.1.1              pyhd8ed1ab_0    conda-forge\n",
      "isoduration               20.11.0            pyhd8ed1ab_0    conda-forge\n",
      "isort                     5.9.3              pyhd3eb1b0_0  \n",
      "itables                   1.6.3              pyhd8ed1ab_0    conda-forge\n",
      "itemadapter               0.3.0              pyhd3eb1b0_0  \n",
      "itemloaders               1.0.4              pyhd3eb1b0_1  \n",
      "itsdangerous              2.0.1              pyhd3eb1b0_0  \n",
      "jaraco.classes            3.2.1              pyhd3eb1b0_0  \n",
      "jedi                      0.18.1          py311h06a4308_1  \n",
      "jeepney                   0.7.1              pyhd3eb1b0_0  \n",
      "jellyfish                 1.0.1           py311hb02cf49_0  \n",
      "jinja2                    3.1.2           py311h06a4308_0  \n",
      "jmespath                  1.0.1           py311h06a4308_0  \n",
      "joblib                    1.2.0           py311h06a4308_0  \n",
      "jpeg                      9e                   h5eee18b_1  \n",
      "jq                        1.6               h27cfd23_1000  \n",
      "json5                     0.9.6              pyhd3eb1b0_0  \n",
      "jsonpatch                 1.32               pyhd3eb1b0_0  \n",
      "jsonpointer               2.1                pyhd3eb1b0_0  \n",
      "jsonschema                4.21.1             pyhd8ed1ab_0    conda-forge\n",
      "jsonschema-specifications 2023.7.1        py311h06a4308_0  \n",
      "jsonschema-with-format-nongpl 4.21.1             pyhd8ed1ab_0    conda-forge\n",
      "jupyter                   1.0.0           py311h06a4308_8  \n",
      "jupyter-lsp               2.2.0           py311h06a4308_0  \n",
      "jupyter_client            8.6.0           py311h06a4308_0  \n",
      "jupyter_console           6.6.3           py311h06a4308_0  \n",
      "jupyter_contrib_core      0.4.0              pyhd8ed1ab_0    conda-forge\n",
      "jupyter_contrib_nbextensions 0.7.0              pyhd8ed1ab_0    conda-forge\n",
      "jupyter_core              5.5.0           py311h06a4308_0  \n",
      "jupyter_events            0.8.0           py311h06a4308_0  \n",
      "jupyter_highlight_selected_word 0.2.0           pyhd8ed1ab_1006    conda-forge\n",
      "jupyter_latex_envs        1.4.6           pyhd8ed1ab_1002    conda-forge\n",
      "jupyter_nbextensions_configurator 0.6.1              pyhd8ed1ab_0    conda-forge\n",
      "jupyter_server            2.10.0          py311h06a4308_0  \n",
      "jupyter_server_terminals  0.4.4           py311h06a4308_1  \n",
      "jupyterlab                4.0.8           py311h06a4308_0  \n",
      "jupyterlab-language-pack-ja-jp 4.0.post6          pyh87cfba7_0    conda-forge\n",
      "jupyterlab_pygments       0.1.2                      py_0  \n",
      "jupyterlab_server         2.25.1          py311h06a4308_0  \n",
      "jupyterlab_widgets        3.0.9              pyhd8ed1ab_0    conda-forge\n",
      "jxrlib                    1.1                  h7b6447c_2  \n",
      "kernel-headers_linux-64   2.6.32              he073ed8_16    conda-forge\n",
      "keyring                   23.13.1         py311h06a4308_0  \n",
      "kiwisolver                1.4.4           py311h6a678d5_0  \n",
      "krb5                      1.20.1               h143b758_1  \n",
      "lazy-object-proxy         1.6.0           py311h5eee18b_0  \n",
      "lazy_loader               0.3             py311h06a4308_0  \n",
      "lcms2                     2.12                 h3be6417_0  \n",
      "ld_impl_linux-64          2.38                 h1181459_1  \n",
      "lerc                      3.0                  h295c915_0  \n",
      "libaec                    1.0.4                he6710b0_1  \n",
      "libarchive                3.6.2                h6ac8c49_2  \n",
      "libavif                   0.11.1               h5eee18b_0  \n",
      "libblas                   3.9.0           21_linux64_openblas    conda-forge\n",
      "libboost                  1.82.0               h109eef0_2  \n",
      "libbrotlicommon           1.0.9                h5eee18b_7  \n",
      "libbrotlidec              1.0.9                h5eee18b_7  \n",
      "libbrotlienc              1.0.9                h5eee18b_7  \n",
      "libcblas                  3.9.0           21_linux64_openblas    conda-forge\n",
      "libclang                  14.0.6          default_hc6dbbc7_1  \n",
      "libclang13                14.0.6          default_he11475f_1  \n",
      "libcups                   2.4.2                h2d74bed_1  \n",
      "libcurl                   8.5.0                h251f7ec_0  \n",
      "libdeflate                1.17                 h5eee18b_1  \n",
      "libedit                   3.1.20230828         h5eee18b_0  \n",
      "libev                     4.33                 h7f8727e_1  \n",
      "libevent                  2.1.12               hdbd6064_1  \n",
      "libffi                    3.4.4                h6a678d5_0  \n",
      "libgcc-devel_linux-64     11.2.0               h1234567_1  \n",
      "libgcc-ng                 13.2.0               h807b86a_3    conda-forge\n",
      "libgfortran-ng            13.2.0               h69a702a_3    conda-forge\n",
      "libgfortran5              13.2.0               ha4646dd_3    conda-forge\n",
      "libgomp                   13.2.0               h807b86a_3    conda-forge\n",
      "liblapack                 3.9.0           21_linux64_openblas    conda-forge\n",
      "liblapacke                3.9.0           21_linux64_openblas    conda-forge\n",
      "liblief                   0.12.3               h6a678d5_0  \n",
      "libllvm14                 14.0.6               hdb19cb5_3  \n",
      "libmamba                  1.5.6                haf1ee3a_0  \n",
      "libmambapy                1.5.6           py311h2dafd23_0  \n",
      "libnghttp2                1.57.0               h2d74bed_0  \n",
      "libopenblas               0.3.26          pthreads_h413a1c8_0    conda-forge\n",
      "libpng                    1.6.39               h5eee18b_0  \n",
      "libpq                     12.15                hdbd6064_1  \n",
      "libprotobuf               3.20.3               he621ea3_0  \n",
      "libsodium                 1.0.18               h7b6447c_0  \n",
      "libsolv                   0.7.24               he621ea3_0  \n",
      "libspatialindex           1.9.3                h2531618_0  \n",
      "libssh2                   1.10.0               hdbd6064_2  \n",
      "libstdcxx-devel_linux-64  11.2.0               h1234567_1  \n",
      "libstdcxx-ng              13.2.0               h7e041cc_3    conda-forge\n",
      "libthrift                 0.15.0               h1795dd8_2  \n",
      "libtiff                   4.5.1                h6a678d5_0  \n",
      "libuuid                   1.41.5               h5eee18b_0  \n",
      "libuv                     1.44.2               h5eee18b_0    anaconda\n",
      "libwebp                   1.3.2                h11a3e52_0  \n",
      "libwebp-base              1.3.2                h5eee18b_0  \n",
      "libxcb                    1.15                 h7f8727e_0  \n",
      "libxkbcommon              1.0.1                h5eee18b_1  \n",
      "libxml2                   2.10.4               hf1b16e4_1  \n",
      "libxslt                   1.1.37               h5eee18b_1  \n",
      "libzlib                   1.2.13               hd590300_5    conda-forge\n",
      "libzopfli                 1.0.3                he6710b0_0  \n",
      "linkify-it-py             2.0.0           py311h06a4308_0  \n",
      "llvm-openmp               17.0.6               h4dfa4b3_0    conda-forge\n",
      "llvmlite                  0.41.1          py311ha6695c7_0    conda-forge\n",
      "locket                    1.0.0           py311h06a4308_0  \n",
      "lxml                      4.9.3           py311hdbbb534_0  \n",
      "lz4                       4.3.2           py311h5eee18b_0  \n",
      "lz4-c                     1.9.4                h6a678d5_0  \n",
      "lzo                       2.10                 h7b6447c_2  \n",
      "make                      4.3                  hd18ef5c_1    conda-forge\n",
      "markdown                  3.4.1           py311h06a4308_0  \n",
      "markdown-it-py            2.2.0           py311h06a4308_1  \n",
      "markupsafe                2.1.3           py311h5eee18b_0  \n",
      "matplotlib                3.8.2           py311h38be061_0    conda-forge\n",
      "matplotlib-base           3.8.2           py311h54ef318_0    conda-forge\n",
      "matplotlib-inline         0.1.6           py311h06a4308_0  \n",
      "matplotlib-venn           0.11.9             pyhd8ed1ab_0    conda-forge\n",
      "mccabe                    0.7.0              pyhd3eb1b0_0  \n",
      "mdit-py-plugins           0.3.0           py311h06a4308_0  \n",
      "mdurl                     0.1.0           py311h06a4308_0  \n",
      "menuinst                  2.0.1           py311h06a4308_1  \n",
      "mistune                   2.0.4           py311h06a4308_0  \n",
      "mkl                       2022.2.1         h84fe81f_16997    conda-forge\n",
      "mkl-service               2.4.0           py311hb711fc7_0    conda-forge\n",
      "mkl_fft                   1.3.8           py311h8e1fd07_0    conda-forge\n",
      "mkl_random                1.2.4           py311h2d0063b_2    conda-forge\n",
      "more-itertools            10.1.0          py311h06a4308_0  \n",
      "mpc                       1.1.0                h10f8cd9_1  \n",
      "mpfr                      4.0.2                hb69a4c5_1  \n",
      "mpi                       1.0                       mpich  \n",
      "mpich                     4.1.1                hbae89fd_0  \n",
      "mpmath                    1.3.0           py311h06a4308_0  \n",
      "msgpack-python            1.0.3           py311hdb19cb5_0  \n",
      "multidict                 6.0.4           py311h5eee18b_0  \n",
      "multipledispatch          0.6.0           py311h06a4308_0  \n",
      "multiprocess              0.70.14         py311h06a4308_0  \n",
      "munkres                   1.1.4                      py_0  \n",
      "mypy_extensions           1.0.0           py311h06a4308_0  \n",
      "mysql                     5.7.24               h721c034_2  \n",
      "navigator-updater         0.4.0           py311h06a4308_1  \n",
      "nbclassic                 1.0.0              pyhb4ecaf3_1    conda-forge\n",
      "nbclient                  0.8.0           py311h06a4308_0  \n",
      "nbconvert                 7.10.0          py311h06a4308_0  \n",
      "nbconvert-core            7.10.0             pyhd8ed1ab_0    conda-forge\n",
      "nbformat                  5.9.2           py311h06a4308_0  \n",
      "ncurses                   6.4                  h6a678d5_0  \n",
      "nest-asyncio              1.5.6           py311h06a4308_0  \n",
      "networkx                  3.1             py311h06a4308_0  \n",
      "nltk                      3.8.1           py311h06a4308_0  \n",
      "nodejs                    18.16.0              h2d74bed_1    anaconda\n",
      "nomkl                     3.0                           0  \n",
      "notebook                  6.5.4              pyha770c72_0    conda-forge\n",
      "notebook-shim             0.2.3           py311h06a4308_0  \n",
      "nspr                      4.35                 h6a678d5_0  \n",
      "nss                       3.89.1               h6a678d5_0  \n",
      "numba                     0.58.1          py311h96b013e_0    conda-forge\n",
      "numexpr                   2.8.8           py311h039bad6_100    conda-forge\n",
      "numpy                     1.26.3          py311h24aa872_0  \n",
      "numpy-base                1.26.3          py311hbfb1bba_0  \n",
      "numpydoc                  1.5.0           py311h06a4308_0  \n",
      "oniguruma                 6.9.7.1              h27cfd23_0  \n",
      "openblas                  0.3.26          pthreads_h7a3da1a_0    conda-forge\n",
      "openjpeg                  2.4.0                h3ad879b_0  \n",
      "openpyxl                  3.0.10          py311h5eee18b_0  \n",
      "openssl                   3.2.0                hd590300_1    conda-forge\n",
      "orc                       1.7.4                hb3bc3d3_1  \n",
      "overrides                 7.4.0           py311h06a4308_0  \n",
      "packaging                 23.1            py311h06a4308_0  \n",
      "pandas                    2.2.0           py311h320fe9a_0    conda-forge\n",
      "pandoc                    3.1.3                h32600fe_0    conda-forge\n",
      "pandocfilters             1.5.0              pyhd3eb1b0_0  \n",
      "panel                     1.3.1           py311h06a4308_0  \n",
      "pango                     1.50.7               h05da053_0  \n",
      "param                     2.0.2           py311h06a4308_0  \n",
      "parsel                    1.6.0           py311h06a4308_0  \n",
      "parso                     0.8.3              pyhd3eb1b0_0  \n",
      "partd                     1.4.1           py311h06a4308_0  \n",
      "patch                     2.7.6             h7b6447c_1001  \n",
      "patchelf                  0.17.2               h6a678d5_0  \n",
      "pathlib                   1.0.1              pyhd3eb1b0_1  \n",
      "pathspec                  0.10.3          py311h06a4308_0  \n",
      "patsy                     0.5.3           py311h06a4308_0  \n",
      "pcre                      8.45                 h295c915_0  \n",
      "pcre2                     10.42                hebb0a14_0  \n",
      "pep8                      1.7.1           py311h06a4308_1  \n",
      "pexpect                   4.8.0              pyhd3eb1b0_3  \n",
      "pickleshare               0.7.5           pyhd3eb1b0_1003  \n",
      "pillow                    10.0.1          py311ha6cbd5a_0  \n",
      "pip                       23.3.2                   pypi_0    pypi\n",
      "pivottablejs              0.9.0           py311h06a4308_0  \n",
      "pixman                    0.43.0               h59595ed_0    conda-forge\n",
      "pkce                      1.0.3           py311h06a4308_0  \n",
      "pkginfo                   1.9.6           py311h06a4308_0  \n",
      "pkgutil-resolve-name      1.3.10             pyhd8ed1ab_1    conda-forge\n",
      "platformdirs              3.10.0          py311h06a4308_0  \n",
      "plotly                    5.18.0             pyhd8ed1ab_0    conda-forge\n",
      "pluggy                    1.0.0           py311h06a4308_1  \n",
      "ply                       3.11            py311h06a4308_0  \n",
      "prometheus_client         0.14.1          py311h06a4308_0  \n",
      "prompt-toolkit            3.0.43          py311h06a4308_0  \n",
      "prompt_toolkit            3.0.43               hd3eb1b0_0  \n",
      "protego                   0.1.16                     py_0  \n",
      "psutil                    5.9.0           py311h5eee18b_0  \n",
      "ptyprocess                0.7.0              pyhd3eb1b0_2  \n",
      "pure_eval                 0.2.2              pyhd3eb1b0_0  \n",
      "py-cpuinfo                9.0.0           py311h06a4308_0  \n",
      "py-lief                   0.12.3          py311h6a678d5_0  \n",
      "pyarrow                   11.0.0          py311hd8e8d9b_1  \n",
      "pyasn1                    0.4.8              pyhd3eb1b0_0  \n",
      "pyasn1-modules            0.2.8                      py_0  \n",
      "pybind11-abi              4                    hd3eb1b0_1  \n",
      "pycodestyle               2.10.0          py311h06a4308_0  \n",
      "pycosat                   0.6.6           py311h5eee18b_0  \n",
      "pycparser                 2.21               pyhd3eb1b0_0  \n",
      "pyct                      0.5.0           py311h06a4308_0  \n",
      "pycurl                    7.45.2          py311hdbd6064_1  \n",
      "pydantic                  1.10.12         py311h5eee18b_1  \n",
      "pydispatcher              2.0.5           py311h06a4308_2  \n",
      "pydocstyle                6.3.0           py311h06a4308_0  \n",
      "pyerfa                    2.0.0           py311h5eee18b_0  \n",
      "pyflakes                  3.0.1           py311h06a4308_0  \n",
      "pygments                  2.15.1          py311h06a4308_1  \n",
      "pyjwt                     2.4.0           py311h06a4308_0  \n",
      "pylint                    2.16.2          py311h06a4308_0  \n",
      "pylint-venv               2.3.0           py311h06a4308_0  \n",
      "pyls-spyder               0.4.0              pyhd3eb1b0_0  \n",
      "pyodbc                    5.0.1           py311h6a678d5_0  \n",
      "pyopenssl                 23.2.0          py311h06a4308_0  \n",
      "pyparsing                 3.0.9           py311h06a4308_0  \n",
      "pyqt                      5.15.10         py311h6a678d5_0  \n",
      "pyqt5-sip                 12.13.0         py311h5eee18b_0  \n",
      "pyqtwebengine             5.15.10         py311h6a678d5_0  \n",
      "pysocks                   1.7.1           py311h06a4308_0  \n",
      "pytables                  3.9.2           py311h9d13977_0  \n",
      "pytest                    7.4.0           py311h06a4308_0  \n",
      "python                    3.11.7               h955ad1f_0  \n",
      "python-dateutil           2.8.2              pyhd3eb1b0_0  \n",
      "python-dotenv             0.21.0          py311h06a4308_0  \n",
      "python-fastjsonschema     2.16.2          py311h06a4308_0  \n",
      "python-json-logger        2.0.7           py311h06a4308_0  \n",
      "python-libarchive-c       2.9                pyhd3eb1b0_1  \n",
      "python-lmdb               1.4.1           py311h6a678d5_0  \n",
      "python-lsp-black          1.2.1           py311h06a4308_0  \n",
      "python-lsp-jsonrpc        1.0.0              pyhd3eb1b0_0  \n",
      "python-lsp-server         1.7.2           py311h06a4308_0  \n",
      "python-slugify            5.0.2              pyhd3eb1b0_0  \n",
      "python-snappy             0.6.1           py311h6a678d5_0  \n",
      "python-tzdata             2023.3             pyhd3eb1b0_0  \n",
      "python-xxhash             2.0.2           py311h5eee18b_1  \n",
      "python_abi                3.11                    2_cp311    conda-forge\n",
      "pytoolconfig              1.2.6           py311h06a4308_0  \n",
      "pytz                      2023.3.post1    py311h06a4308_0  \n",
      "pyviz_comms               3.0.0           py311h06a4308_0  \n",
      "pywavelets                1.5.0           py311hf4808d0_0  \n",
      "pyxdg                     0.27               pyhd3eb1b0_0  \n",
      "pyyaml                    6.0.1           py311h5eee18b_0  \n",
      "pyzmq                     25.1.2          py311h6a678d5_0  \n",
      "qdarkstyle                3.0.2              pyhd3eb1b0_0  \n",
      "qstylizer                 0.2.2           py311h06a4308_0  \n",
      "qt-main                   5.15.2              h53bd1ea_10  \n",
      "qt-webengine              5.15.9               h9ab4d14_7  \n",
      "qtawesome                 1.2.2           py311h06a4308_0  \n",
      "qtconsole                 5.4.2           py311h06a4308_0  \n",
      "qtpy                      2.4.1           py311h06a4308_0  \n",
      "queuelib                  1.6.2           py311h06a4308_0  \n",
      "r-askpass                 1.2.0             r43h76d94ec_0    r\n",
      "r-assertthat              0.2.1             r43hc72bb7e_4    conda-forge\n",
      "r-backports               1.4.1             r43h76d94ec_0    r\n",
      "r-base                    4.3.1                h1ae530e_0    r\n",
      "r-base64enc               0.1_3             r43h76d94ec_4    r\n",
      "r-bit                     4.0.5             r43h76d94ec_0    r\n",
      "r-bit64                   4.0.5             r43h76d94ec_0    r\n",
      "r-blob                    1.2.4             r43hc72bb7e_1    conda-forge\n",
      "r-boot                    1.3_28.1          r43hc72bb7e_1    conda-forge\n",
      "r-broom                   1.0.5             r43hc72bb7e_1    conda-forge\n",
      "r-bslib                   0.5.1             r43hc72bb7e_0    conda-forge\n",
      "r-cachem                  1.0.8             r43h76d94ec_0    r\n",
      "r-callr                   3.7.3             r43hc72bb7e_1    conda-forge\n",
      "r-caret                   6.0_94            r43h76d94ec_0    r\n",
      "r-cellranger              1.1.0           r43hc72bb7e_1006    conda-forge\n",
      "r-class                   7.3_22            r43h76d94ec_0    r\n",
      "r-cli                     3.6.1             r43h884c59f_0    r\n",
      "r-clipr                   0.8.0             r43hc72bb7e_2    conda-forge\n",
      "r-clock                   0.7.0             r43h884c59f_0    r\n",
      "r-cluster                 2.1.4             r43h640688f_0    r\n",
      "r-codetools               0.2_19            r43hc72bb7e_1    conda-forge\n",
      "r-colorspace              2.1_0             r43h76d94ec_0    r\n",
      "r-commonmark              1.9.0             r43h76d94ec_0    r\n",
      "r-conflicted              1.2.0             r43h785f33e_1    conda-forge\n",
      "r-cpp11                   0.4.7             r43hc72bb7e_0    conda-forge\n",
      "r-crayon                  1.5.2             r43hc72bb7e_2    conda-forge\n",
      "r-curl                    5.1.0             r43h76d94ec_0    r\n",
      "r-data.table              1.14.8            r43h76d94ec_0    r\n",
      "r-dbi                     1.2.1             r43hc72bb7e_0    conda-forge\n",
      "r-dbplyr                  2.4.0             r43hc72bb7e_0    conda-forge\n",
      "r-diagram                 1.6.5             r43ha770c72_2    conda-forge\n",
      "r-digest                  0.6.33            r43h884c59f_0    r\n",
      "r-dplyr                   1.1.3             r43h884c59f_0    r\n",
      "r-dtplyr                  1.3.1             r43hc72bb7e_1    conda-forge\n",
      "r-e1071                   1.7_13            r43h884c59f_0    r\n",
      "r-ellipsis                0.3.2             r43h76d94ec_0    r\n",
      "r-essentials              4.3.1                     r43_0    r\n",
      "r-evaluate                0.23              r43hc72bb7e_0    conda-forge\n",
      "r-fansi                   1.0.5             r43h76d94ec_0    r\n",
      "r-farver                  2.1.1             r43h884c59f_0    r\n",
      "r-fastmap                 1.1.1             r43h884c59f_0    r\n",
      "r-fontawesome             0.5.2             r43hc72bb7e_0    conda-forge\n",
      "r-forcats                 1.0.0             r43hc72bb7e_1    conda-forge\n",
      "r-foreach                 1.5.2             r43hc72bb7e_2    conda-forge\n",
      "r-foreign                 0.8_85            r43h76d94ec_0    r\n",
      "r-formatr                 1.14              r43hc72bb7e_1    conda-forge\n",
      "r-fs                      1.6.3             r43h884c59f_0    r\n",
      "r-future                  1.33.1            r43hc72bb7e_0    conda-forge\n",
      "r-future.apply            1.11.1            r43hc72bb7e_0    conda-forge\n",
      "r-gargle                  1.5.2             r43h785f33e_0    conda-forge\n",
      "r-generics                0.1.3             r43hc72bb7e_2    conda-forge\n",
      "r-ggplot2                 3.4.4             r43hc72bb7e_0    conda-forge\n",
      "r-glmnet                  4.1_8             r43hb5eb8f6_0    r\n",
      "r-globals                 0.16.2            r43hc72bb7e_1    conda-forge\n",
      "r-glue                    1.6.2             r43h76d94ec_0    r\n",
      "r-googledrive             2.1.1             r43hc72bb7e_1    conda-forge\n",
      "r-googlesheets4           1.1.1             r43h785f33e_1    conda-forge\n",
      "r-gower                   1.0.1             r43h76d94ec_0    r\n",
      "r-gtable                  0.3.4             r43hc72bb7e_0    conda-forge\n",
      "r-hardhat                 1.3.0             r43hc72bb7e_1    conda-forge\n",
      "r-haven                   2.5.3             r43h884c59f_0    r\n",
      "r-highr                   0.10              r43hc72bb7e_1    conda-forge\n",
      "r-hms                     1.1.3             r43hc72bb7e_1    conda-forge\n",
      "r-htmltools               0.5.6.1           r43h76d94ec_0    r\n",
      "r-httpuv                  1.6.11            r43h884c59f_0    r\n",
      "r-httr                    1.4.7             r43hc72bb7e_0    conda-forge\n",
      "r-ids                     1.0.1             r43hc72bb7e_3    conda-forge\n",
      "r-ipred                   0.9_14            r43h76d94ec_0    r\n",
      "r-irdisplay               1.1               r43hd8ed1ab_2    conda-forge\n",
      "r-irkernel                1.3.2             r43h785f33e_1    conda-forge\n",
      "r-isoband                 0.2.7             r43h884c59f_0    r\n",
      "r-iterators               1.0.14            r43hc72bb7e_2    conda-forge\n",
      "r-jquerylib               0.1.4             r43hc72bb7e_2    conda-forge\n",
      "r-jsonlite                1.8.7             r43h76d94ec_0    r\n",
      "r-kernsmooth              2.23_22           r43h640688f_0    r\n",
      "r-knitr                   1.45              r43hc72bb7e_0    conda-forge\n",
      "r-labeling                0.4.3             r43hc72bb7e_0    conda-forge\n",
      "r-later                   1.3.1             r43h884c59f_0    r\n",
      "r-lattice                 0.21_9            r43h76d94ec_0    r\n",
      "r-lava                    1.7.3             r43hc72bb7e_0    conda-forge\n",
      "r-lifecycle               1.0.4             r43hc72bb7e_0    conda-forge\n",
      "r-listenv                 0.9.0             r43hc72bb7e_1    conda-forge\n",
      "r-lubridate               1.9.3             r43h57805ef_0    conda-forge\n",
      "r-magrittr                2.0.3             r43h76d94ec_0    r\n",
      "r-mass                    7.3_60            r43h76d94ec_0    r\n",
      "r-matrix                  1.6_1.1           r43h76d94ec_0    r\n",
      "r-memoise                 2.0.1             r43hc72bb7e_2    conda-forge\n",
      "r-mgcv                    1.9_0             r43h76d94ec_0    r\n",
      "r-mime                    0.12              r43h76d94ec_0    r\n",
      "r-modelmetrics            1.2.2.2           r43h884c59f_0    r\n",
      "r-modelr                  0.1.11            r43hc72bb7e_1    conda-forge\n",
      "r-munsell                 0.5.0           r43hc72bb7e_1006    conda-forge\n",
      "r-nlme                    3.1_163           r43h640688f_0    r\n",
      "r-nnet                    7.3_19            r43h76d94ec_0    r\n",
      "r-numderiv                2016.8_1.1        r43hc72bb7e_5    conda-forge\n",
      "r-openssl                 2.1.1             r43h76d94ec_0    r\n",
      "r-parallelly              1.36.0            r43hc72bb7e_1    conda-forge\n",
      "r-pbdzmq                  0.3_10            r43h884c59f_0    r\n",
      "r-pillar                  1.9.0             r43hc72bb7e_1    conda-forge\n",
      "r-pkgconfig               2.0.3             r43hc72bb7e_3    conda-forge\n",
      "r-plyr                    1.8.9             r43h884c59f_0    r\n",
      "r-prettyunits             1.2.0             r43hc72bb7e_0    conda-forge\n",
      "r-proc                    1.18.4            r43h884c59f_0    r\n",
      "r-processx                3.8.2             r43h76d94ec_0    r\n",
      "r-prodlim                 2023.08.28        r43h884c59f_0    r\n",
      "r-progress                1.2.3             r43hc72bb7e_0    conda-forge\n",
      "r-progressr               0.14.0            r43hc72bb7e_0    conda-forge\n",
      "r-promises                1.2.1             r43h884c59f_0    r\n",
      "r-proxy                   0.4_27            r43h76d94ec_0    r\n",
      "r-ps                      1.7.5             r43h76d94ec_0    r\n",
      "r-purrr                   1.0.2             r43h76d94ec_0    r\n",
      "r-quantmod                0.4.25            r43hc72bb7e_0    conda-forge\n",
      "r-r6                      2.5.1             r43hc72bb7e_2    conda-forge\n",
      "r-ragg                    1.2.6             r43h884c59f_0    r\n",
      "r-randomforest            4.7_1.1           r43h640688f_0    r\n",
      "r-rappdirs                0.3.3             r43h76d94ec_0    r\n",
      "r-rcolorbrewer            1.1_3             r43h785f33e_2    conda-forge\n",
      "r-rcpp                    1.0.11            r43h884c59f_0    r\n",
      "r-rcppeigen               0.3.3.9.3         r43h884c59f_0    r\n",
      "r-readr                   2.1.4             r43h884c59f_0    r\n",
      "r-readxl                  1.4.3             r43h884c59f_0    r\n",
      "r-recipes                 1.0.9             r43hc72bb7e_0    conda-forge\n",
      "r-recommended             4.3.1                     r43_0  \n",
      "r-rematch                 2.0.0             r43hc72bb7e_0    conda-forge\n",
      "r-rematch2                2.1.2             r43hc72bb7e_3    conda-forge\n",
      "r-repr                    1.1.6             r43h785f33e_1    conda-forge\n",
      "r-reprex                  2.1.0             r43hc72bb7e_0    conda-forge\n",
      "r-reshape2                1.4.4             r43h884c59f_0    r\n",
      "r-rlang                   1.1.1             r43h884c59f_0    r\n",
      "r-rmarkdown               2.25              r43hc72bb7e_0    conda-forge\n",
      "r-rpart                   4.1.19            r43h76d94ec_0    r\n",
      "r-rstudioapi              0.15.0            r43hc72bb7e_0    conda-forge\n",
      "r-rvest                   1.0.3             r43hc72bb7e_2    conda-forge\n",
      "r-sass                    0.4.8             r43ha503ecb_0    conda-forge\n",
      "r-scales                  1.3.0             r43hc72bb7e_0    conda-forge\n",
      "r-selectr                 0.4_2             r43hc72bb7e_3    conda-forge\n",
      "r-shape                   1.4.6             r43ha770c72_2    conda-forge\n",
      "r-shiny                   1.8.0             r43h785f33e_0    conda-forge\n",
      "r-sourcetools             0.1.7_1           r43ha503ecb_1    conda-forge\n",
      "r-spatial                 7.3_17            r43h57805ef_0    conda-forge\n",
      "r-squarem                 2021.1            r43hc72bb7e_2    conda-forge\n",
      "r-stringi                 1.7.12            r43h884c59f_0  \n",
      "r-stringr                 1.5.1             r43h785f33e_0    conda-forge\n",
      "r-survival                3.5_7             r43h57805ef_0    conda-forge\n",
      "r-sys                     3.4.2             r43h57805ef_1    conda-forge\n",
      "r-systemfonts             1.0.5             r43haf97adc_0    conda-forge\n",
      "r-textshaping             0.3.7             r43h884c59f_0  \n",
      "r-tibble                  3.2.1             r43h57805ef_2    conda-forge\n",
      "r-tidyr                   1.3.0             r43ha503ecb_1    conda-forge\n",
      "r-tidyselect              1.2.0             r43hc72bb7e_1    conda-forge\n",
      "r-tidyverse               2.0.0             r43h785f33e_1    conda-forge\n",
      "r-timechange              0.3.0             r43ha503ecb_0    conda-forge\n",
      "r-timedate                4032.109          r43hc72bb7e_0    conda-forge\n",
      "r-tinytex                 0.49              r43hc72bb7e_1    conda-forge\n",
      "r-ttr                     0.24.4            r43h57805ef_0    conda-forge\n",
      "r-tzdb                    0.4.0             r43ha503ecb_1    conda-forge\n",
      "r-utf8                    1.2.4             r43h57805ef_0    conda-forge\n",
      "r-uuid                    1.2_0             r43h57805ef_0    conda-forge\n",
      "r-vctrs                   0.6.5             r43ha503ecb_0    conda-forge\n",
      "r-viridislite             0.4.2             r43hc72bb7e_1    conda-forge\n",
      "r-vroom                   1.6.5             r43ha503ecb_0    conda-forge\n",
      "r-withr                   3.0.0             r43hc72bb7e_0    conda-forge\n",
      "r-xfun                    0.41              r43ha503ecb_0    conda-forge\n",
      "r-xml2                    1.3.5             r43h884c59f_0  \n",
      "r-xtable                  1.8_4             r43hc72bb7e_5    conda-forge\n",
      "r-xts                     0.13.2            r43h57805ef_0    conda-forge\n",
      "r-yaml                    2.3.8             r43h57805ef_0    conda-forge\n",
      "r-zoo                     1.8_12            r43h57805ef_1    conda-forge\n",
      "re2                       2022.04.01           h295c915_0  \n",
      "readline                  8.2                  h5eee18b_0  \n",
      "referencing               0.30.2          py311h06a4308_0  \n",
      "regex                     2023.10.3       py311h5eee18b_0  \n",
      "reproc                    14.2.4               h295c915_1  \n",
      "reproc-cpp                14.2.4               h295c915_1  \n",
      "requests                  2.31.0          py311h06a4308_0  \n",
      "requests-file             1.5.1              pyhd3eb1b0_0  \n",
      "requests-toolbelt         1.0.0           py311h06a4308_0  \n",
      "responses                 0.13.3             pyhd3eb1b0_0  \n",
      "rfc3339-validator         0.1.4           py311h06a4308_0  \n",
      "rfc3986-validator         0.1.1           py311h06a4308_0  \n",
      "rich                      13.3.5          py311h06a4308_0  \n",
      "rope                      1.7.0           py311h06a4308_0  \n",
      "rpds-py                   0.10.6          py311hb02cf49_0  \n",
      "rtree                     1.0.1           py311h06a4308_0  \n",
      "ruamel.yaml               0.17.21         py311h5eee18b_0  \n",
      "ruamel_yaml               0.17.21         py311h5eee18b_0  \n",
      "s3fs                      2023.10.0       py311h06a4308_0  \n",
      "safetensors               0.4.0           py311h24d97f6_0  \n",
      "scikit-image              0.20.0          py311h6a678d5_0  \n",
      "scikit-learn              1.4.0           py311hc009520_0    conda-forge\n",
      "scikit-learn-intelex      2023.1.1        py311h06a4308_0  \n",
      "scipy                     1.12.0          py311h64a7726_1    conda-forge\n",
      "scrapy                    2.8.0           py311h06a4308_0  \n",
      "seaborn                   0.13.1               hd8ed1ab_0    conda-forge\n",
      "seaborn-base              0.13.1             pyhd8ed1ab_0    conda-forge\n",
      "secretstorage             3.3.1           py311h06a4308_1  \n",
      "semver                    2.13.0             pyhd3eb1b0_0  \n",
      "send2trash                1.8.2           py311h06a4308_0  \n",
      "service_identity          18.1.0             pyhd3eb1b0_1  \n",
      "setuptools                67.8.0          py311h06a4308_0  \n",
      "sip                       6.7.12          py311h6a678d5_0  \n",
      "six                       1.16.0             pyhd3eb1b0_1  \n",
      "smart_open                5.2.1           py311h06a4308_0  \n",
      "snappy                    1.1.10               h6a678d5_1  \n",
      "sniffio                   1.3.0           py311h06a4308_0  \n",
      "snowballstemmer           2.2.0              pyhd3eb1b0_0  \n",
      "sortedcontainers          2.4.0              pyhd3eb1b0_0  \n",
      "soupsieve                 2.5             py311h06a4308_0  \n",
      "sphinx                    5.0.2           py311h06a4308_0  \n",
      "sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0  \n",
      "sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0  \n",
      "sphinxcontrib-htmlhelp    2.0.0              pyhd3eb1b0_0  \n",
      "sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0  \n",
      "sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0  \n",
      "sphinxcontrib-serializinghtml 1.1.5              pyhd3eb1b0_0  \n",
      "spyder                    5.4.3           py311h06a4308_1  \n",
      "spyder-kernels            2.4.4           py311h06a4308_0  \n",
      "sqlalchemy                2.0.25          py311h5eee18b_0  \n",
      "sqlite                    3.41.2               h5eee18b_0  \n",
      "stack_data                0.2.0              pyhd3eb1b0_0  \n",
      "statsmodels               0.14.0          py311hf4808d0_0  \n",
      "sympy                     1.12            py311h06a4308_0  \n",
      "sysroot_linux-64          2.12                he073ed8_16    conda-forge\n",
      "tabulate                  0.9.0           py311h06a4308_0  \n",
      "tbb                       2021.8.0             hdb19cb5_0  \n",
      "tbb4py                    2021.8.0        py311hdb19cb5_0  \n",
      "tblib                     1.7.0              pyhd3eb1b0_0  \n",
      "tenacity                  8.2.2           py311h06a4308_0  \n",
      "terminado                 0.17.1          py311h06a4308_0  \n",
      "text-unidecode            1.3                pyhd3eb1b0_0  \n",
      "textdistance              4.2.1              pyhd3eb1b0_0  \n",
      "threadpoolctl             2.2.0              pyh0d69192_0  \n",
      "three-merge               0.1.1              pyhd3eb1b0_0  \n",
      "tifffile                  2023.4.12       py311h06a4308_0  \n",
      "tinycss2                  1.2.1           py311h06a4308_0  \n",
      "tk                        8.6.13          noxft_h4845f30_101    conda-forge\n",
      "tktable                   2.10                 h0c5db8f_4    conda-forge\n",
      "tldextract                3.2.0              pyhd3eb1b0_0  \n",
      "tokenizers                0.13.3          py311h22610ee_0  \n",
      "toml                      0.10.2             pyhd3eb1b0_0  \n",
      "tomlkit                   0.11.1          py311h06a4308_0  \n",
      "toolz                     0.12.0          py311h06a4308_0  \n",
      "tornado                   6.3.3           py311h5eee18b_0  \n",
      "tqdm                      4.65.0          py311h92b7b1e_0  \n",
      "traitlets                 5.7.1           py311h06a4308_0  \n",
      "traittypes                0.2.1              pyh9f0ad1d_2    conda-forge\n",
      "transformers              4.32.1          py311h06a4308_0  \n",
      "truststore                0.8.0           py311h06a4308_0  \n",
      "twisted                   22.10.0         py311h5eee18b_0  \n",
      "typing-extensions         4.9.0           py311h06a4308_1  \n",
      "typing_extensions         4.9.0           py311h06a4308_1  \n",
      "tzdata                    2023d                h04d1e81_0  \n",
      "uc-micro-py               1.0.1           py311h06a4308_0  \n",
      "ujson                     5.4.0           py311h6a678d5_0  \n",
      "unidecode                 1.2.0              pyhd3eb1b0_0  \n",
      "unixodbc                  2.3.11               h5eee18b_0  \n",
      "uri-template              1.3.0              pyhd8ed1ab_0    conda-forge\n",
      "urllib3                   1.26.18         py311h06a4308_0  \n",
      "utf8proc                  2.6.1                h27cfd23_0  \n",
      "w3lib                     1.21.0             pyhd3eb1b0_0  \n",
      "watchdog                  2.1.6           py311h06a4308_0  \n",
      "wcwidth                   0.2.5              pyhd3eb1b0_0  \n",
      "webcolors                 1.13               pyhd8ed1ab_0    conda-forge\n",
      "webencodings              0.5.1           py311h06a4308_1  \n",
      "websocket-client          0.58.0          py311h06a4308_4  \n",
      "werkzeug                  2.2.3           py311h06a4308_0  \n",
      "whatthepatch              1.0.2           py311h06a4308_0  \n",
      "wheel                     0.38.4          py311h06a4308_0  \n",
      "widgetsnbextension        4.0.9              pyhd8ed1ab_0    conda-forge\n",
      "wrapt                     1.14.1          py311h5eee18b_0  \n",
      "wurlitzer                 3.0.2           py311h06a4308_0  \n",
      "xarray                    2023.6.0        py311h06a4308_0  \n",
      "xxhash                    0.8.0                h7f8727e_3  \n",
      "xyzservices               2022.9.0        py311h06a4308_1  \n",
      "xz                        5.4.5                h5eee18b_0  \n",
      "yaml                      0.2.5                h7b6447c_0  \n",
      "yaml-cpp                  0.8.0                h6a678d5_0  \n",
      "yapf                      0.31.0             pyhd3eb1b0_0  \n",
      "yarl                      1.9.3           py311h5eee18b_0  \n",
      "zeromq                    4.3.5                h6a678d5_0  \n",
      "zfp                       1.0.0                h6a678d5_0  \n",
      "zict                      3.0.0           py311h06a4308_0  \n",
      "zipp                      3.17.0          py311h06a4308_0  \n",
      "zlib                      1.2.13               hd590300_5    conda-forge\n",
      "zlib-ng                   2.0.7                h5eee18b_0  \n",
      "zope                      1.0             py311h06a4308_1  \n",
      "zope.interface            5.4.0           py311h5eee18b_0  \n",
      "zstandard                 0.19.0          py311h5eee18b_0  \n",
      "zstd                      1.5.5                hc292b87_0  \n"
     ]
    }
   ],
   "source": [
    "! conda list"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4a1a71b0-acda-4b40-966b-ae03b40befa3",
   "metadata": {},
   "source": [
    "作業ディレクトリを変更する。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "2c12faca-d37a-4c1b-87c0-81b617dcd695",
   "metadata": {},
   "outputs": [],
   "source": [
    "! cd /home/inou_doi/demo/7_motif"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "a9d0a0be-03fa-456f-8c93-f14d41d64a87",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/inou_doi/demo/7_motif\n"
     ]
    }
   ],
   "source": [
    "! pwd"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a9143452-ba4e-40fc-bd31-1b6eda86bb9c",
   "metadata": {},
   "source": [
    "使用方法/動作を確認するために公式helpを出力する。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "f314b9f2-009b-4486-8b55-f49e3bac6f6b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tCurrent base directory for HOMER is /opt/conda/envs/homer/share/homer/\n",
      "\n",
      "\n",
      "\tUsage: configureHomer.pl [options]\n",
      "\tThis program will install HOMER in the directory containing configureHomer.pl\n",
      "\t\ti.e. save this in a directory named homer/ or something like that\n",
      "\tTo install the program from scratch, run the following command:\n",
      "\t\tperl path-to-homer/configureHomer.pl -install\n",
      "\tIf upgrading, make sure HOMER is not running (may prevent replacement of executables)\n",
      "\n",
      "\tOptions:\n",
      "\t\t-list (prints available and installed packages to screen)\n",
      "\t\t-install (install basic Homer software from scratch)\n",
      "\t\t-install <package name1> [package name2] ... (installs packages)\n",
      "\t\t-version <version> (install specific version of package, i.e. '-version v4.9')\n",
      "\t\t\tOlder version numbers can be found here: http://homer.ucsd.edu/homer/data/\n",
      "\t\t-nodep (do NOT include additional dependent packages during installation)\n",
      "\t\t-remove <package name1> [package name2] ... (safely removes packages)\n",
      "\t\t-update (updates all packages to the newest version)\n",
      "\t\t-reinstall (forces the reinstallation of all currently installed packages)\n",
      "\t\t-all (install everything - will take a while...)\n",
      "\t\t-getFacts (add humor to HOMER - to remove delete contents of data/misc/)\n",
      "\t\t-check (checks for required 3rd party software)\n",
      "\t\t-make (reconfigure files, recompile exectables)\n",
      "\t\t-sun (Use with SunOS - i.e. gmake and gtar instead of make and tar)\n",
      "\t\t-keepScript (do NOT replace configureHomer.pl script if changes are detected with website)\n",
      "\t\t-url <installation url> (For use if the Homer website changes)\n",
      "\t\t\tdefault is: http://homer.ucsd.edu/homer/\n",
      "\t\tHubs & BigWig settings (with read existing settings from config.txt if upgrading):\n",
      "\t\t\t-bigWigUrl <base urls for bigWigs> (Setting for makeBigWigs.pl)\n",
      "\t\t\t-bigWigDir <base directory for bigWigs> (Setting for makeBigWigs.pl)\n",
      "\t\t\t-hubsUrl <base urls for hubs> (Setting for makeMultiWigHub.pl)\n",
      "\t\t\t-hubsDir <base directory for hubs> (Setting for makeMultiWigHub.pl)\n",
      "\tConfiguration files: downloads update.txt from the homer website,\n",
      "\t\tand corrects config.txt with the appropriate options\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! perl /opt/conda/envs/homer/share/homer/configureHomer.pl"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d0449c7d-2f0b-4d07-96a0-c1dbeb634f96",
   "metadata": {},
   "source": [
    "HOMERへリファレンスゲノムデータをインストールしておく。\n",
    "-installオプションで、mouseの場合mm10、humanの場合hg38を指定すると良い。  \n",
    "今回は使用データがhg19だったため、hg19を使用した。  \n",
    "→hg19が使用可能であることが確認できた。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "d689548a-aa2c-40e9-b8af-171b16b6a2f7",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tCurrent base directory for HOMER is /opt/conda/envs/homer/share/homer/\n",
      "\n",
      "\tWill install hg19\n",
      "--2024-11-22 06:33:43--  http://homer.ucsd.edu/homer/update.txt\n",
      "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
      "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 19497 (19K) [text/plain]\n",
      "Saving to: ‘/opt/conda/envs/homer/share/homer//update.txt’\n",
      "\n",
      "/opt/conda/envs/hom 100%[===================>]  19.04K  --.-KB/s    in 0.1s    \n",
      "\n",
      "2024-11-22 06:33:44 (166 KB/s) - ‘/opt/conda/envs/homer/share/homer//update.txt’ saved [19497/19497]\n",
      "\n",
      "\tUpdating Settings...\n",
      "`wget -O 0.124140714368881.tmp http://homer.ucsd.edu/homer/configureHomer.pl`;\n",
      "--2024-11-22 06:33:44--  http://homer.ucsd.edu/homer/configureHomer.pl\n",
      "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
      "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 27679 (27K) [application/x-perl]\n",
      "Saving to: ‘0.124140714368881.tmp’\n",
      "\n",
      "0.124140714368881.t 100%[===================>]  27.03K  --.-KB/s    in 0.1s    \n",
      "\n",
      "2024-11-22 06:33:44 (225 KB/s) - ‘0.124140714368881.tmp’ saved [27679/27679]\n",
      "\n",
      "\tconfigureHomer.pl script is up-to-date\n",
      "\t\tInstalling/Updating Organism human (v7.0) for package hg19\n",
      "\tPackages to Install...\n",
      "\t\thuman -> v7.0\n",
      "\t\thg19 -> \n",
      "\tPausing for 3 seconds (hit ctrl-C to stop)\n",
      "\tInstalling: human\n",
      "\t\tDownloading...\n",
      "--2024-11-22 06:33:47--  http://homer.ucsd.edu/homer/data/organisms/human.v7.0.zip\n",
      "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
      "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 60651483 (58M) [application/zip]\n",
      "Saving to: ‘homer.package.zip’\n",
      "\n",
      "homer.package.zip   100%[===================>]  57.84M  10.8MB/s    in 6.1s    \n",
      "\n",
      "2024-11-22 06:33:53 (9.51 MB/s) - ‘homer.package.zip’ saved [60651483/60651483]\n",
      "\n",
      "\t\tUnzipping...\n",
      "\t\tFinished Installing human\n",
      "\n",
      "\tInstalling: hg19\n",
      "\t\tDownloading...\n",
      "--2024-11-22 06:33:55--  http://homer.ucsd.edu/homer/data/genomes/hg19.v7.0.zip\n",
      "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
      "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1511297282 (1.4G) [application/zip]\n",
      "Saving to: ‘homer.package.zip’\n",
      "\n",
      "homer.package.zip   100%[===================>]   1.41G  11.2MB/s    in 2m 11s  \n",
      "\n",
      "2024-11-22 06:36:06 (11.0 MB/s) - ‘homer.package.zip’ saved [1511297282/1511297282]\n",
      "\n",
      "\t\tUnzipping...\n",
      "\t\tFinished Installing hg19\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! perl /opt/conda/envs/homer/share/homer/configureHomer.pl -install hg19"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3c0bffac-0e25-42d7-932d-8b378e9cd1ad",
   "metadata": {},
   "source": [
    "使用方法/動作を確認するために公式helpを出力する。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "49a017c6-c2df-44a4-829d-b3ea935f4aee",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tProgram will find de novo and known motifs in regions in the genome\n",
      "\n",
      "\tUsage: findMotifsGenome.pl <pos file> <genome> <output directory> [additional options]\n",
      "\tExample: findMotifsGenome.pl peaks.txt mm8r peakAnalysis -size 200 -len 8\n",
      "\n",
      "\tPossible Genomes:\n",
      "\t\thg19\thuman\n",
      "\t\t\t-- or --\n",
      "\t\tCustom: provide the path to genome FASTA files (directory or single file)\n",
      "\t\t\tHeads up: will create the directory \"preparsed/\" in same location.\n",
      "\n",
      "\tBasic options:\n",
      "\t\t-mask (mask repeats/lower case sequence, can also add 'r' to genome, i.e. mm9r)\n",
      "\t\t-bg <background position file> (genomic positions to be used as background, default=automatic)\n",
      "\t\t\tremoves background positions overlapping with target positions unless -keepOverlappingBg is used\n",
      "\t\t\t-chopify (chop up large background regions to the avg size of target regions)\n",
      "\t\t-len <#>[,<#>,<#>...] (motif length, default=8,10,12) [NOTE: values greater 12 may cause the program\n",
      "\t\t\tto run out of memory - in these cases decrease the number of sequences analyzed (-N),\n",
      "\t\t\tor try analyzing shorter sequence regions (i.e. -size 100)]\n",
      "\t\t-size <#> (fragment size to use for motif finding, default=200)\n",
      "\t\t\t-size <#,#> (i.e. -size -100,50 will get sequences from -100 to +50 relative from center)\n",
      "\t\t\t-size given (uses the exact regions you give it)\n",
      "\t\t-S <#> (Number of motifs to optimize, default: 25)\n",
      "\t\t-mis <#> (global optimization: searches for strings with # mismatches, default: 2)\n",
      "\t\t-norevopp (don't search reverse strand for motifs)\n",
      "\t\t-nomotif (don't search for de novo motif enrichment)\n",
      "\t\t-rna (output RNA motif logos and compare to RNA motif database, automatically sets -norevopp)\n",
      "\n",
      "\tScanning sequence for motifs\n",
      "\t\t-find <motif file> (This will cause the program to only scan for motifs)\n",
      "\n",
      "\tKnown Motif Options/Visualization\n",
      "\t\t-mset <vertebrates|insects|worms|plants|yeast|all> (check against motif collects, default: auto)\n",
      "\t\t-basic (just visualize de novo motifs, don't check similarity with known motifs)\n",
      "\t\t-bits (scale sequence logos by information content, default: doesn't scale)\n",
      "\t\t-nocheck (don't search for de novo vs. known motif similarity)\n",
      "\t\t-mcheck <motif file> (known motifs to check against de novo motifs,\n",
      "\t\t-float (allow adjustment of the degeneracy threshold for known motifs to improve p-value[dangerous])\n",
      "\t\t-noknown (don't search for known motif enrichment, default: -known)\n",
      "\t\t-mknown <motif file> (known motifs to check for enrichment,\n",
      "\t\t-nofacts (omit humor)\n",
      "\t\t-seqlogo (use weblogo/seqlogo/ghostscript to generate logos, default uses SVG now)\n",
      "\n",
      "\tSequence normalization options:\n",
      "\t\t-gc (use GC% for sequence content normalization, now the default)\n",
      "\t\t-cpg (use CpG% instead of GC% for sequence content normalization)\n",
      "\t\t-noweight (no CG correction)\n",
      "\t\tAlso -nlen <#>, -olen <#>, see homer2 section below.\n",
      "\n",
      "\tAdvanced options:\n",
      "\t\t-h (use hypergeometric for p-values, binomial is default)\n",
      "\t\t-N <#> (Number of sequences to use for motif finding, default=max(50k, 2x input)\n",
      "\t\t-local <#> (use local background, # of equal size regions around peaks to use i.e. 2)\n",
      "\t\t-redundant <#> (Remove redundant sequences matching greater than # percent, i.e. -redundant 0.5)\n",
      "\t\t-maxN <#> (maximum percentage of N's in sequence to consider for motif finding, default: 0.7)\n",
      "\t\t-maskMotif <motif file1> [motif file 2]... (motifs to mask before motif finding)\n",
      "\t\t-opt <motif file1> [motif file 2]... (motifs to optimize or change length of)\n",
      "\t\t-rand (randomize target and background sequences labels)\n",
      "\t\t-ref <peak file> (use file for target and background - first argument is list of peak ids for targets)\n",
      "\t\t-oligo (perform analysis of individual oligo enrichment)\n",
      "\t\t-dumpFasta (Dump fasta files for target and background sequences for use with other programs)\n",
      "\t\t-preparse (force new background files to be created)\n",
      "\t\t-preparsedDir <directory> (location to search for preparsed file and/or place new files)\n",
      "\t\t-keepFiles (keep temporary files)\n",
      "\t\t-fdr <#> (Calculate empirical FDR for de novo discovery #=number of randomizations)\n",
      "\n",
      "\thomer2 specific options:\n",
      "\t\t-homer2 (use homer2 instead of original homer, default)\n",
      "\t\t-nlen <#> (length of lower-order oligos to normalize in background, default: -nlen 3)\n",
      "\t\t\t-nmax <#> (Max normalization iterations, default: 160)\n",
      "\t\t\t-neutral (weight sequences to neutral frequencies, i.e. 25%, 6.25%, etc.)\n",
      "\t\t-olen <#> (lower-order oligo normalization for oligo table, use if -nlen isn't working well)\n",
      "\t\t-p <#> (Number of processors to use, default: 1)\n",
      "\t\t-e <#> (Maximum expected motif instance per bp in random sequence, default: 0.01)\n",
      "\t\t-cache <#> (size in MB for statistics cache, default: 500)\n",
      "\t\t-quickMask (skip full masking after finding motifs, similar to original homer)\n",
      "\t\t-minlp <#> (stop looking for motifs when seed logp score gets above #, default: -10)\n",
      "\n",
      "\tOriginal homer specific options:\n",
      "\t\t-homer1 (to force the use of the original homer)\n",
      "\t\t-depth [low|med|high|allnight] (time spent on local optimization default: med)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! findMotifsGenome.pl"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "a2c0584b-3637-47a8-be20-2296f9609513",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tProgram will find de novo and known motifs in regions in the genome\n",
      "\n",
      "\tUsage: findMotifsGenome.pl <pos file> <genome> <output directory> [additional options]\n",
      "\tExample: findMotifsGenome.pl peaks.txt mm8r peakAnalysis -size 200 -len 8\n",
      "\n",
      "\tPossible Genomes:\n",
      "\t\thg19\thuman\n",
      "\t\t\t-- or --\n",
      "\t\tCustom: provide the path to genome FASTA files (directory or single file)\n",
      "\t\t\tHeads up: will create the directory \"preparsed/\" in same location.\n",
      "\n",
      "\tBasic options:\n",
      "\t\t-mask (mask repeats/lower case sequence, can also add 'r' to genome, i.e. mm9r)\n",
      "\t\t-bg <background position file> (genomic positions to be used as background, default=automatic)\n",
      "\t\t\tremoves background positions overlapping with target positions unless -keepOverlappingBg is used\n",
      "\t\t\t-chopify (chop up large background regions to the avg size of target regions)\n",
      "\t\t-len <#>[,<#>,<#>...] (motif length, default=8,10,12) [NOTE: values greater 12 may cause the program\n",
      "\t\t\tto run out of memory - in these cases decrease the number of sequences analyzed (-N),\n",
      "\t\t\tor try analyzing shorter sequence regions (i.e. -size 100)]\n",
      "\t\t-size <#> (fragment size to use for motif finding, default=200)\n",
      "\t\t\t-size <#,#> (i.e. -size -100,50 will get sequences from -100 to +50 relative from center)\n",
      "\t\t\t-size given (uses the exact regions you give it)\n",
      "\t\t-S <#> (Number of motifs to optimize, default: 25)\n",
      "\t\t-mis <#> (global optimization: searches for strings with # mismatches, default: 2)\n",
      "\t\t-norevopp (don't search reverse strand for motifs)\n",
      "\t\t-nomotif (don't search for de novo motif enrichment)\n",
      "\t\t-rna (output RNA motif logos and compare to RNA motif database, automatically sets -norevopp)\n",
      "\n",
      "\tScanning sequence for motifs\n",
      "\t\t-find <motif file> (This will cause the program to only scan for motifs)\n",
      "\n",
      "\tKnown Motif Options/Visualization\n",
      "\t\t-mset <vertebrates|insects|worms|plants|yeast|all> (check against motif collects, default: auto)\n",
      "\t\t-basic (just visualize de novo motifs, don't check similarity with known motifs)\n",
      "\t\t-bits (scale sequence logos by information content, default: doesn't scale)\n",
      "\t\t-nocheck (don't search for de novo vs. known motif similarity)\n",
      "\t\t-mcheck <motif file> (known motifs to check against de novo motifs,\n",
      "\t\t-float (allow adjustment of the degeneracy threshold for known motifs to improve p-value[dangerous])\n",
      "\t\t-noknown (don't search for known motif enrichment, default: -known)\n",
      "\t\t-mknown <motif file> (known motifs to check for enrichment,\n",
      "\t\t-nofacts (omit humor)\n",
      "\t\t-seqlogo (use weblogo/seqlogo/ghostscript to generate logos, default uses SVG now)\n",
      "\n",
      "\tSequence normalization options:\n",
      "\t\t-gc (use GC% for sequence content normalization, now the default)\n",
      "\t\t-cpg (use CpG% instead of GC% for sequence content normalization)\n",
      "\t\t-noweight (no CG correction)\n",
      "\t\tAlso -nlen <#>, -olen <#>, see homer2 section below.\n",
      "\n",
      "\tAdvanced options:\n",
      "\t\t-h (use hypergeometric for p-values, binomial is default)\n",
      "\t\t-N <#> (Number of sequences to use for motif finding, default=max(50k, 2x input)\n",
      "\t\t-local <#> (use local background, # of equal size regions around peaks to use i.e. 2)\n",
      "\t\t-redundant <#> (Remove redundant sequences matching greater than # percent, i.e. -redundant 0.5)\n",
      "\t\t-maxN <#> (maximum percentage of N's in sequence to consider for motif finding, default: 0.7)\n",
      "\t\t-maskMotif <motif file1> [motif file 2]... (motifs to mask before motif finding)\n",
      "\t\t-opt <motif file1> [motif file 2]... (motifs to optimize or change length of)\n",
      "\t\t-rand (randomize target and background sequences labels)\n",
      "\t\t-ref <peak file> (use file for target and background - first argument is list of peak ids for targets)\n",
      "\t\t-oligo (perform analysis of individual oligo enrichment)\n",
      "\t\t-dumpFasta (Dump fasta files for target and background sequences for use with other programs)\n",
      "\t\t-preparse (force new background files to be created)\n",
      "\t\t-preparsedDir <directory> (location to search for preparsed file and/or place new files)\n",
      "\t\t-keepFiles (keep temporary files)\n",
      "\t\t-fdr <#> (Calculate empirical FDR for de novo discovery #=number of randomizations)\n",
      "\n",
      "\thomer2 specific options:\n",
      "\t\t-homer2 (use homer2 instead of original homer, default)\n",
      "\t\t-nlen <#> (length of lower-order oligos to normalize in background, default: -nlen 3)\n",
      "\t\t\t-nmax <#> (Max normalization iterations, default: 160)\n",
      "\t\t\t-neutral (weight sequences to neutral frequencies, i.e. 25%, 6.25%, etc.)\n",
      "\t\t-olen <#> (lower-order oligo normalization for oligo table, use if -nlen isn't working well)\n",
      "\t\t-p <#> (Number of processors to use, default: 1)\n",
      "\t\t-e <#> (Maximum expected motif instance per bp in random sequence, default: 0.01)\n",
      "\t\t-cache <#> (size in MB for statistics cache, default: 500)\n",
      "\t\t-quickMask (skip full masking after finding motifs, similar to original homer)\n",
      "\t\t-minlp <#> (stop looking for motifs when seed logp score gets above #, default: -10)\n",
      "\n",
      "\tOriginal homer specific options:\n",
      "\t\t-homer1 (to force the use of the original homer)\n",
      "\t\t-depth [low|med|high|allnight] (time spent on local optimization default: med)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! /opt/conda/envs/homer/share/homer/bin/findMotifsGenome.pl"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4a0913a9-f3f3-472a-948e-6b7036dd1a33",
   "metadata": {},
   "source": [
    "### モチーフ検索（ピークコールはmacs2)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dff59f66-9b0e-4653-9f26-45461cc342e5",
   "metadata": {},
   "source": [
    "作業ディレクトリを変更する。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "26942274-63c3-4693-b3a4-bedc5fabc954",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/inou_doi/demo/7_motif\n"
     ]
    }
   ],
   "source": [
    "%cd /home/inou_doi/demo/7_motif"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "490d3878-3d04-428c-a236-d3e1a7b1930f",
   "metadata": {},
   "source": [
    "使用するデータファイルについて内容を確認する。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "6229c502-af07-41a3-adf2-28e29ccb408a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr1\t755431\t755432\tSRR7475870_peak_1\t5.84759\n",
      "chr1\t762866\t762867\tSRR7475870_peak_2\t20.73892\n",
      "chr1\t805500\t805501\tSRR7475870_peak_3\t3.85079\n",
      "chr1\t848541\t848542\tSRR7475870_peak_4\t2.94547\n",
      "chr1\t849441\t849442\tSRR7475870_peak_5\t3.85079\n",
      "chr1\t894700\t894701\tSRR7475870_peak_6\t15.00052\n",
      "chr1\t902322\t902323\tSRR7475870_peak_7\t4.31729\n",
      "chr1\t909240\t909241\tSRR7475870_peak_8\t4.24073\n",
      "chr1\t914671\t914672\tSRR7475870_peak_9\t2.47933\n",
      "chr1\t919743\t919744\tSRR7475870_peak_10\t3.85079\n"
     ]
    }
   ],
   "source": [
    "! head /home/inou_doi/demo/3_peak_call/MACS2/SRR7475870_summits.bed"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8a8bb556-4f4b-4d5b-ad53-b5b6dca305c8",
   "metadata": {},
   "source": [
    "findMotifsGenome.pl  \n",
    "HOMERツールを使用して、MACS2で検出されたピーク領域の中心付近（summits）でDNAモチーフの発見とエンリッチメント解析を行う。  \n",
    "入力ファイル、使用するリファレンスゲノム、出力先のディレクトリはそのまま記載する。  \n",
    "--size 200 各ピークの中心から上下100bpを対象にする。デフォルトは領域全体が解析される。   \n",
    "-mask　繰り返し配列を無視して解析する。  \n",
    "-p 12 モチーフ解析を12スレッドで実行する。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "5f6587fc-6740-4f1e-8dd0-a7194d2cb225",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = /home/inou_doi/demo/3_peak_call/MACS2/SRR7475870_summits.bed\n",
      "\tGenome = hg19\n",
      "\tOutput Directory = SRR7475870_motif_macs2\n",
      "\tFragment size set to 200\n",
      "\tWill use repeat masked sequences\n",
      "\tUsing 12 CPUs\n",
      "\tFound mset for \"human\", will check against vertebrates motifs\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 157911\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 157911\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "ls: cannot access '/opt/conda/envs/homer/share/homer/.//data/genomes/hg19//preparsed//hg19r.*.cgbins': No such file or directory\n",
      "\tCould not find background files for 200 bp fragments\n",
      "\tBelow are the sizes that are already available prepared.\n",
      "\tHOMER will now create background files for 200 bp fragments\n",
      "\tTo CANCEL and rerun with a differet \"-size <#>\", hit <CTRL+C> now!\n",
      "\t\t5\n",
      "\t\t4\n",
      "\t\t3\n",
      "\t\t2\n",
      "\t\t1\n",
      "\tPreparsing genome for 200 bp fragments...(will probably take 1-5 min)\n",
      "\tpreparse size set to 200\n",
      "\n",
      "\tBy default, using /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//hg19.tss for reference positions\n",
      "\tOutput files will be placed in /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//preparsed//hg19r.200.* ...\n",
      "\tBy default, HOMER will set the the preparsed Directory to be group read/write\n",
      "\tExtracting sequences\n",
      "\tCustom genome sequence directory: /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//\n",
      "\n",
      "\tExtracting sequences from directory: /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//\n",
      "\tExtracting 96908 sequences from chr1\n",
      "\tExtracting 73734 sequences from chr2\n",
      "\tExtracting 63527 sequences from chr3\n",
      "\tExtracting 48993 sequences from chr4\n",
      "\tExtracting 52580 sequences from chr5\n",
      "\tExtracting 54450 sequences from chr6\n",
      "\tExtracting 51504 sequences from chr7\n",
      "\tExtracting 42110 sequences from chr8\n",
      "\tExtracting 41421 sequences from chr9\n",
      "\tExtracting 46085 sequences from chr10\n",
      "\tExtracting 53576 sequences from chr11\n",
      "\tExtracting 50419 sequences from chr12\n",
      "\tExtracting 26508 sequences from chr13\n",
      "\tExtracting 30132 sequences from chr14\n",
      "\tExtracting 35199 sequences from chr15\n",
      "\tExtracting 33933 sequences from chr16\n",
      "\tExtracting 44454 sequences from chr17\n",
      "\tExtracting 20724 sequences from chr18\n",
      "\tExtracting 42151 sequences from chr19\n",
      "\tExtracting 26006 sequences from chr20\n",
      "\tExtracting 13944 sequences from chr21\n",
      "\tExtracting 20245 sequences from chr22\n",
      "\tExtracting 45168 sequences from chrX\n",
      "\tExtracting 6008 sequences from chrY\n",
      "\tExtracting 38 sequences from chrM\n",
      "\tExtracting 805 sequences from chr17_ctg5_hap1\n",
      "\tExtracting 59 sequences from chr17_gl000205_random\n",
      "\tExtracting 189 sequences from chr19_gl000209_random\n",
      "\tExtracting 50 sequences from chr1_gl000191_random\n",
      "\tExtracting 67 sequences from chr1_gl000192_random\n",
      "\tExtracting 331 sequences from chr4_ctg9_hap1\n",
      "\tExtracting 57 sequences from chr4_gl000193_random\n",
      "\tExtracting 53 sequences from chr4_gl000194_random\n",
      "\tExtracting 2679 sequences from chr6_apd_hap1\n",
      "\tExtracting 4556 sequences from chr6_cox_hap2\n",
      "\tExtracting 4199 sequences from chr6_dbb_hap3\n",
      "\tExtracting 4017 sequences from chr6_mann_hap4\n",
      "\tExtracting 4189 sequences from chr6_mcf_hap5\n",
      "\tExtracting 4223 sequences from chr6_qbl_hap6\n",
      "\tExtracting 4166 sequences from chr6_ssto_hap7\n",
      "\tExtracting 80 sequences from chr7_gl000195_random\n",
      "\tExtracting 65 sequences from chrUn_gl000211\n",
      "\tExtracting 46 sequences from chrUn_gl000212\n",
      "\tExtracting 127 sequences from chrUn_gl000213\n",
      "\tExtracting 74 sequences from chrUn_gl000215\n",
      "\tExtracting 103 sequences from chrUn_gl000218\n",
      "\tExtracting 114 sequences from chrUn_gl000219\n",
      "\tExtracting 173 sequences from chrUn_gl000220\n",
      "\tExtracting 106 sequences from chrUn_gl000222\n",
      "\tExtracting 80 sequences from chrUn_gl000223\n",
      "\tExtracting 56 sequences from chrUn_gl000224\n",
      "\tExtracting 64 sequences from chrUn_gl000227\n",
      "\tExtracting 95 sequences from chrUn_gl000228\n",
      "\tExtracting 58 sequences from chrUn_gl000241\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t370563 of 1050434 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 201 bp\n",
      "\t\t100000\n",
      "\t\t200000\n",
      "\t\t300000\n",
      "\t\t400000\n",
      "\t\t500000\n",
      "\t\t600000\n",
      "\t\t679871 total\n",
      "\n",
      "\tFrequency Bins: 0.01 0.02 0.03 0.04 0.05 0.06 0.08 0.1 0.12 0.14 0.16\n",
      "\tFreq\tBin\tCount\n",
      "\t0.01\t0\t300072\n",
      "\t0.02\t1\t156259\n",
      "\t0.03\t2\t68225\n",
      "\t0.04\t3\t36583\n",
      "\t0.05\t4\t22894\n",
      "\t0.06\t5\t16665\n",
      "\t0.08\t6\t25348\n",
      "\t0.1\t7\t20127\n",
      "\t0.12\t8\t16319\n",
      "\t0.14\t9\t10336\n",
      "\t0.16\t10\t5007\n",
      "\t11\t11\t2036\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t973\n",
      "\t0.25\t1\t10467\n",
      "\t0.3\t2\t47670\n",
      "\t0.35\t3\t90103\n",
      "\t0.4\t4\t101984\n",
      "\t0.45\t5\t93798\n",
      "\t0.5\t6\t77951\n",
      "\t0.6\t7\t134393\n",
      "\t0.7\t8\t87875\n",
      "\t0.8\t9\t33016\n",
      "\t10\t10\t1641\n",
      "\n",
      "\tCustom genome sequence directory: /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//\n",
      "\n",
      "\tExtracting sequences from directory: /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//\n",
      "\tExtracting 13942 sequences from chr1\n",
      "\tExtracting 13630 sequences from chr2\n",
      "\tExtracting 10846 sequences from chr3\n",
      "\tExtracting 8562 sequences from chr4\n",
      "\tExtracting 9662 sequences from chr5\n",
      "\tExtracting 8983 sequences from chr6\n",
      "\tExtracting 8963 sequences from chr7\n",
      "\tExtracting 8009 sequences from chr8\n",
      "\tExtracting 6680 sequences from chr9\n",
      "\tExtracting 8080 sequences from chr10\n",
      "\tExtracting 7535 sequences from chr11\n",
      "\tExtracting 8050 sequences from chr12\n",
      "\tExtracting 4481 sequences from chr13\n",
      "\tExtracting 5047 sequences from chr14\n",
      "\tExtracting 4796 sequences from chr15\n",
      "\tExtracting 5132 sequences from chr16\n",
      "\tExtracting 5384 sequences from chr17\n",
      "\tExtracting 4190 sequences from chr18\n",
      "\tExtracting 5046 sequences from chr19\n",
      "\tExtracting 4024 sequences from chr20\n",
      "\tExtracting 1929 sequences from chr21\n",
      "\tExtracting 2885 sequences from chr22\n",
      "\tExtracting 1572 sequences from chrX\n",
      "\tExtracting 92 sequences from chrY\n",
      "\tExtracting 3 sequences from chr11_gl000202_random\n",
      "\tExtracting 1 sequences from chr17_ctg5_hap1\n",
      "\tExtracting 1 sequences from chr17_gl000203_random\n",
      "\tExtracting 4 sequences from chr17_gl000204_random\n",
      "\tExtracting 15 sequences from chr17_gl000205_random\n",
      "\tExtracting 8 sequences from chr19_gl000208_random\n",
      "\tExtracting 1 sequences from chr19_gl000209_random\n",
      "\tExtracting 1 sequences from chr1_gl000191_random\n",
      "\tExtracting 5 sequences from chr1_gl000192_random\n",
      "\tExtracting 34 sequences from chr4_gl000193_random\n",
      "\tExtracting 18 sequences from chr4_gl000194_random\n",
      "\tExtracting 3 sequences from chr6_cox_hap2\n",
      "\tExtracting 2 sequences from chr6_dbb_hap3\n",
      "\tExtracting 4 sequences from chr6_mcf_hap5\n",
      "\tExtracting 4 sequences from chr6_qbl_hap6\n",
      "\tExtracting 1 sequences from chr6_ssto_hap7\n",
      "\tExtracting 21 sequences from chr7_gl000195_random\n",
      "\tExtracting 12 sequences from chr9_gl000198_random\n",
      "\tExtracting 15 sequences from chr9_gl000199_random\n",
      "\tExtracting 5 sequences from chrUn_gl000211\n",
      "\tExtracting 9 sequences from chrUn_gl000212\n",
      "\tExtracting 31 sequences from chrUn_gl000214\n",
      "\tExtracting 5 sequences from chrUn_gl000216\n",
      "\tExtracting 13 sequences from chrUn_gl000218\n",
      "\tExtracting 27 sequences from chrUn_gl000219\n",
      "\tExtracting 36 sequences from chrUn_gl000220\n",
      "\tExtracting 35 sequences from chrUn_gl000224\n",
      "\tExtracting 23 sequences from chrUn_gl000225\n",
      "\tExtracting 5 sequences from chrUn_gl000226\n",
      "\tExtracting 3 sequences from chrUn_gl000228\n",
      "\tExtracting 4 sequences from chrUn_gl000229\n",
      "\tExtracting 5 sequences from chrUn_gl000231\n",
      "\tExtracting 8 sequences from chrUn_gl000232\n",
      "\tExtracting 10 sequences from chrUn_gl000234\n",
      "\tExtracting 4 sequences from chrUn_gl000235\n",
      "\tExtracting 9 sequences from chrUn_gl000237\n",
      "\tExtracting 2 sequences from chrUn_gl000239\n",
      "\tExtracting 4 sequences from chrUn_gl000241\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t46464 of 157911 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tNot removing redundant sequences\n",
      "\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 201 bp\n",
      "\t\t100000\n",
      "\t\t111447 total\n",
      "\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t29\n",
      "\t0.25\t1\t298\n",
      "\t0.3\t2\t1876\n",
      "\t0.35\t3\t6254\n",
      "\t0.4\t4\t11924\n",
      "\t0.45\t5\t15524\n",
      "\t0.5\t6\t15813\n",
      "\t0.6\t7\t29639\n",
      "\t0.7\t8\t20482\n",
      "\t0.8\t9\t9153\n",
      "\t10\t10\t455\n",
      "\n",
      "\tTotal sequences set to 222894\n",
      "\n",
      "\tChoosing background that matches in CpG/GC content...\n",
      "\tBin\t# Targets\t# Background\tBackground Weight\n",
      "\t0\t29\t29\t1.000\n",
      "\t1\t298\t298\t1.000\n",
      "\t2\t1876\t1876\t1.000\n",
      "\t3\t6254\t6254\t1.000\n",
      "\t4\t11924\t11924\t1.000\n",
      "\t5\t15524\t15524\t1.000\n",
      "\t6\t15813\t15813\t1.000\n",
      "\t7\t29639\t29639\t1.000\n",
      "\t8\t20482\t20482\t1.000\n",
      "\t9\t9153\t9153\t1.000\n",
      "\t10\t455\t455\t1.000\n",
      "\tAssembling sequence file...\n",
      "\tNormalizing lower order oligos using homer2\n",
      "\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t24.05%\t24.10%\t0.998\n",
      "\t\tC\t25.95%\t25.90%\t1.002\n",
      "\t\tG\t25.95%\t25.90%\t1.002\n",
      "\t\tT\t24.05%\t24.10%\t0.998\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t6.47%\t6.73%\t0.961\n",
      "\t\tCA\t7.40%\t7.14%\t1.037\n",
      "\t\tGA\t6.10%\t6.21%\t0.983\n",
      "\t\tTA\t4.07%\t4.02%\t1.012\n",
      "\t\tAC\t5.05%\t4.88%\t1.035\n",
      "\t\tCC\t7.98%\t8.20%\t0.973\n",
      "\t\tGC\t6.83%\t6.62%\t1.031\n",
      "\t\tTC\t6.10%\t6.21%\t0.983\n",
      "\t\tAG\t7.63%\t7.55%\t1.010\n",
      "\t\tCG\t2.95%\t3.02%\t0.979\n",
      "\t\tGG\t7.98%\t8.20%\t0.973\n",
      "\t\tTG\t7.40%\t7.14%\t1.037\n",
      "\t\tAT\t4.89%\t4.93%\t0.992\n",
      "\t\tCT\t7.63%\t7.55%\t1.010\n",
      "\t\tGT\t5.05%\t4.88%\t1.035\n",
      "\t\tTT\t6.47%\t6.73%\t0.961\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tNormalization weights can be found in file: SRR7475870_motif_macs2/seq.autonorm.tsv\n",
      "\tConverging on autonormalization solution:\n",
      "\t...............................................................................\n",
      "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t24.05%\t24.04%\t1.000\n",
      "\t\tC\t25.95%\t25.96%\t1.000\n",
      "\t\tG\t25.95%\t25.96%\t1.000\n",
      "\t\tT\t24.05%\t24.04%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t6.47%\t6.48%\t0.998\n",
      "\t\tCA\t7.40%\t7.39%\t1.002\n",
      "\t\tGA\t6.10%\t6.11%\t0.998\n",
      "\t\tTA\t4.07%\t4.06%\t1.002\n",
      "\t\tAC\t5.05%\t5.04%\t1.003\n",
      "\t\tCC\t7.98%\t8.00%\t0.997\n",
      "\t\tGC\t6.83%\t6.81%\t1.003\n",
      "\t\tTC\t6.10%\t6.11%\t0.998\n",
      "\t\tAG\t7.63%\t7.61%\t1.002\n",
      "\t\tCG\t2.95%\t2.96%\t0.998\n",
      "\t\tGG\t7.98%\t8.00%\t0.997\n",
      "\t\tTG\t7.40%\t7.39%\t1.002\n",
      "\t\tAT\t4.89%\t4.90%\t0.998\n",
      "\t\tCT\t7.63%\t7.61%\t1.002\n",
      "\t\tGT\t5.05%\t5.04%\t1.003\n",
      "\t\tTT\t6.47%\t6.48%\t0.998\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tFinished preparing sequence/group files\n",
      "\n",
      "\t----------------------------------------------------------\n",
      "\tKnown motif enrichment\n",
      "\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\t440 motifs loaded\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 440 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tPreparing HTML output with sequence logos...\n",
      "\t\t1 of 440 (1e-3169) CTCF(Zf)/CD4+-CTCF-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t2 of 440 (1e-2150) BORIS(Zf)/K562-CTCFL-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t3 of 440 (1e-602) ETS1(ETS)/Jurkat-ETS1-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t4 of 440 (1e-590) Fli1(ETS)/CD8-FLI-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t5 of 440 (1e-539) ETV1(ETS)/GIST48-ETV1-ChIP-Seq(GSE22441)/Homer\n",
      "\t\t6 of 440 (1e-534) Elf4(ETS)/BMDM-Elf4-ChIP-Seq(GSE88699)/Homer\n",
      "\t\t7 of 440 (1e-516) ETV4(ETS)/HepG2-ETV4-ChIP-Seq(ENCODE)/Homer\n",
      "\t\t8 of 440 (1e-514) GABPA(ETS)/Jurkat-GABPa-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t9 of 440 (1e-496) Etv2(ETS)/ES-ER71-ChIP-Seq(GSE59402)/Homer\n",
      "\t\t10 of 440 (1e-453) ERG(ETS)/VCaP-ERG-ChIP-Seq(GSE14097)/Homer\n",
      "\t\t11 of 440 (1e-431) PU.1(ETS)/ThioMac-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t12 of 440 (1e-404) Sp1(Zf)/Promoter/Homer\n",
      "\t\t13 of 440 (1e-400) IRF8(IRF)/BMDM-IRF8-ChIP-Seq(GSE77884)/Homer\n",
      "\t\t14 of 440 (1e-376) Ets1-distal(ETS)/CD4+-PolII-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t15 of 440 (1e-361) EHF(ETS)/LoVo-EHF-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t16 of 440 (1e-358) ETS(ETS)/Promoter/Homer\n",
      "\t\t17 of 440 (1e-345) BATF(bZIP)/Th17-BATF-ChIP-Seq(GSE39756)/Homer\n",
      "\t\t18 of 440 (1e-340) JunB(bZIP)/DendriticCells-Junb-ChIP-Seq(GSE36099)/Homer\n",
      "\t\t19 of 440 (1e-340) ELF5(ETS)/T47D-ELF5-ChIP-Seq(GSE30407)/Homer\n",
      "\t\t20 of 440 (1e-337) ELF1(ETS)/Jurkat-ELF1-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t21 of 440 (1e-333) Fos(bZIP)/TSC-Fos-ChIP-Seq(GSE110950)/Homer\n",
      "\t\t22 of 440 (1e-327) Jun-AP1(bZIP)/K562-cJun-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t23 of 440 (1e-327) Fra2(bZIP)/Striatum-Fra2-ChIP-Seq(GSE43429)/Homer\n",
      "\t\t24 of 440 (1e-326) Elk1(ETS)/Hela-Elk1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t25 of 440 (1e-323) Fosl2(bZIP)/3T3L1-Fosl2-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t26 of 440 (1e-321) Fra1(bZIP)/BT549-Fra1-ChIP-Seq(GSE46166)/Homer\n",
      "\t\t27 of 440 (1e-318) IRF3(IRF)/BMDM-Irf3-ChIP-Seq(GSE67343)/Homer\n",
      "\t\t28 of 440 (1e-317) Atf3(bZIP)/GBM-ATF3-ChIP-Seq(GSE33912)/Homer\n",
      "\t\t29 of 440 (1e-317) IRF2(IRF)/Erythroblas-IRF2-ChIP-Seq(GSE36985)/Homer\n",
      "\t\t30 of 440 (1e-309) AP-1(bZIP)/ThioMac-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t31 of 440 (1e-305) IRF1(IRF)/PBMC-IRF1-ChIP-Seq(GSE43036)/Homer\n",
      "\t\t32 of 440 (1e-304) Elk4(ETS)/Hela-Elk4-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t33 of 440 (1e-296) EWS:FLI1-fusion(ETS)/SK_N_MC-EWS:FLI1-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t34 of 440 (1e-294) ELF3(ETS)/PDAC-ELF3-ChIP-Seq(GSE64557)/Homer\n",
      "\t\t35 of 440 (1e-286) KLF1(Zf)/HUDEP2-KLF1-CutnRun(GSE136251)/Homer\n",
      "\t\t36 of 440 (1e-285) PU.1:IRF8(ETS:IRF)/pDC-Irf8-ChIP-Seq(GSE66899)/Homer\n",
      "\t\t37 of 440 (1e-285) SpiB(ETS)/OCILY3-SPIB-ChIP-Seq(GSE56857)/Homer\n",
      "\t\t38 of 440 (1e-272) Sp5(Zf)/mES-Sp5.Flag-ChIP-Seq(GSE72989)/Homer\n",
      "\t\t39 of 440 (1e-267) KLF3(Zf)/MEF-Klf3-ChIP-Seq(GSE44748)/Homer\n",
      "\t\t40 of 440 (1e-255) EWS:ERG-fusion(ETS)/CADO_ES1-EWS:ERG-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t41 of 440 (1e-244) ISRE(IRF)/ThioMac-LPS-Expression(GSE23622)/Homer\n",
      "\t\t42 of 440 (1e-223) RUNX1(Runt)/Jurkat-RUNX1-ChIP-Seq(GSE29180)/Homer\n",
      "\t\t43 of 440 (1e-223) RUNX-AML(Runt)/CD4+-PolII-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t44 of 440 (1e-212) RUNX(Runt)/HPC7-Runx1-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t45 of 440 (1e-211) KLF5(Zf)/LoVo-KLF5-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t46 of 440 (1e-211) Sp2(Zf)/HEK293-Sp2.eGFP-ChIP-Seq(Encode)/Homer\n",
      "\t\t47 of 440 (1e-195) RUNX2(Runt)/PCa-RUNX2-ChIP-Seq(GSE33889)/Homer\n",
      "\t\t48 of 440 (1e-192) KLF6(Zf)/PDAC-KLF6-ChIP-Seq(GSE64557)/Homer\n",
      "\t\t49 of 440 (1e-187) ETS:RUNX(ETS,Runt)/Jurkat-RUNX1-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t50 of 440 (1e-183) PU.1-IRF(ETS:IRF)/Bcell-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t51 of 440 (1e-175) Klf9(Zf)/GBM-Klf9-ChIP-Seq(GSE62211)/Homer\n",
      "\t\t52 of 440 (1e-168) SPDEF(ETS)/VCaP-SPDEF-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t53 of 440 (1e-166) Bach2(bZIP)/OCILy7-Bach2-ChIP-Seq(GSE44420)/Homer\n",
      "\t\t54 of 440 (1e-155) KLF14(Zf)/HEK293-KLF14.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t55 of 440 (1e-155) IRF4(IRF)/GM12878-IRF4-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t56 of 440 (1e-140) NFY(CCAAT)/Promoter/Homer\n",
      "\t\t57 of 440 (1e-109) Klf4(Zf)/mES-Klf4-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t58 of 440 (1e-97) bZIP:IRF(bZIP,IRF)/Th17-BatF-ChIP-Seq(GSE39756)/Homer\n",
      "\t\t59 of 440 (1e-71) Ronin(THAP)/ES-Thap11-ChIP-Seq(GSE51522)/Homer\n",
      "\t\t60 of 440 (1e-68) NF-E2(bZIP)/K562-NFE2-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t61 of 440 (1e-60) Atf1(bZIP)/K562-ATF1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t62 of 440 (1e-59) Usf2(bHLH)/C2C12-Usf2-ChIP-Seq(GSE36030)/Homer\n",
      "\t\t63 of 440 (1e-58) CRE(bZIP)/Promoter/Homer\n",
      "\t\t64 of 440 (1e-58) NFIL3(bZIP)/HepG2-NFIL3-ChIP-Seq(Encode)/Homer\n",
      "\t\t65 of 440 (1e-52) Bach1(bZIP)/K562-Bach1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t66 of 440 (1e-51) GFY-Staf(?,Zf)/Promoter/Homer\n",
      "\t\t67 of 440 (1e-50) PRDM1(Zf)/Hela-PRDM1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t68 of 440 (1e-49) MafK(bZIP)/C2C12-MafK-ChIP-Seq(GSE36030)/Homer\n",
      "\t\t69 of 440 (1e-47) Maz(Zf)/HepG2-Maz-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t70 of 440 (1e-46) Oct11(POU,Homeobox)/NCIH1048-POU2F3-ChIP-seq(GSE115123)/Homer\n",
      "\t\t71 of 440 (1e-46) Atf7(bZIP)/3T3L1-Atf7-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t72 of 440 (1e-46) TFE3(bHLH)/MEF-TFE3-ChIP-Seq(GSE75757)/Homer\n",
      "\t\t73 of 440 (1e-46) Bcl11a(Zf)/HSPC-BCL11A-ChIP-Seq(GSE104676)/Homer\n",
      "\t\t74 of 440 (1e-45) KLF10(Zf)/HEK293-KLF10.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t75 of 440 (1e-44) NeuroD1(bHLH)/Islet-NeuroD1-ChIP-Seq(GSE30298)/Homer\n",
      "\t\t76 of 440 (1e-44) Nrf2(bZIP)/Lymphoblast-Nrf2-ChIP-Seq(GSE37589)/Homer\n",
      "\t\t77 of 440 (1e-41) CTCF-SatelliteElement(Zf?)/CD4+-CTCF-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t78 of 440 (1e-40) Oct2(POU,Homeobox)/Bcell-Oct2-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t79 of 440 (1e-39) MITF(bHLH)/MastCells-MITF-ChIP-Seq(GSE48085)/Homer\n",
      "\t\t80 of 440 (1e-38) HLF(bZIP)/HSC-HLF.Flag-ChIP-Seq(GSE69817)/Homer\n",
      "\t\t81 of 440 (1e-38) Brn1(POU,Homeobox)/NPC-Brn1-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t82 of 440 (1e-37) GFY(?)/Promoter/Homer\n",
      "\t\t83 of 440 (1e-36) Egr2(Zf)/Thymocytes-Egr2-ChIP-Seq(GSE34254)/Homer\n",
      "\t\t84 of 440 (1e-36) IRF:BATF(IRF:bZIP)/pDC-Irf8-ChIP-Seq(GSE66899)/Homer\n",
      "\t\t85 of 440 (1e-36) Atf2(bZIP)/3T3L1-Atf2-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t86 of 440 (1e-36) Oct4(POU,Homeobox)/mES-Oct4-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t87 of 440 (1e-35) c-Jun-CRE(bZIP)/K562-cJun-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t88 of 440 (1e-35) RFX(HTH)/K562-RFX3-ChIP-Seq(SRA012198)/Homer\n",
      "\t\t89 of 440 (1e-35) Unknown-ESC-element(?)/mES-Nanog-ChIP-Seq(GSE11724)/Homer\n",
      "\t\t90 of 440 (1e-34) CREB5(bZIP)/LNCaP-CREB5.V5-ChIP-Seq(GSE137775)/Homer\n",
      "\t\t91 of 440 (1e-34) E-box(bHLH)/Promoter/Homer\n",
      "\t\t92 of 440 (1e-34) NRF1(NRF)/MCF7-NRF1-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t93 of 440 (1e-34) USF1(bHLH)/GM12878-Usf1-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t94 of 440 (1e-33) Rfx2(HTH)/LoVo-RFX2-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t95 of 440 (1e-33) n-Myc(bHLH)/mES-nMyc-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t96 of 440 (1e-31) NFE2L2(bZIP)/HepG2-NFE2L2-ChIP-Seq(Encode)/Homer\n",
      "\t\t97 of 440 (1e-31) NFkB-p65-Rel(RHD)/ThioMac-LPS-Expression(GSE23622)/Homer\n",
      "\t\t98 of 440 (1e-30) Rfx5(HTH)/GM12878-Rfx5-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t99 of 440 (1e-30) Tcf12(bHLH)/GM12878-Tcf12-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t100 of 440 (1e-28) CLOCK(bHLH)/Liver-Clock-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t101 of 440 (1e-28) NFkB-p65(RHD)/GM12787-p65-ChIP-Seq(GSE19485)/Homer\n",
      "\t\t102 of 440 (1e-28) EKLF(Zf)/Erythrocyte-Klf1-ChIP-Seq(GSE20478)/Homer\n",
      "\t\t103 of 440 (1e-26) BMAL1(bHLH)/Liver-Bmal1-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t104 of 440 (1e-26) Max(bHLH)/K562-Max-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t105 of 440 (1e-25) Ptf1a(bHLH)/Panc1-Ptf1a-ChIP-Seq(GSE47459)/Homer\n",
      "\t\t106 of 440 (1e-25) THRb(NR)/HepG2-THRb.Flag-ChIP-Seq(Encode)/Homer\n",
      "\t\t107 of 440 (1e-25) Oct6(POU,Homeobox)/NPC-Pou3f1-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t108 of 440 (1e-25) JunD(bZIP)/K562-JunD-ChIP-Seq/Homer\n",
      "\t\t109 of 440 (1e-23) MafA(bZIP)/Islet-MafA-ChIP-Seq(GSE30298)/Homer\n",
      "\t\t110 of 440 (1e-23) Ascl1(bHLH)/NeuralTubes-Ascl1-ChIP-Seq(GSE55840)/Homer\n",
      "\t\t111 of 440 (1e-23) NRF(NRF)/Promoter/Homer\n",
      "\t\t112 of 440 (1e-23) SCL(bHLH)/HPC7-Scl-ChIP-Seq(GSE13511)/Homer\n",
      "\t\t113 of 440 (1e-22) Tgif2(Homeobox)/mES-Tgif2-ChIP-Seq(GSE55404)/Homer\n",
      "\t\t114 of 440 (1e-22) Zic3(Zf)/mES-Zic3-ChIP-Seq(GSE37889)/Homer\n",
      "\t\t115 of 440 (1e-21) GLIS3(Zf)/Thyroid-Glis3.GFP-ChIP-Seq(GSE103297)/Homer\n",
      "\t\t116 of 440 (1e-20) HEB(bHLH)/mES-Heb-ChIP-Seq(GSE53233)/Homer\n",
      "\t\t117 of 440 (1e-20) c-Myc(bHLH)/LNCAP-cMyc-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t118 of 440 (1e-20) Egr1(Zf)/K562-Egr1-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t119 of 440 (1e-19) YY1(Zf)/Promoter/Homer\n",
      "\t\t120 of 440 (1e-19) MNT(bHLH)/HepG2-MNT-ChIP-Seq(Encode)/Homer\n",
      "\t\t121 of 440 (1e-19) GFX(?)/Promoter/Homer\n",
      "\t\t122 of 440 (1e-19) Foxo1(Forkhead)/RAW-Foxo1-ChIP-Seq(Fan_et_al.)/Homer\n",
      "\t\t123 of 440 (1e-18) BHLHA15(bHLH)/NIH3T3-BHLHB8.HA-ChIP-Seq(GSE119782)/Homer\n",
      "\t\t124 of 440 (1e-18) CEBP(bZIP)/ThioMac-CEBPb-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t125 of 440 (1e-18) MafB(bZIP)/BMM-Mafb-ChIP-Seq(GSE75722)/Homer\n",
      "\t\t126 of 440 (1e-18) Ascl2(bHLH)/ESC-Ascl2-ChIP-Seq(GSE97712)/Homer\n",
      "\t\t127 of 440 (1e-18) TCF4(bHLH)/SHSY5Y-TCF4-ChIP-Seq(GSE96915)/Homer\n",
      "\t\t128 of 440 (1e-18) Atf4(bZIP)/MEF-Atf4-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t129 of 440 (1e-17) bHLHE40(bHLH)/HepG2-BHLHE40-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t130 of 440 (1e-17) CEBP:AP1(bZIP)/ThioMac-CEBPb-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t131 of 440 (1e-16) NPAS(bHLH)/Liver-NPAS-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t132 of 440 (1e-16) Tgif1(Homeobox)/mES-Tgif1-ChIP-Seq(GSE55404)/Homer\n",
      "\t\t133 of 440 (1e-16) Tcf7(HMG)/GM12878-TCF7-ChIP-Seq(Encode)/Homer\n",
      "\t\t134 of 440 (1e-15) Twist2(bHLH)/Myoblast-Twist2.Ty1-ChIP-Seq(GSE127998)/Homer\n",
      "\t\t135 of 440 (1e-15) Chop(bZIP)/MEF-Chop-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t136 of 440 (1e-14) Smad3(MAD)/NPC-Smad3-ChIP-Seq(GSE36673)/Homer\n",
      "\t\t137 of 440 (1e-14) c-Myc(bHLH)/mES-cMyc-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t138 of 440 (1e-14) En1(Homeobox)/SUM149-EN1-ChIP-Seq(GSE120957)/Homer\n",
      "\t\t139 of 440 (1e-13) Mef2d(MADS)/Retina-Mef2d-ChIP-Seq(GSE61391)/Homer\n",
      "\t\t140 of 440 (1e-13) X-box(HTH)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer\n",
      "\t\t141 of 440 (1e-13) T1ISRE(IRF)/ThioMac-Ifnb-Expression/Homer\n",
      "\t\t142 of 440 (1e-12) bHLHE41(bHLH)/proB-Bhlhe41-ChIP-Seq(GSE93764)/Homer\n",
      "\t\t143 of 440 (1e-12) Zic2(Zf)/ESC-Zic2-ChIP-Seq(SRP197560)/Homer\n",
      "\t\t144 of 440 (1e-12) E2A(bHLH)/proBcell-E2A-ChIP-Seq(GSE21978)/Homer\n",
      "\t\t145 of 440 (1e-12) Atoh1(bHLH)/Cerebellum-Atoh1-ChIP-Seq(GSE22111)/Homer\n",
      "\t\t146 of 440 (1e-12) NPAS2(bHLH)/Liver-NPAS2-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t147 of 440 (1e-12) WT1(Zf)/Kidney-WT1-ChIP-Seq(GSE90016)/Homer\n",
      "\t\t148 of 440 (1e-11) Ap4(bHLH)/AML-Tfap4-ChIP-Seq(GSE45738)/Homer\n",
      "\t\t149 of 440 (1e-11) Tcf21(bHLH)/ArterySmoothMuscle-Tcf21-ChIP-Seq(GSE61369)/Homer\n",
      "\t\t150 of 440 (1e-11) Slug(Zf)/Mesoderm-Snai2-ChIP-Seq(GSE61475)/Homer\n",
      "\t\t151 of 440 (1e-11) LEF1(HMG)/H1-LEF1-ChIP-Seq(GSE64758)/Homer\n",
      "\t\t152 of 440 (1e-11) Myf5(bHLH)/GM-Myf5-ChIP-Seq(GSE24852)/Homer\n",
      "\t\t153 of 440 (1e-10) ZBTB33(Zf)/GM12878-ZBTB33-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t154 of 440 (1e-10) Tbx5(T-box)/HL1-Tbx5.biotin-ChIP-Seq(GSE21529)/Homer\n",
      "\t\t155 of 440 (1e-10) NeuroG2(bHLH)/Fibroblast-NeuroG2-ChIP-Seq(GSE75910)/Homer\n",
      "\t\t156 of 440 (1e-10) NFkB-p50,p52(RHD)/Monocyte-p50-ChIP-Chip(Schreiber_et_al.)/Homer\n",
      "\t\t157 of 440 (1e-10) Mef2a(MADS)/HL1-Mef2a.biotin-ChIP-Seq(GSE21529)/Homer\n",
      "\t\t158 of 440 (1e-10) Erra(NR)/HepG2-Erra-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t159 of 440 (1e-9) Olig2(bHLH)/Neuron-Olig2-ChIP-Seq(GSE30882)/Homer\n",
      "\t\t160 of 440 (1e-9) Rfx1(HTH)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer\n",
      "\t\t161 of 440 (1e-9) Snail1(Zf)/LS174T-SNAIL1.HA-ChIP-Seq(GSE127183)/Homer\n",
      "\t\t162 of 440 (1e-9) ETS:E-box(ETS,bHLH)/HPC7-Scl-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t163 of 440 (1e-9) PBX2(Homeobox)/K562-PBX2-ChIP-Seq(Encode)/Homer\n",
      "\t\t164 of 440 (1e-8) MyoG(bHLH)/C2C12-MyoG-ChIP-Seq(GSE36024)/Homer\n",
      "\t\t165 of 440 (1e-8) Tbet(T-box)/CD8-Tbet-ChIP-Seq(GSE33802)/Homer\n",
      "\t\t166 of 440 (1e-8) Tbx20(T-box)/Heart-Tbx20-ChIP-Seq(GSE29636)/Homer\n",
      "\t\t167 of 440 (1e-8) Mef2c(MADS)/GM12878-Mef2c-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t168 of 440 (1e-8) Tcf3(HMG)/mES-Tcf3-ChIP-Seq(GSE11724)/Homer\n",
      "\t\t169 of 440 (1e-7) Hoxa10(Homeobox)/ChickenMSG-Hoxa10.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t170 of 440 (1e-7) Fox:Ebox(Forkhead,bHLH)/Panc1-Foxa2-ChIP-Seq(GSE47459)/Homer\n",
      "\t\t171 of 440 (1e-7) Meis1(Homeobox)/MastCells-Meis1-ChIP-Seq(GSE48085)/Homer\n",
      "\t\t172 of 440 (1e-7) Pbx3(Homeobox)/GM12878-PBX3-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t173 of 440 (1e-7) ZNF467(Zf)/HEK293-ZNF467.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t174 of 440 (1e-6) NFAT:AP1(RHD,bZIP)/Jurkat-NFATC1-ChIP-Seq(Jolma_et_al.)/Homer\n",
      "\t\t175 of 440 (1e-6) ZNF341(Zf)/EBV-ZNF341-ChIP-Seq(GSE113194)/Homer\n",
      "\t\t176 of 440 (1e-6) HIC1(Zf)/Treg-ZBTB29-ChIP-Seq(GSE99889)/Homer\n",
      "\t\t177 of 440 (1e-6) THRa(NR)/C17.2-THRa-ChIP-Seq(GSE38347)/Homer\n",
      "\t\t178 of 440 (1e-6) FOXA1(Forkhead)/MCF7-FOXA1-ChIP-Seq(GSE26831)/Homer\n",
      "\t\t179 of 440 (1e-5) PBX1(Homeobox)/MCF7-PBX1-ChIP-Seq(GSE28007)/Homer\n",
      "\t\t180 of 440 (1e-5) Tbx21(T-box)/GM12878-TBX21-ChIP-Seq(Encode)/Homer\n",
      "\t\t181 of 440 (1e-5) Gfi1b(Zf)/HPC7-Gfi1b-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t182 of 440 (1e-5) Hand2(bHLH)/Mesoderm-Hand2-ChIP-Seq(GSE61475)/Homer\n",
      "\t\t183 of 440 (1e-5) FOXA1(Forkhead)/LNCAP-FOXA1-ChIP-Seq(GSE27824)/Homer\n",
      "\t\t184 of 440 (1e-5) Mef2b(MADS)/HEK293-Mef2b.V5-ChIP-Seq(GSE67450)/Homer\n",
      "\t\t185 of 440 (1e-5) Foxa2(Forkhead)/Liver-Foxa2-ChIP-Seq(GSE25694)/Homer\n",
      "\t\t186 of 440 (1e-5) Stat3+il21(Stat)/CD4-Stat3-ChIP-Seq(GSE19198)/Homer\n",
      "\t\t187 of 440 (1e-5) E2F4(E2F)/K562-E2F4-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t188 of 440 (1e-5) Rfx6(HTH)/Min6b1-Rfx6.HA-ChIP-Seq(GSE62844)/Homer\n",
      "\t\t189 of 440 (1e-4) Pknox1(Homeobox)/ES-Prep1-ChIP-Seq(GSE63282)/Homer\n",
      "\t\t190 of 440 (1e-4) E2F3(E2F)/MEF-E2F3-ChIP-Seq(GSE71376)/Homer\n",
      "\t\t191 of 440 (1e-4) Zic(Zf)/Cerebellum-ZIC1.2-ChIP-Seq(GSE60731)/Homer\n",
      "\t\t192 of 440 (1e-4) ZBTB18(Zf)/HEK293-ZBTB18.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t193 of 440 (1e-4) FOXM1(Forkhead)/MCF7-FOXM1-ChIP-Seq(GSE72977)/Homer\n",
      "\t\t194 of 440 (1e-4) COUP-TFII(NR)/Artia-Nr2f2-ChIP-Seq(GSE46497)/Homer\n",
      "\t\t195 of 440 (1e-4) ZNF143|STAF(Zf)/CUTLL-ZNF143-ChIP-Seq(GSE29600)/Homer\n",
      "\t\t196 of 440 (1e-4) Nanog(Homeobox)/mES-Nanog-ChIP-Seq(GSE11724)/Homer\n",
      "\t\t197 of 440 (1e-4) Sox6(HMG)/Myotubes-Sox6-ChIP-Seq(GSE32627)/Homer\n",
      "\t\t198 of 440 (1e-4) Sox2(HMG)/mES-Sox2-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t199 of 440 (1e-3) TCFL2(HMG)/K562-TCF7L2-ChIP-Seq(GSE29196)/Homer\n",
      "\t\t200 of 440 (1e-3) Tbx6(T-box)/ESC-Tbx6-ChIP-Seq(GSE93524)/Homer\n",
      "\t\t201 of 440 (1e-3) REST-NRSF(Zf)/Jurkat-NRSF-ChIP-Seq/Homer\n",
      "\t\t202 of 440 (1e-3) PRDM9(Zf)/Testis-DMC1-ChIP-Seq(GSE35498)/Homer\n",
      "\t\t203 of 440 (1e-3) HOXA2(Homeobox)/mES-Hoxa2-ChIP-Seq(Donaldson_et_al.)/Homer\n",
      "\t\t204 of 440 (1e-3) Foxa3(Forkhead)/Liver-Foxa3-ChIP-Seq(GSE77670)/Homer\n",
      "\t\t205 of 440 (1e-3) Eomes(T-box)/H9-Eomes-ChIP-Seq(GSE26097)/Homer\n",
      "\t\t206 of 440 (1e-3) MyoD(bHLH)/Myotube-MyoD-ChIP-Seq(GSE21614)/Homer\n",
      "\t\t207 of 440 (1e-3) ZFX(Zf)/mES-Zfx-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t208 of 440 (1e-3) Smad2(MAD)/ES-SMAD2-ChIP-Seq(GSE29422)/Homer\n",
      "\t\t209 of 440 (1e-3) LHX9(Homeobox)/Hct116-LHX9.V5-ChIP-Seq(GSE116822)/Homer\n",
      "\t\t210 of 440 (1e-3) E2F6(E2F)/Hela-E2F6-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t211 of 440 (1e-3) Brn2(POU,Homeobox)/NPC-Brn2-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t212 of 440 (1e-3) CArG(MADS)/PUER-Srf-ChIP-Seq(Sullivan_et_al.)/Homer\n",
      "\t\t213 of 440 (1e-3) GATA3(Zf),DR4/iTreg-Gata3-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t214 of 440 (1e-3) PAX5(Paired,Homeobox),condensed/GM12878-PAX5-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t215 of 440 (1e-3) NF1(CTF)/LNCAP-NF1-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t216 of 440 (1e-3) OCT:OCT(POU,Homeobox,IR1)/NPC-Brn2-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t217 of 440 (1e-3) ZNF519(Zf)/HEK293-ZNF519.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t218 of 440 (1e-2) STAT4(Stat)/CD4-Stat4-ChIP-Seq(GSE22104)/Homer\n",
      "\t\t219 of 440 (1e-2) Pdx1(Homeobox)/Islet-Pdx1-ChIP-Seq(SRA008281)/Homer\n",
      "\t\t220 of 440 (1e-2) ZNF7(Zf)/HepG2-ZNF7.Flag-ChIP-Seq(Encode)/Homer\n",
      "\t\t221 of 440 (1e-2) PSE(SNAPc)/K562-mStart-Seq/Homer\n",
      "\t\t222 of 440 (1e-2) Hoxd10(Homeobox)/ChickenMSG-Hoxd10.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t223 of 440 (1e-2) Sox21(HMG)/ESC-SOX21-ChIP-Seq(GSE110505)/Homer\n",
      "\t\t224 of 440 (1e-2) Brachyury(T-box)/Mesoendoderm-Brachyury-ChIP-exo(GSE54963)/Homer\n",
      "\t\t225 of 440 (1e-2) AMYB(HTH)/Testes-AMYB-ChIP-Seq(GSE44588)/Homer\n",
      "\t\t226 of 440 (1e-2) GATA(Zf),IR3/iTreg-Gata3-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t227 of 440 (1e-2) Stat3(Stat)/mES-Stat3-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t228 of 440 (1e-2) Smad4(MAD)/ESC-SMAD4-ChIP-Seq(GSE29422)/Homer\n",
      "\t\t229 of 440 (1e-2) ZNF528(Zf)/HEK293-ZNF528.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t230 of 440 (1e-2) Tbr1(T-box)/Cortex-Tbr1-ChIP-Seq(GSE71384)/Homer\n",
      "\t\t231 of 440 (1e-2) Tlx?(NR)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer\n",
      "\t\t232 of 440 (1e-2) Foxo3(Forkhead)/U2OS-Foxo3-ChIP-Seq(E-MTAB-2701)/Homer\n",
      "\t\t233 of 440 (1e-2) Duxbl(Homeobox)/NIH3T3-Duxbl.HA-ChIP-Seq(GSE119782)/Homer\n",
      "\t\t234 of 440 (1e-2) Pit1(Homeobox)/GCrat-Pit1-ChIP-Seq(GSE58009)/Homer\n",
      "\t\t235 of 440 (1e-2) ZNF711(Zf)/SHSY5Y-ZNF711-ChIP-Seq(GSE20673)/Homer\n",
      "\t\t236 of 440 (1e-2) Six2(Homeobox)/NephronProgenitor-Six2-ChIP-Seq(GSE39837)/Homer\n",
      "\t\t237 of 440 (1e-2) Zfp281(Zf)/ES-Zfp281-ChIP-Seq(GSE81042)/Homer\n",
      "\t\t238 of 440 (1e-2) MYB(HTH)/ERMYB-Myb-ChIPSeq(GSE22095)/Homer\n",
      "\t----------------------------------------------------------\n",
      "\tDe novo motif finding (HOMER)\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 12 (from 10) to work well with 12 CPUs\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 32896\n",
      "\tAutoadjustment for sequence coverage in background: 0.99x\n",
      "\n",
      "\tOligos: 32896 of 34497 max\n",
      "\tTree  : 66996 of 172485 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 2 mismatches...\n",
      "\n",
      "\tScreening oligos 32896 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t0.00% skipped, 100.00% checked (32896 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32896 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t0.00% skipped, 100.00% checked (32896 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 53.29% increased p-value, 51.25% high p-value\n",
      "\n",
      "\tScreening oligos 32896 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t67.29% skipped, 32.71% checked (10761 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 19.26% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: GGCGGGGC... (-856.395)\n",
      "\t\tRound 1: -1086.31 GGCGGGGC T:11321.0(9.66%),B:6859.4(6.07%),P:1e-471\n",
      "\t\tRound 2: -1086.31 GGCGGGGC T:11321.0(9.66%),B:6859.4(6.07%),P:1e-471\n",
      "\t\t=Final=: -560.19 GGCGGGGC T:7430.0(6.67%),B:4947.2(4.46%),P:1e-243\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: TGACTCAT... (-646.926)\n",
      "\t\tRound 1: -1552.90 TGACTCAT T:11782.0(10.03%),B:6508.8(5.76%),P:1e-674\n",
      "\t\tRound 2: -1552.90 TGACTCAT T:11782.0(10.03%),B:6508.8(5.76%),P:1e-674\n",
      "\t\t=Final=: -740.03 TGACTCAT T:4982.0(4.47%),B:2756.1(2.48%),P:1e-321\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: AGGAAGTG... (-552.558)\n",
      "\t\tRound 1: -1215.39 AGGAAGTG T:15002.0(12.59%),B:9442.4(8.25%),P:1e-527\n",
      "\t\tRound 2: -1215.39 AGGAAGTG T:15002.0(12.59%),B:9442.4(8.25%),P:1e-527\n",
      "\t\t=Final=: -1155.81 AGGAAGTG T:13572.0(12.18%),B:8878.7(8.00%),P:1e-501\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: TGTGGTTA... (-352.523)\n",
      "\t\tRound 1: -578.25 TGTGGTTW T:25150.0(20.20%),B:19554.9(16.34%),P:1e-251\n",
      "\t\tRound 2: -578.25 TGTGGTTW T:25150.0(20.20%),B:19554.9(16.34%),P:1e-251\n",
      "\t\t=Final=: -528.95 TGTGGTTW T:19758.0(17.73%),B:15784.4(14.23%),P:1e-229\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: AGGGGGCG... (-325.860)\n",
      "\t\tRound 1: -1035.81 AGGGGGCG T:14218.0(11.98%),B:9180.0(8.03%),P:1e-449\n",
      "\t\tRound 2: -1035.81 AGGGGGCG T:14218.0(11.98%),B:9180.0(8.03%),P:1e-449\n",
      "\t\t=Final=: -1154.14 AGGGGGCG T:13195.0(11.84%),B:8573.2(7.73%),P:1e-501\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: GCCAATCG... (-216.656)\n",
      "\t\tRound 1: -481.60 GCCAATCG T:10553.0(9.03%),B:7501.3(6.61%),P:1e-209\n",
      "\t\tRound 2: -481.60 GCCAATCG T:10553.0(9.03%),B:7501.3(6.61%),P:1e-209\n",
      "\t\t=Final=: -316.58 GCCAATCG T:4283.0(3.84%),B:2853.1(2.57%),P:1e-137\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: AAGTGAAA... (-183.558)\n",
      "\t\tRound 1: -197.58 AASTGAAA T:10427.0(8.93%),B:8363.3(7.35%),P:1e-85\n",
      "\t\tRound 2: -211.68 AMGYGAAA T:7735.0(6.71%),B:5951.3(5.28%),P:1e-91\n",
      "\t\tRound 3: -258.81 AACCGAAA T:9819.0(8.43%),B:7579.6(6.68%),P:1e-112\n",
      "\t\tRound 4: -258.81 AACCGAAA T:9819.0(8.43%),B:7579.6(6.68%),P:1e-112\n",
      "\t\t=Final=: -334.31 AACCGAAA T:10539.0(9.46%),B:8168.8(7.36%),P:1e-145\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: CTTCCTGT... (-148.381)\n",
      "\t\tRound 1: -308.88 CTTCCGST T:11662.0(9.94%),B:8984.3(7.87%),P:1e-134\n",
      "\t\tRound 2: -308.88 CTTCCGST T:11662.0(9.94%),B:8984.3(7.87%),P:1e-134\n",
      "\t\t=Final=: -308.34 CTTCCGST T:10751.0(9.65%),B:8443.4(7.61%),P:1e-133\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: CACGTGAC... (-132.659)\n",
      "\t\tRound 1: -237.42 SACGTCAC T:6576.0(5.73%),B:4872.4(4.35%),P:1e-103\n",
      "\t\tRound 2: -263.53 YACGTSAY T:8876.0(7.66%),B:6749.1(5.97%),P:1e-114\n",
      "\t\tRound 3: -263.53 YACGTSAY T:8876.0(7.66%),B:6749.1(5.97%),P:1e-114\n",
      "\t\t=Final=: -188.76 YACGTSAY T:5342.0(4.79%),B:4061.3(3.66%),P:1e-81\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: ATTTGCAT... (-98.949)\n",
      "\t\tRound 1: -98.95 ATTTGCAT T:1347.0(1.20%),B:885.4(0.80%),P:1e-42\n",
      "\t\tRound 2: -98.95 ATTTGCAT T:1347.0(1.20%),B:885.4(0.80%),P:1e-42\n",
      "\t\t=Final=: -104.16 ATTTGCAT T:1319.0(1.18%),B:866.3(0.78%),P:1e-45\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: GGGGCGGG... (-97.748)\n",
      "\t\tRound 1: -307.20 GGGGCGGG T:15531.0(13.01%),B:12359.2(10.66%),P:1e-133\n",
      "\t\tRound 2: -307.20 GGGGCGGG T:15531.0(13.01%),B:12359.2(10.66%),P:1e-133\n",
      "\t\t=Final=: -180.67 GGGGCGGG T:9082.0(8.15%),B:7427.5(6.70%),P:1e-78\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TCTCGCGA... (-76.045)\n",
      "\t\tRound 1: -78.78 TCTCGCGA T:432.0(0.39%),B:221.1(0.20%),P:1e-34\n",
      "\t\tRound 2: -78.78 TCTCGCGA T:432.0(0.39%),B:221.1(0.20%),P:1e-34\n",
      "\t\t=Final=: -47.35 TCTCGCGA T:255.0(0.23%),B:132.6(0.12%),P:1e-20\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: GCATGCGC... (-73.339)\n",
      "\t\tRound 1: -73.34 GCATGCGC T:1061.0(0.95%),B:708.2(0.64%),P:1e-31\n",
      "\t\tRound 2: -73.34 GCATGCGC T:1061.0(0.95%),B:708.2(0.64%),P:1e-31\n",
      "\t\t=Final=: -51.50 GCATGCGC T:766.0(0.69%),B:524.0(0.47%),P:1e-22\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: CGACCAAT... (-66.712)\n",
      "\t\tRound 1: -115.85 CGACCAAT T:3318.0(2.93%),B:2481.2(2.24%),P:1e-50\n",
      "\t\tRound 2: -115.85 CGACCAAT T:3318.0(2.93%),B:2481.2(2.24%),P:1e-50\n",
      "\t\t=Final=: -106.87 CGACCAAT T:3173.0(2.85%),B:2427.1(2.19%),P:1e-46\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: CCACTAGA... (-58.936)\n",
      "\t\tRound 1: -106.56 CCACTAGR T:1388.0(1.24%),B:903.8(0.82%),P:1e-46\n",
      "\t\tRound 2: -116.92 CCGCTAGG T:1826.0(1.63%),B:1233.9(1.12%),P:1e-50\n",
      "\t\tRound 3: -116.92 CCGCTAGG T:1826.0(1.63%),B:1233.9(1.12%),P:1e-50\n",
      "\t\t=Final=: -124.05 CCGCTAGG T:1808.0(1.62%),B:1223.5(1.10%),P:1e-53\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: ATGACTAA... (-58.481)\n",
      "\t\tRound 1: -147.94 ATGACTCA T:9902.0(8.50%),B:8136.9(7.15%),P:1e-64\n",
      "\t\tRound 2: -147.94 ATGACTCA T:9902.0(8.50%),B:8136.9(7.15%),P:1e-64\n",
      "\t\t=Final=: -105.14 ATGACTCA T:6432.0(5.77%),B:5356.1(4.83%),P:1e-45\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: TTTCCTGT... (-57.261)\n",
      "\t\tRound 1: -80.91 BTTCCTGT T:8280.0(7.16%),B:7060.2(6.24%),P:1e-35\n",
      "\t\tRound 2: -97.43 GTTCCKGC T:24563.0(19.78%),B:21992.5(18.18%),P:1e-42\n",
      "\t\tRound 3: -105.77 BTTCCGKY T:11646.0(9.92%),B:9966.3(8.69%),P:1e-45\n",
      "\t\tRound 4: -105.77 BTTCCGKY T:11646.0(9.92%),B:9966.3(8.69%),P:1e-45\n",
      "\t\t=Final=: -108.41 BTTCCGKY T:10865.0(9.75%),B:9442.7(8.51%),P:1e-47\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: AGTTCCTC... (-44.620)\n",
      "\t\tRound 1: -59.82 AGTTCCTC T:4244.0(3.74%),B:3524.0(3.16%),P:1e-25\n",
      "\t\tRound 2: -80.93 RSTTCCYY T:10237.0(8.78%),B:8852.1(7.76%),P:1e-35\n",
      "\t\tRound 3: -94.75 ACTTCCTC T:5704.0(4.99%),B:4654.2(4.16%),P:1e-41\n",
      "\t\tRound 4: -94.75 ACTTCCTC T:5704.0(4.99%),B:4654.2(4.16%),P:1e-41\n",
      "\t\t=Final=: -107.39 ACTTCCTC T:5512.0(4.95%),B:4509.2(4.07%),P:1e-46\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: TTTTTTTT... (-39.389)\n",
      "\t\tRound 1: -39.39 TTTTTTTT T:8863.0(7.64%),B:7936.4(6.98%),P:1e-17\n",
      "\t\tRound 2: -39.39 TTTTTTTT T:8863.0(7.64%),B:7936.4(6.98%),P:1e-17\n",
      "\t\t=Final=: -6.56 TTTTTTTT T:2924.0(2.62%),B:2754.8(2.48%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: CTAGCAAC... (-35.316)\n",
      "\t\tRound 1: -50.18 CTAGCAAC T:890.0(0.80%),B:620.2(0.56%),P:1e-21\n",
      "\t\tRound 2: -50.18 CTAGCAAC T:890.0(0.80%),B:620.2(0.56%),P:1e-21\n",
      "\t\t=Final=: -49.62 CTAGCAAC T:872.0(0.78%),B:615.8(0.56%),P:1e-21\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: TGACGTCA... (-35.009)\n",
      "\t\tRound 1: -59.70 TGACGTCA T:1857.0(1.65%),B:1411.9(1.28%),P:1e-25\n",
      "\t\tRound 2: -83.46 TTRCRTMA T:8135.0(7.04%),B:6912.7(6.11%),P:1e-36\n",
      "\t\tRound 3: -83.46 TTRCRTMA T:8135.0(7.04%),B:6912.7(6.11%),P:1e-36\n",
      "\t\t=Final=: -90.78 TTRCRTMA T:7734.0(6.94%),B:6631.4(5.98%),P:1e-39\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: TATGACAC... (-33.466)\n",
      "\t\tRound 1: -55.64 TATGACAC T:12540.0(10.64%),B:11210.1(9.72%),P:1e-24\n",
      "\t\tRound 2: -55.64 TATGACAC T:12540.0(10.64%),B:11210.1(9.72%),P:1e-24\n",
      "\t\t=Final=: -72.15 TATGACAC T:21316.0(19.13%),B:19710.8(17.77%),P:1e-31\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: CTGATTGG... (-30.532)\n",
      "\t\tRound 1: -45.87 CTGATTGG T:1428.0(1.27%),B:1087.9(0.99%),P:1e-19\n",
      "\t\tRound 2: -45.87 CTGATTGG T:1428.0(1.27%),B:1087.9(0.99%),P:1e-19\n",
      "\t\t=Final=: -48.14 CTGATTGG T:1382.0(1.24%),B:1055.8(0.95%),P:1e-20\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: GTGACTAA... (-28.329)\n",
      "\t\tRound 1: -35.02 GTGACTAA T:904.0(0.81%),B:673.7(0.61%),P:1e-15\n",
      "\t\tRound 2: -35.02 GTGACTAA T:904.0(0.81%),B:673.7(0.61%),P:1e-15\n",
      "\t\t=Final=: -37.70 GTGACTAA T:893.0(0.80%),B:665.0(0.60%),P:1e-16\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: GGGAGGAG... (-26.772)\n",
      "\t\tRound 1: -60.05 GGGAGGAG T:17228.0(14.32%),B:15554.0(13.23%),P:1e-26\n",
      "\t\tRound 2: -60.05 GGGAGGAG T:17228.0(14.32%),B:15554.0(13.23%),P:1e-26\n",
      "\t\t=Final=: -59.33 GGGAGGAG T:14945.0(13.41%),B:13702.2(12.35%),P:1e-25\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: SRR7475870_motif_macs2/homerMotifs.motifs8\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 12 (from 10) to work well with 12 CPUs\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 524154\n",
      "\tAutoadjustment for sequence coverage in background: 0.99x\n",
      "\n",
      "\tOligos: 524154 of 551887 max\n",
      "\tTree  : 1070912 of 2759435 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 2 mismatches...\n",
      "\n",
      "\tScreening oligos 524154 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t0.61% skipped, 99.39% checked (520944 of 524154), of those checked:\n",
      "\t\t0.61% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 524154 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t0.61% skipped, 99.39% checked (520944 of 524154), of those checked:\n",
      "\t\t0.00% not in target, 52.78% increased p-value, 48.90% high p-value\n",
      "\n",
      "\tScreening oligos 524154 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t72.60% skipped, 27.40% checked (143615 of 524154), of those checked:\n",
      "\t\t0.00% not in target, 12.55% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: CCCCCTAGTG... (-931.938)\n",
      "\t\tRound 1: -2888.33 CCMYCTRGTG T:6636.0(5.78%),B:2125.9(1.92%),P:1e-1254\n",
      "\t\tRound 2: -2888.33 CCMYCTRGTG T:6636.0(5.78%),B:2125.9(1.92%),P:1e-1254\n",
      "\t\t=Final=: -2986.25 CCMYCTRGTG T:6424.0(5.76%),B:2073.0(1.87%),P:1e-1296\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: GGGCGGGGCC... (-910.366)\n",
      "\t\tRound 1: -1716.37 GGGCGGGGCC T:20733.0(16.98%),B:12891.7(11.10%),P:1e-745\n",
      "\t\tRound 2: -1757.49 GGGCGKRGCY T:17402.0(14.46%),B:10315.4(8.98%),P:1e-763\n",
      "\t\tRound 3: -1760.38 GGGCGKRRCC T:15066.0(12.64%),B:8593.8(7.54%),P:1e-764\n",
      "\t\tRound 4: -1760.38 GGGCGKRRCC T:15066.0(12.64%),B:8593.8(7.54%),P:1e-764\n",
      "\t\t=Final=: -873.75 GGGCGKRRCC T:8439.0(7.57%),B:5218.7(4.71%),P:1e-379\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: ACTTCCGGTT... (-593.063)\n",
      "\t\tRound 1: -1265.71 ACTTCCGGTT T:20683.0(16.94%),B:13774.1(11.81%),P:1e-549\n",
      "\t\tRound 2: -1485.61 ACTTCCKGTT T:17094.0(14.22%),B:10545.2(9.17%),P:1e-645\n",
      "\t\tRound 3: -1613.68 ACTTCCKGTT T:17920.0(14.85%),B:10944.4(9.50%),P:1e-700\n",
      "\t\tRound 4: -1688.63 ACTTCCGGTH T:20746.0(16.99%),B:12953.0(11.15%),P:1e-733\n",
      "\t\tRound 5: -1688.63 ACTTCCGGTH T:20746.0(16.99%),B:12953.0(11.15%),P:1e-733\n",
      "\t\t=Final=: -1546.88 ACTTCCGGTH T:17984.0(16.14%),B:11826.0(10.66%),P:1e-671\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: ATGAGTCATA... (-414.057)\n",
      "\t\tRound 1: -1451.01 ATGAGTCATA T:11163.0(9.53%),B:6197.2(5.50%),P:1e-630\n",
      "\t\tRound 2: -1655.26 VTGASTCATH T:11674.0(9.95%),B:6297.6(5.58%),P:1e-718\n",
      "\t\tRound 3: -1655.26 VTGASTCATH T:11674.0(9.95%),B:6297.6(5.58%),P:1e-718\n",
      "\t\t=Final=: -818.26 VTGASTCATH T:6221.0(5.58%),B:3583.1(3.23%),P:1e-355\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: GAAAGTGAAA... (-289.166)\n",
      "\t\tRound 1: -528.02 GAAAGTGAAA T:3393.0(3.00%),B:1799.4(1.63%),P:1e-229\n",
      "\t\tRound 2: -538.63 GAAAVTGAAA T:4423.0(3.89%),B:2531.4(2.28%),P:1e-233\n",
      "\t\tRound 3: -588.66 AGAASTGAAA T:3569.0(3.15%),B:1854.2(1.68%),P:1e-255\n",
      "\t\tRound 4: -588.66 AGAASTGAAA T:3569.0(3.15%),B:1854.2(1.68%),P:1e-255\n",
      "\t\t=Final=: -542.03 AGAASTGAAA T:3383.0(3.04%),B:1820.0(1.64%),P:1e-235\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: TAAACCACAG... (-193.180)\n",
      "\t\tRound 1: -410.70 TAAACCACAG T:7527.0(6.53%),B:5195.4(4.63%),P:1e-178\n",
      "\t\tRound 2: -535.04 NAAACCACAG T:17508.0(14.54%),B:13174.6(11.33%),P:1e-232\n",
      "\t\tRound 3: -559.51 NWAACCACAD T:20980.0(17.16%),B:16044.1(13.62%),P:1e-242\n",
      "\t\tRound 4: -560.29 NWAACCACAD T:20510.0(16.81%),B:15639.8(13.30%),P:1e-243\n",
      "\t\tRound 5: -569.39 NWAACCACAR T:10587.0(9.06%),B:7304.1(6.45%),P:1e-247\n",
      "\t\tRound 6: -569.39 NWAACCACAR T:10587.0(9.06%),B:7304.1(6.45%),P:1e-247\n",
      "\t\t=Final=: -546.52 NWAACCACAR T:9777.0(8.77%),B:6931.6(6.25%),P:1e-237\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: CGATTGGCCG... (-162.529)\n",
      "\t\tRound 1: -541.73 CGATTGGCCG T:5006.0(4.39%),B:2961.4(2.67%),P:1e-235\n",
      "\t\tRound 2: -632.43 CGATTGGCCG T:6820.0(5.94%),B:4192.0(3.75%),P:1e-274\n",
      "\t\tRound 3: -632.43 CGATTGGCCG T:6820.0(5.94%),B:4192.0(3.75%),P:1e-274\n",
      "\t\t=Final=: -408.29 CGATTGGCCG T:4087.0(3.67%),B:2534.4(2.29%),P:1e-177\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: TGACGTCACG... (-117.609)\n",
      "\t\tRound 1: -277.79 TGACGTCACG T:3714.0(3.28%),B:2414.9(2.18%),P:1e-120\n",
      "\t\tRound 2: -348.35 TGACGTCACN T:8228.0(7.12%),B:5934.5(5.27%),P:1e-151\n",
      "\t\tRound 3: -391.10 TGACGTCACS T:8472.0(7.32%),B:6021.2(5.35%),P:1e-169\n",
      "\t\tRound 4: -393.96 TGACGTCACS T:7512.0(6.52%),B:5223.1(4.65%),P:1e-171\n",
      "\t\tRound 5: -396.70 TSACGTCACS T:8646.0(7.46%),B:6150.2(5.46%),P:1e-172\n",
      "\t\tRound 6: -396.70 TSACGTCACS T:8646.0(7.46%),B:6150.2(5.46%),P:1e-172\n",
      "\t\t=Final=: -239.81 TSACGTCACS T:6474.0(5.81%),B:4892.6(4.41%),P:1e-104\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: TGCGCATGCG... (-113.688)\n",
      "\t\tRound 1: -188.96 TGCGCATGCG T:4741.0(4.16%),B:3466.8(3.11%),P:1e-82\n",
      "\t\tRound 2: -188.96 TGCGCATGCG T:4741.0(4.16%),B:3466.8(3.11%),P:1e-82\n",
      "\t\t=Final=: -83.17 TGCGCATGCG T:2189.0(1.96%),B:1646.7(1.48%),P:1e-36\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: CCACTAGGGG... (-86.660)\n",
      "\t\tRound 1: -209.90 CCACTAGGGG T:1657.0(1.48%),B:944.2(0.86%),P:1e-91\n",
      "\t\tRound 2: -209.90 CCACTAGGGG T:1657.0(1.48%),B:944.2(0.86%),P:1e-91\n",
      "\t\t=Final=: -218.03 CCACTAGGGG T:1638.0(1.47%),B:935.9(0.84%),P:1e-94\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: GCCCCGCCTA... (-79.940)\n",
      "\t\tRound 1: -162.76 GCCCCGCCTA T:7024.0(6.11%),B:5519.6(4.91%),P:1e-70\n",
      "\t\tRound 2: -240.39 GCCCCGCCYA T:8796.0(7.59%),B:6761.1(5.98%),P:1e-104\n",
      "\t\tRound 3: -282.14 GCCCCGCCCM T:14781.0(12.42%),B:11810.4(10.22%),P:1e-122\n",
      "\t\tRound 4: -287.14 GCYCCGCCYM T:13286.0(11.24%),B:10478.1(9.12%),P:1e-124\n",
      "\t\tRound 5: -311.74 GCCCCGCCCM T:17565.0(14.58%),B:14123.0(12.09%),P:1e-135\n",
      "\t\tRound 6: -318.15 GCCCCGCCCH T:16182.0(13.52%),B:12877.0(11.09%),P:1e-138\n",
      "\t\tRound 7: -339.09 GCCCCGCCCH T:10104.0(8.67%),B:7542.9(6.65%),P:1e-147\n",
      "\t\tRound 8: -347.24 GNHCCGCCCH T:15858.0(13.26%),B:12467.1(10.75%),P:1e-150\n",
      "\t\tRound 9: -348.21 GYYCCGCCCH T:25914.0(20.75%),B:21325.1(17.68%),P:1e-151\n",
      "\t\tRound 10: -348.21 GYYCCGCCCH T:25914.0(20.75%),B:21325.1(17.68%),P:1e-151\n",
      "\t\t=Final=: -210.66 GYYCCGCCCH T:20213.0(18.14%),B:17602.7(15.87%),P:1e-91\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: CTTCCGCTTT... (-70.081)\n",
      "\t\tRound 1: -123.86 CTTCCGCTTT T:3904.0(3.44%),B:2956.4(2.66%),P:1e-53\n",
      "\t\tRound 2: -174.38 CTTCCGCTTT T:5351.0(4.69%),B:4029.5(3.61%),P:1e-75\n",
      "\t\tRound 3: -209.55 STTYCGSTTT T:3923.0(3.46%),B:2730.2(2.46%),P:1e-91\n",
      "\t\tRound 4: -209.55 STTYCGSTTT T:3923.0(3.46%),B:2730.2(2.46%),P:1e-91\n",
      "\t\t=Final=: -194.38 STTYCGSTTT T:3738.0(3.35%),B:2668.4(2.41%),P:1e-84\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: CATTTGCATA... (-63.699)\n",
      "\t\tRound 1: -100.03 NATTTGCATN T:1031.0(0.92%),B:635.4(0.58%),P:1e-43\n",
      "\t\tRound 2: -107.37 CATTTGCATA T:1552.0(1.38%),B:1032.1(0.94%),P:1e-46\n",
      "\t\tRound 3: -107.37 CATTTGCATA T:1552.0(1.38%),B:1032.1(0.94%),P:1e-46\n",
      "\t\t=Final=: -111.96 CATTTGCATA T:1510.0(1.35%),B:1005.2(0.91%),P:1e-48\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GGGGATTTCC... (-59.752)\n",
      "\t\tRound 1: -99.11 GGGGATTTCC T:1636.0(1.46%),B:1118.4(1.02%),P:1e-43\n",
      "\t\tRound 2: -99.11 GGGGATTTCC T:1636.0(1.46%),B:1118.4(1.02%),P:1e-43\n",
      "\t\t=Final=: -92.18 GGGGATTTCC T:942.0(0.85%),B:588.5(0.53%),P:1e-40\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: ATAGGATGTG... (-53.905)\n",
      "\t\tRound 1: -90.65 ATAGGATGTG T:484.0(0.43%),B:244.7(0.22%),P:1e-39\n",
      "\t\tRound 2: -90.65 ATAGGATGTG T:484.0(0.43%),B:244.7(0.22%),P:1e-39\n",
      "\t\t=Final=: -92.17 ATAGGATGTG T:480.0(0.43%),B:243.8(0.22%),P:1e-40\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: AGCGCCACCT... (-52.064)\n",
      "\t\tRound 1: -135.26 AGCGCCACCT T:848.0(0.76%),B:451.0(0.41%),P:1e-58\n",
      "\t\tRound 2: -148.04 RGCGMCAYCT T:1247.0(1.11%),B:726.4(0.66%),P:1e-64\n",
      "\t\tRound 3: -148.04 RGCGMCAYCT T:1247.0(1.11%),B:726.4(0.66%),P:1e-64\n",
      "\t\t=Final=: -144.57 RGCGMCAYCT T:1210.0(1.09%),B:714.3(0.64%),P:1e-62\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: TACAACTCCC... (-49.550)\n",
      "\t\tRound 1: -69.31 TACAACTCCC T:399.0(0.36%),B:208.2(0.19%),P:1e-30\n",
      "\t\tRound 2: -73.92 TACATTTCCC T:568.0(0.51%),B:324.6(0.30%),P:1e-32\n",
      "\t\tRound 3: -84.78 TACATKTCCC T:662.0(0.59%),B:379.2(0.35%),P:1e-36\n",
      "\t\tRound 4: -94.96 TACAACTCCC T:722.0(0.65%),B:409.1(0.37%),P:1e-41\n",
      "\t\tRound 5: -94.96 TACAACTCCC T:722.0(0.65%),B:409.1(0.37%),P:1e-41\n",
      "\t\t=Final=: -81.02 TACAACTCCC T:676.0(0.61%),B:400.1(0.36%),P:1e-35\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: TCTCGCGAGA... (-48.505)\n",
      "\t\tRound 1: -63.04 TCTCGCGAGA T:303.0(0.27%),B:148.9(0.13%),P:1e-27\n",
      "\t\tRound 2: -63.04 TCTCGCGAGA T:303.0(0.27%),B:148.9(0.13%),P:1e-27\n",
      "\t\t=Final=: -64.65 TCTCGCGAGA T:240.0(0.22%),B:106.6(0.10%),P:1e-28\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: TCGACAGGTG... (-45.997)\n",
      "\t\tRound 1: -52.63 TCGACAGGTG T:30.0(0.03%),B:3.4(0.00%),P:1e-22\n",
      "\t\tRound 2: -52.63 TCGACAGGTG T:30.0(0.03%),B:3.4(0.00%),P:1e-22\n",
      "\t\t=Final=: -44.47 TCGACAGGTG T:30.0(0.03%),B:3.4(0.00%),P:1e-19\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: GGGTGTGACG... (-44.822)\n",
      "\t\tRound 1: -44.82 GGGTGTGACG T:35.0(0.03%),B:5.5(0.00%),P:1e-19\n",
      "\t\tRound 2: -44.82 GGGTGTGACG T:35.0(0.03%),B:5.5(0.00%),P:1e-19\n",
      "\t\t=Final=: -40.52 GGGTGTGACG T:35.0(0.03%),B:5.5(0.00%),P:1e-17\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: AGTTTCAATA... (-42.507)\n",
      "\t\tRound 1: -67.20 AGTTTCAATA T:766.0(0.68%),B:486.5(0.44%),P:1e-29\n",
      "\t\tRound 2: -93.05 AGTTTCAATA T:1584.0(1.41%),B:1089.1(0.99%),P:1e-40\n",
      "\t\tRound 3: -129.10 AGTTTCADTT T:1993.0(1.77%),B:1342.7(1.22%),P:1e-56\n",
      "\t\tRound 4: -146.92 RGTTTCARTH T:6003.0(5.24%),B:4690.8(4.19%),P:1e-63\n",
      "\t\tRound 5: -146.92 RGTTTCARTH T:6003.0(5.24%),B:4690.8(4.19%),P:1e-63\n",
      "\t\t=Final=: -144.08 RGTTTCARTH T:5729.0(5.14%),B:4555.2(4.11%),P:1e-62\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: GTGACTACTG... (-40.047)\n",
      "\t\tRound 1: -42.06 GTGACTACTG T:77.0(0.07%),B:23.9(0.02%),P:1e-18\n",
      "\t\tRound 2: -42.06 GTGACTACTG T:77.0(0.07%),B:23.9(0.02%),P:1e-18\n",
      "\t\t=Final=: -41.54 GTGACTACTG T:77.0(0.07%),B:23.9(0.02%),P:1e-18\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: CGTGTACATG... (-39.985)\n",
      "\t\tRound 1: -39.99 CGTGTACATG T:20.0(0.02%),B:3.0(0.00%),P:1e-17\n",
      "\t\tRound 2: -39.99 CGTGTACATG T:20.0(0.02%),B:3.0(0.00%),P:1e-17\n",
      "\t\t=Final=: -30.29 CGTGTACATG T:20.0(0.02%),B:3.0(0.00%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: TATGACACAT... (-39.957)\n",
      "\t\tRound 1: -90.25 TATGACACAT T:3110.0(2.75%),B:2387.8(2.15%),P:1e-39\n",
      "\t\tRound 2: -90.25 TATGACACAT T:3110.0(2.75%),B:2387.8(2.15%),P:1e-39\n",
      "\t\t=Final=: -99.69 TATGACACAT T:3041.0(2.73%),B:2335.2(2.11%),P:1e-43\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: CCAATGAGCG... (-38.455)\n",
      "\t\tRound 1: -80.05 CCAATGAGCG T:844.0(0.75%),B:524.8(0.48%),P:1e-34\n",
      "\t\tRound 2: -82.63 CCAATGAGCG T:578.0(0.52%),B:320.4(0.29%),P:1e-35\n",
      "\t\tRound 3: -89.18 CCAATGAGCG T:698.0(0.62%),B:399.8(0.36%),P:1e-38\n",
      "\t\tRound 4: -93.63 CCAATSAGCG T:1297.0(1.16%),B:856.2(0.78%),P:1e-40\n",
      "\t\tRound 5: -93.63 CCAATSAGCG T:1297.0(1.16%),B:856.2(0.78%),P:1e-40\n",
      "\t\t=Final=: -99.67 CCAATSAGCG T:1281.0(1.15%),B:844.6(0.76%),P:1e-43\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: SRR7475870_motif_macs2/homerMotifs.motifs10\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 12 (from 10) to work well with 12 CPUs\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 6578979\n",
      "\tAutoadjustment for sequence coverage in background: 0.99x\n",
      "\n",
      "\tOligos: 6578979 of 8830118 max\n",
      "\tTree  : 13985028 of 44150590 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 2 mismatches...\n",
      "\n",
      "\tScreening oligos 6578979 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t16.61% skipped, 83.39% checked (5485923 of 6578979), of those checked:\n",
      "\t\t16.61% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 6578979 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t16.61% skipped, 83.39% checked (5485923 of 6578979), of those checked:\n",
      "\t\t0.00% not in target, 38.62% increased p-value, 46.38% high p-value\n",
      "\n",
      "\tScreening oligos 6578979 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t74.18% skipped, 25.82% checked (1699009 of 6578979), of those checked:\n",
      "\t\t0.00% not in target, 16.73% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: GCCCCCTAGTGG... (-2173.645)\n",
      "\t\tRound 1: -5028.91 GCCCCCTAGTGG T:4479.0(3.94%),B:590.1(0.54%),P:1e-2184\n",
      "\t\tRound 2: -5028.91 GCCCCCTAGTGG T:4479.0(3.94%),B:590.1(0.54%),P:1e-2184\n",
      "\t\t=Final=: -5160.75 GCCCCCTAGTGG T:4561.0(4.09%),B:629.9(0.57%),P:1e-2241\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: GGGGGCGGGGCC... (-673.739)\n",
      "\t\tRound 1: -1574.72 GGGGGCGGGGCC T:11100.0(9.48%),B:5990.8(5.32%),P:1e-683\n",
      "\t\tRound 2: -1901.13 GGGGGCGGGGCC T:15763.0(13.19%),B:8894.1(7.80%),P:1e-825\n",
      "\t\tRound 3: -2003.09 GGGGGCGGGGCC T:14854.0(12.48%),B:8103.8(7.13%),P:1e-869\n",
      "\t\tRound 4: -2003.09 GGGGGCGGGGCC T:14854.0(12.48%),B:8103.8(7.13%),P:1e-869\n",
      "\t\t=Final=: -991.09 GGGGGCGGGGCC T:7147.0(6.41%),B:4057.5(3.66%),P:1e-430\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: CACTTCCTGTTT... (-548.711)\n",
      "\t\tRound 1: -1026.72 CACTTCCTGTTT T:13867.0(11.70%),B:8920.3(7.82%),P:1e-445\n",
      "\t\tRound 2: -1233.87 CACTTCCTGTTT T:15406.0(12.91%),B:9713.9(8.48%),P:1e-535\n",
      "\t\tRound 3: -1339.85 CACTTCCTSTTT T:16893.0(14.07%),B:10657.6(9.27%),P:1e-581\n",
      "\t\tRound 4: -1390.38 CACTTCCTSTTT T:13792.0(11.64%),B:8215.5(7.22%),P:1e-603\n",
      "\t\tRound 5: -1470.16 CACTTCCTSTTT T:14697.0(12.35%),B:8763.3(7.69%),P:1e-638\n",
      "\t\tRound 6: -1470.16 CACTTCCTSTTT T:14697.0(12.35%),B:8763.3(7.69%),P:1e-638\n",
      "\t\t=Final=: -1359.04 CACTTCCTSTTT T:13270.0(11.91%),B:8298.8(7.48%),P:1e-590\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: TGAAAGTGAAAC... (-218.499)\n",
      "\t\tRound 1: -400.58 TGAAAGTGAAAC T:1320.0(1.18%),B:532.3(0.48%),P:1e-173\n",
      "\t\tRound 2: -512.39 TGAAAGTGAAAC T:2311.0(2.05%),B:1074.5(0.98%),P:1e-222\n",
      "\t\tRound 3: -534.17 TGAAASTGAAAC T:2682.0(2.38%),B:1300.3(1.18%),P:1e-231\n",
      "\t\tRound 4: -555.78 TRAAASTGAAAC T:1600.0(1.43%),B:598.3(0.54%),P:1e-241\n",
      "\t\tRound 5: -587.52 HRRAAVTGAAAC T:1859.0(1.65%),B:731.2(0.67%),P:1e-255\n",
      "\t\tRound 6: -592.98 DRRAAVTGAAAC T:2459.0(2.18%),B:1101.3(1.00%),P:1e-257\n",
      "\t\tRound 7: -592.98 DRRAAVTGAAAC T:2459.0(2.18%),B:1101.3(1.00%),P:1e-257\n",
      "\t\t=Final=: -562.02 DRRAAVTGAAAC T:2030.0(1.82%),B:872.9(0.79%),P:1e-244\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: ATGAGTCATCAT... (-201.735)\n",
      "\t\tRound 1: -644.50 ATGAGTCATCAT T:6744.0(5.87%),B:4113.2(3.68%),P:1e-279\n",
      "\t\tRound 2: -1160.69 ATGAGTCATCAT T:7394.0(6.42%),B:3879.3(3.48%),P:1e-504\n",
      "\t\tRound 3: -1528.31 RTGASTCATYHY T:9642.0(8.29%),B:5022.7(4.48%),P:1e-663\n",
      "\t\tRound 4: -1528.31 RTGASTCATYHY T:9642.0(8.29%),B:5022.7(4.48%),P:1e-663\n",
      "\t\t=Final=: -771.28 RTGASTCATYHY T:4183.0(3.75%),B:2144.0(1.93%),P:1e-334\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: TCCGATTGGCCG... (-163.491)\n",
      "\t\tRound 1: -398.11 TCCGATTGGCCG T:2593.0(2.30%),B:1384.0(1.26%),P:1e-172\n",
      "\t\tRound 2: -477.53 TCYGATTGGCYG T:4428.0(3.90%),B:2624.2(2.37%),P:1e-207\n",
      "\t\tRound 3: -525.07 TCYSATTGGCCS T:3894.0(3.43%),B:2162.8(1.95%),P:1e-228\n",
      "\t\tRound 4: -560.61 YCYSATTGGCYG T:3128.0(2.77%),B:1579.0(1.43%),P:1e-243\n",
      "\t\tRound 5: -565.84 BCYSATTGGYCV T:3671.0(3.24%),B:1952.9(1.77%),P:1e-245\n",
      "\t\tRound 6: -565.84 BCYSATTGGYCV T:3671.0(3.24%),B:1952.9(1.77%),P:1e-245\n",
      "\t\t=Final=: -446.57 BCYSATTGGYCV T:2998.0(2.69%),B:1652.1(1.49%),P:1e-193\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: CGCCCTCTGGTG... (-141.909)\n",
      "\t\tRound 1: -352.93 CGCCCTCTGGTG T:1893.0(1.68%),B:945.8(0.86%),P:1e-153\n",
      "\t\tRound 2: -536.08 CGCCCYCTGSTG T:5126.0(4.50%),B:3059.3(2.75%),P:1e-232\n",
      "\t\tRound 3: -645.89 CGCCCYCTGSTG T:4618.0(4.06%),B:2530.4(2.28%),P:1e-280\n",
      "\t\tRound 4: -729.96 CGCCMYCTGSTG T:2893.0(2.56%),B:1267.5(1.15%),P:1e-317\n",
      "\t\tRound 5: -764.68 CGCCMYCTRSTG T:3708.0(3.27%),B:1769.9(1.60%),P:1e-332\n",
      "\t\tRound 6: -764.68 CGCCMYCTRSTG T:3708.0(3.27%),B:1769.9(1.60%),P:1e-332\n",
      "\t\t=Final=: -814.25 CGCCMYCTRSTG T:3664.0(3.29%),B:1742.7(1.57%),P:1e-353\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: GGTTGTGGTTTC... (-128.500)\n",
      "\t\tRound 1: -283.13 GGTTGTGGTTTC T:11489.0(9.80%),B:8928.9(7.83%),P:1e-122\n",
      "\t\tRound 2: -283.13 GGTTGTGGTTTC T:11489.0(9.80%),B:8928.9(7.83%),P:1e-122\n",
      "\t\t=Final=: -266.90 GGTTGTGGTTTC T:10528.0(9.45%),B:8391.6(7.57%),P:1e-115\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: GTGACGTCACGT... (-115.566)\n",
      "\t\tRound 1: -142.14 GTGACGTCACGT T:2873.0(2.54%),B:2031.1(1.84%),P:1e-61\n",
      "\t\tRound 2: -150.05 GTGACGTCACTK T:3607.0(3.18%),B:2623.9(2.37%),P:1e-65\n",
      "\t\tRound 3: -156.68 GTGACGTCABBC T:4857.0(4.26%),B:3663.5(3.29%),P:1e-68\n",
      "\t\tRound 4: -161.98 GTGACGTCABBB T:2434.0(2.16%),B:1627.2(1.47%),P:1e-70\n",
      "\t\tRound 5: -161.98 GTGACGTCABBB T:2434.0(2.16%),B:1627.2(1.47%),P:1e-70\n",
      "\t\t=Final=: -102.67 GTGACGTCABBB T:1858.0(1.67%),B:1312.8(1.18%),P:1e-44\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: GCGCATGCGCAT... (-106.415)\n",
      "\t\tRound 1: -167.17 GCGCATGCGCAT T:2300.0(2.04%),B:1509.9(1.37%),P:1e-72\n",
      "\t\tRound 2: -193.28 GCGCATGCGCRS T:3260.0(2.88%),B:2226.8(2.01%),P:1e-83\n",
      "\t\tRound 3: -193.28 GCGCATGCGCRS T:3260.0(2.88%),B:2226.8(2.01%),P:1e-83\n",
      "\t\t=Final=: -107.45 GCGCATGCGCRS T:1505.0(1.35%),B:1010.9(0.91%),P:1e-46\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: GTCACGTGACGT... (-97.980)\n",
      "\t\tRound 1: -183.21 GTCACGTGACGT T:1217.0(1.09%),B:658.0(0.60%),P:1e-79\n",
      "\t\tRound 2: -183.21 GTCACGTGACGT T:1217.0(1.09%),B:658.0(0.60%),P:1e-79\n",
      "\t\t=Final=: -97.55 GTCACGTGACGT T:785.0(0.70%),B:459.5(0.41%),P:1e-42\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: ACTACAATTCCC... (-83.284)\n",
      "\t\tRound 1: -155.17 ACTACAATTCCC T:1006.0(0.90%),B:540.9(0.49%),P:1e-67\n",
      "\t\tRound 2: -164.37 ACTACAATTCCC T:922.0(0.82%),B:471.9(0.43%),P:1e-71\n",
      "\t\tRound 3: -164.37 ACTACAATTCCC T:922.0(0.82%),B:471.9(0.43%),P:1e-71\n",
      "\t\t=Final=: -135.06 ACTACAATTCCC T:825.0(0.74%),B:441.1(0.40%),P:1e-58\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: TACCGGAAGTCG... (-70.089)\n",
      "\t\tRound 1: -148.80 NMCCGGAAGTCG T:1640.0(1.46%),B:1023.5(0.93%),P:1e-64\n",
      "\t\tRound 2: -271.99 NVCCGGAAGTSG T:3633.0(3.21%),B:2361.7(2.13%),P:1e-118\n",
      "\t\tRound 3: -343.32 NSCCGGAAGYGG T:6927.0(6.03%),B:4869.0(4.35%),P:1e-149\n",
      "\t\tRound 4: -367.61 NSCCGGAAGYGG T:4967.0(4.36%),B:3229.4(2.90%),P:1e-159\n",
      "\t\tRound 5: -367.61 NSCCGGAAGYGG T:4967.0(4.36%),B:3229.4(2.90%),P:1e-159\n",
      "\t\t=Final=: -310.65 NSCCGGAAGYGG T:4374.0(3.92%),B:2939.9(2.65%),P:1e-134\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: CACTTCTGCTTT... (-58.153)\n",
      "\t\tRound 1: -84.90 CACTTCTGCTTT T:92.0(0.08%),B:17.7(0.02%),P:1e-36\n",
      "\t\tRound 2: -86.55 CACTTCTGCTTT T:65.0(0.06%),B:8.9(0.01%),P:1e-37\n",
      "\t\tRound 3: -96.29 CACTTCAGCTTT T:104.0(0.09%),B:19.2(0.02%),P:1e-41\n",
      "\t\tRound 4: -108.62 CACTTCAGCTTT T:96.0(0.09%),B:14.3(0.01%),P:1e-47\n",
      "\t\tRound 5: -108.62 CACTTCAGCTTT T:96.0(0.09%),B:14.3(0.01%),P:1e-47\n",
      "\t\t=Final=: -105.51 CACTTCAGCTTT T:96.0(0.09%),B:14.3(0.01%),P:1e-45\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: CACTTCCTTCCG... (-55.019)\n",
      "\t\tRound 1: -73.24 CACTTCCTTCCG T:126.0(0.11%),B:35.3(0.03%),P:1e-31\n",
      "\t\tRound 2: -73.24 CACTTCCTTCCG T:126.0(0.11%),B:35.3(0.03%),P:1e-31\n",
      "\t\t=Final=: -73.01 CACTTCCTTCCG T:126.0(0.11%),B:35.3(0.03%),P:1e-31\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: CTTATTTGCATA... (-54.438)\n",
      "\t\tRound 1: -115.87 CTTATTTGCATA T:396.0(0.35%),B:165.1(0.15%),P:1e-50\n",
      "\t\tRound 2: -115.87 CTTATTTGCATA T:396.0(0.35%),B:165.1(0.15%),P:1e-50\n",
      "\t\t=Final=: -119.25 CTTATTTGCATA T:394.0(0.35%),B:163.9(0.15%),P:1e-51\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: GGGGAATTCCCC... (-51.943)\n",
      "\t\tRound 1: -89.62 GGGGAATTCCCC T:2138.0(1.90%),B:1558.5(1.41%),P:1e-38\n",
      "\t\tRound 2: -92.25 DGGGAWTTCCCC T:1017.0(0.91%),B:637.6(0.58%),P:1e-40\n",
      "\t\tRound 3: -96.93 DGGGAWTTCCCC T:1496.0(1.33%),B:1009.8(0.92%),P:1e-42\n",
      "\t\tRound 4: -101.92 DGGGRNTTYCCC T:1325.0(1.18%),B:862.2(0.78%),P:1e-44\n",
      "\t\tRound 5: -104.20 NGGGRNTTYCCC T:1517.0(1.35%),B:1010.4(0.92%),P:1e-45\n",
      "\t\tRound 6: -116.92 RGGGRNTTCCCC T:1050.0(0.94%),B:623.7(0.57%),P:1e-50\n",
      "\t\tRound 7: -116.92 RGGGRNTTCCCC T:1050.0(0.94%),B:623.7(0.57%),P:1e-50\n",
      "\t\t=Final=: -108.32 RGGGRNTTCCCC T:975.0(0.87%),B:588.1(0.53%),P:1e-47\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: CTGTGGTTAACC... (-50.622)\n",
      "\t\tRound 1: -102.53 YTGTGGTTAACN T:7950.0(6.88%),B:6626.3(5.87%),P:1e-44\n",
      "\t\tRound 2: -122.66 NTGTGGTTAACH T:1791.0(1.59%),B:1192.7(1.08%),P:1e-53\n",
      "\t\tRound 3: -132.78 YTGTGGTTAAAW T:4958.0(4.35%),B:3834.4(3.44%),P:1e-57\n",
      "\t\tRound 4: -143.87 NTGTGGTTAGHK T:9707.0(8.34%),B:7979.7(7.02%),P:1e-62\n",
      "\t\tRound 5: -159.28 BTGTGGTTAGAD T:7194.0(6.25%),B:5680.4(5.05%),P:1e-69\n",
      "\t\tRound 6: -159.28 BTGTGGTTAGAD T:7194.0(6.25%),B:5680.4(5.05%),P:1e-69\n",
      "\t\t=Final=: -171.61 BTGTGGTTAGAD T:6880.0(6.17%),B:5479.9(4.94%),P:1e-74\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: TCTCGCGAGATA... (-47.747)\n",
      "\t\tRound 1: -96.66 TCTCGCGAGATA T:506.0(0.45%),B:253.2(0.23%),P:1e-41\n",
      "\t\tRound 2: -96.87 TCTCGCGAGAKY T:528.0(0.47%),B:268.5(0.24%),P:1e-42\n",
      "\t\tRound 3: -96.87 TCTCGCGAGAKY T:528.0(0.47%),B:268.5(0.24%),P:1e-42\n",
      "\t\t=Final=: -51.70 TCTCGCGAGAKY T:222.0(0.20%),B:105.4(0.10%),P:1e-22\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: AGGGCGTGGCCT... (-46.699)\n",
      "\t\tRound 1: -67.17 AGGGCGTGGCCT T:4945.0(4.34%),B:4113.0(3.68%),P:1e-29\n",
      "\t\tRound 2: -75.95 RGGGCGKGGCCT T:1468.0(1.31%),B:1034.5(0.94%),P:1e-32\n",
      "\t\tRound 3: -77.10 RGGGCGTGGCCT T:1840.0(1.64%),B:1342.1(1.22%),P:1e-33\n",
      "\t\tRound 4: -77.10 RGGGCGTGGCCT T:1840.0(1.64%),B:1342.1(1.22%),P:1e-33\n",
      "\t\t=Final=: -77.83 RGGGCGTGGCCT T:4471.0(4.01%),B:3700.0(3.34%),P:1e-33\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: CCAGGCGGCGCC... (-46.459)\n",
      "\t\tRound 1: -69.95 CCAGGCGGCGCC T:46.0(0.04%),B:6.0(0.00%),P:1e-30\n",
      "\t\tRound 2: -69.95 CCAGGCGGCGCC T:46.0(0.04%),B:6.0(0.00%),P:1e-30\n",
      "\t\t=Final=: -63.62 CCAGGCGGCGCC T:46.0(0.04%),B:6.0(0.01%),P:1e-27\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: AGGTGGCGCTGT... (-45.841)\n",
      "\t\tRound 1: -87.54 AGGTGGCGCTGT T:478.0(0.43%),B:244.0(0.22%),P:1e-38\n",
      "\t\tRound 2: -87.54 AGGTGGCGCTGT T:478.0(0.43%),B:244.0(0.22%),P:1e-38\n",
      "\t\t=Final=: -84.14 AGGTGGCGCTGT T:468.0(0.42%),B:244.0(0.22%),P:1e-36\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: TACCATGGTAAC... (-44.348)\n",
      "\t\tRound 1: -67.94 TACCATGGTAAC T:764.0(0.68%),B:484.0(0.44%),P:1e-29\n",
      "\t\tRound 2: -72.60 YACCATGGYAAC T:407.0(0.36%),B:210.9(0.19%),P:1e-31\n",
      "\t\tRound 3: -80.61 TGCCATGGYAAC T:875.0(0.78%),B:547.3(0.50%),P:1e-35\n",
      "\t\tRound 4: -95.13 YGCCATGGCAAC T:882.0(0.79%),B:529.3(0.48%),P:1e-41\n",
      "\t\tRound 5: -95.13 YGCCATGGCAAC T:882.0(0.79%),B:529.3(0.48%),P:1e-41\n",
      "\t\t=Final=: -62.73 YGCCATGGCAAC T:709.0(0.64%),B:455.9(0.41%),P:1e-27\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: GGTGGTTCTGAG... (-42.962)\n",
      "\t\tRound 1: -49.07 GGTGGTTCTGAG T:23.0(0.02%),B:2.8(0.00%),P:1e-21\n",
      "\t\tRound 2: -49.07 GGTGGTTCTGAG T:23.0(0.02%),B:2.8(0.00%),P:1e-21\n",
      "\t\t=Final=: -30.29 GGTGGTTCTGAG T:20.0(0.02%),B:2.8(0.00%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: TAGATGCGTCAA... (-42.962)\n",
      "\t\tRound 1: -52.19 TAGATGCGTCAA T:24.0(0.02%),B:2.4(0.00%),P:1e-22\n",
      "\t\tRound 2: -58.55 TAGATGCGTCAA T:26.0(0.02%),B:2.5(0.00%),P:1e-25\n",
      "\t\tRound 3: -58.55 TAGATGCGTCAA T:26.0(0.02%),B:2.5(0.00%),P:1e-25\n",
      "\t\t=Final=: -46.24 TAGATGCGTCAA T:21.0(0.02%),B:1.5(0.00%),P:1e-20\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t222894 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: SRR7475870_motif_macs2/homerMotifs.motifs12\n",
      "\n",
      "\t(Motifs in homer2 format)\n",
      "\tDetermining similar motifs... 75 reduced to 22 motifs\n",
      "\tOutputing HTML and sequence logos for motif comparison...\n",
      "\tChecking de novo motifs against known motifs...\n",
      "\tFormatting HTML page...\n",
      "\t\t1 of 22 (1e-2241) similar to BORIS(Zf)/K562-CTCFL-ChIP-Seq(GSE32465)/Homer(0.922)\n",
      "\t\t2 of 22 (1e-687) similar to Fli1(ETS)/CD8-FLI-ChIP-Seq(GSE20898)/Homer(0.968)\n",
      "\t\t3 of 22 (1e-434) similar to Sp1(Zf)/Promoter/Homer(0.980)\n",
      "\t\t4 of 22 (1e-376) similar to BATF(bZIP)/Th17-BATF-ChIP-Seq(GSE39756)/Homer(0.995)\n",
      "\t\t5 of 22 (1e-370) similar to IRF8(IRF)/BMDM-IRF8-ChIP-Seq(GSE77884)/Homer(0.920)\n",
      "\t\t6 of 22 (1e-279) similar to RUNX(Runt)/HPC7-Runx1-ChIP-Seq(GSE22178)/Homer(0.977)\n",
      "\t\t7 of 22 (1e-271) similar to PB0117.1_Eomes_2/Jaspar(0.735)\n",
      "\t\t8 of 22 (1e-200) similar to NFY(CCAAT)/Promoter/Homer(0.971)\n",
      "\t\t9 of 22 (1e-132) similar to CREB3L4(var.2)/MA1475.1/Jaspar(0.953)\n",
      "\t\t10 of 22 (1e-64) similar to GFY(?)/Promoter/Homer(0.987)\n",
      "\t\t11 of 22 (1e-54) similar to PH0144.1_Pou2f2/Jaspar(0.919)\n",
      "\t\t12 of 22 (1e-52) similar to MF0003.1_REL_class/Jaspar(0.910)\n",
      "\t\t13 of 22 (1e-47) similar to NRF1(NRF)/MCF7-NRF1-ChIP-Seq(Unpublished)/Homer(0.981)\n",
      "\t\t14 of 22 (1e-42) similar to SpiB(ETS)/OCILY3-SPIB-ChIP-Seq(GSE56857)/Homer(0.707)\n",
      "\t\t15 of 22 (1e-28) similar to Rfx1(HTH)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer(0.929)\n",
      "\t\t16 of 22 (1e-28) similar to GFX(?)/Promoter/Homer(0.977)\n",
      "\t\t17 of 22 (1e-22) similar to PB0056.1_Rfxdc2_1/Jaspar(0.891)\n",
      "\t\t18 of 22 (1e-16) similar to SCRT1/MA0743.2/Jaspar(0.833)\n",
      "\t\t19 of 22 (1e-15) similar to PAX3:FKHR-fusion(Paired,Homeobox)/Rh4-PAX3:FKHR-ChIP-Seq(GSE19063)/Homer(0.661)\n",
      "\t\t20 of 22 (1e-13) similar to PB0044.1_Mtf1_1/Jaspar(0.645)\n",
      "\t\t21 of 22 (1e-10) similar to PB0196.1_Zbtb7b_2/Jaspar(0.620)\n",
      "\t\t22 of 22 (1e-3) similar to ZNF384/MA1125.1/Jaspar(0.960)\n",
      "\tJob finished - if results look good, please send beer to ..\n",
      "\n",
      "\tCleaning up tmp files...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! findMotifsGenome.pl \\\n",
    "    /home/inou_doi/demo/3_peak_call/MACS2/SRR7475870_summits.bed \\\n",
    "    hg19 \\\n",
    "    SRR7475870_motif_macs2  \\\n",
    "    -size 200 \\\n",
    "    -mask \\\n",
    "    -p 12"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "86d742c1-7ee4-4ede-986c-83a7eca2f4f7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr1\t755343\t755344\tSRR7475873_peak_1\t4.65694\n",
      "chr1\t756930\t756931\tSRR7475873_peak_2\t3.85791\n",
      "chr1\t762509\t762510\tSRR7475873_peak_3\t4.65694\n",
      "chr1\t763107\t763108\tSRR7475873_peak_4\t18.06672\n",
      "chr1\t776722\t776723\tSRR7475873_peak_5\t3.85791\n",
      "chr1\t805263\t805264\tSRR7475873_peak_6\t15.72547\n",
      "chr1\t847012\t847013\tSRR7475873_peak_7\t4.64537\n",
      "chr1\t847798\t847799\tSRR7475873_peak_8\t3.51921\n",
      "chr1\t871006\t871007\tSRR7475873_peak_9\t4.64322\n",
      "chr1\t894692\t894693\tSRR7475873_peak_10\t11.47504\n"
     ]
    }
   ],
   "source": [
    "! head /home/inou_doi/demo/3_peak_call/MACS2/SRR7475873_summits.bed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "f1d5cf07-fbdb-4a83-b5ec-2e69065668df",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = /home/inou_doi/demo/3_peak_call/MACS2/SRR7475873_summits.bed\n",
      "\tGenome = hg19\n",
      "\tOutput Directory = SRR7475873_motif_macs2\n",
      "\tFragment size set to 200\n",
      "\tWill use repeat masked sequences\n",
      "\tUsing 12 CPUs\n",
      "\tFound mset for \"human\", will check against vertebrates motifs\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 98168\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 98168\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground files for 200 bp fragments found.\n",
      "\tCustom genome sequence directory: /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//\n",
      "\n",
      "\tExtracting sequences from directory: /opt/conda/envs/homer/share/homer/.//data/genomes/hg19//\n",
      "\tExtracting 8801 sequences from chr1\n",
      "\tExtracting 8422 sequences from chr2\n",
      "\tExtracting 6489 sequences from chr3\n",
      "\tExtracting 5180 sequences from chr4\n",
      "\tExtracting 5225 sequences from chr5\n",
      "\tExtracting 6174 sequences from chr6\n",
      "\tExtracting 5636 sequences from chr7\n",
      "\tExtracting 4163 sequences from chr8\n",
      "\tExtracting 3740 sequences from chr9\n",
      "\tExtracting 5084 sequences from chr10\n",
      "\tExtracting 5158 sequences from chr11\n",
      "\tExtracting 5633 sequences from chr12\n",
      "\tExtracting 2969 sequences from chr13\n",
      "\tExtracting 2806 sequences from chr14\n",
      "\tExtracting 2337 sequences from chr15\n",
      "\tExtracting 2861 sequences from chr16\n",
      "\tExtracting 4102 sequences from chr17\n",
      "\tExtracting 2501 sequences from chr18\n",
      "\tExtracting 3864 sequences from chr19\n",
      "\tExtracting 1995 sequences from chr20\n",
      "\tExtracting 1516 sequences from chr21\n",
      "\tExtracting 1578 sequences from chr22\n",
      "\tExtracting 1449 sequences from chrX\n",
      "\tExtracting 74 sequences from chrY\n",
      "\tExtracting 4 sequences from chr11_gl000202_random\n",
      "\tExtracting 1 sequences from chr17_ctg5_hap1\n",
      "\tExtracting 1 sequences from chr17_gl000203_random\n",
      "\tExtracting 2 sequences from chr17_gl000204_random\n",
      "\tExtracting 26 sequences from chr17_gl000205_random\n",
      "\tExtracting 1 sequences from chr18_gl000207_random\n",
      "\tExtracting 7 sequences from chr19_gl000208_random\n",
      "\tExtracting 1 sequences from chr1_gl000191_random\n",
      "\tExtracting 6 sequences from chr1_gl000192_random\n",
      "\tExtracting 43 sequences from chr4_gl000193_random\n",
      "\tExtracting 19 sequences from chr4_gl000194_random\n",
      "\tExtracting 3 sequences from chr6_dbb_hap3\n",
      "\tExtracting 2 sequences from chr6_mann_hap4\n",
      "\tExtracting 2 sequences from chr6_mcf_hap5\n",
      "\tExtracting 4 sequences from chr6_ssto_hap7\n",
      "\tExtracting 28 sequences from chr7_gl000195_random\n",
      "\tExtracting 7 sequences from chr9_gl000198_random\n",
      "\tExtracting 13 sequences from chr9_gl000199_random\n",
      "\tExtracting 1 sequences from chrUn_gl000211\n",
      "\tExtracting 7 sequences from chrUn_gl000212\n",
      "\tExtracting 43 sequences from chrUn_gl000214\n",
      "\tExtracting 11 sequences from chrUn_gl000216\n",
      "\tExtracting 8 sequences from chrUn_gl000218\n",
      "\tExtracting 9 sequences from chrUn_gl000219\n",
      "\tExtracting 47 sequences from chrUn_gl000220\n",
      "\tExtracting 29 sequences from chrUn_gl000224\n",
      "\tExtracting 31 sequences from chrUn_gl000225\n",
      "\tExtracting 6 sequences from chrUn_gl000226\n",
      "\tExtracting 4 sequences from chrUn_gl000228\n",
      "\tExtracting 5 sequences from chrUn_gl000229\n",
      "\tExtracting 2 sequences from chrUn_gl000231\n",
      "\tExtracting 9 sequences from chrUn_gl000232\n",
      "\tExtracting 8 sequences from chrUn_gl000234\n",
      "\tExtracting 1 sequences from chrUn_gl000235\n",
      "\tExtracting 15 sequences from chrUn_gl000237\n",
      "\tExtracting 5 sequences from chrUn_gl000241\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\tPotential Problem with Sequences (ignore if preparsing genome):\n",
      "\t\t26002 of 98168 removed because they had >70.00% Ns (i.e. masked repeats)\n",
      "\tDue to \thigh repeat content, consider using a non-repeat masked genome\n",
      "\n",
      "\n",
      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n",
      "\n",
      "\tNot removing redundant sequences\n",
      "\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 201 bp\n",
      "\t\t72166 total\n",
      "\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t2\n",
      "\t0.25\t1\t22\n",
      "\t0.3\t2\t174\n",
      "\t0.35\t3\t1193\n",
      "\t0.4\t4\t3996\n",
      "\t0.45\t5\t6806\n",
      "\t0.5\t6\t8067\n",
      "\t0.6\t7\t20452\n",
      "\t0.7\t8\t21216\n",
      "\t0.8\t9\t9766\n",
      "\t10\t10\t472\n",
      "\n",
      "\tTotal sequences set to 144332\n",
      "\n",
      "\tChoosing background that matches in CpG/GC content...\n",
      "\tBin\t# Targets\t# Background\tBackground Weight\n",
      "\t0\t2\t2\t1.000\n",
      "\t1\t22\t22\t1.000\n",
      "\t2\t174\t174\t1.000\n",
      "\t3\t1193\t1193\t1.000\n",
      "\t4\t3996\t3996\t1.000\n",
      "\t5\t6806\t6806\t1.000\n",
      "\t6\t8067\t8067\t1.000\n",
      "\t7\t20452\t20452\t1.000\n",
      "\t8\t21216\t21216\t1.000\n",
      "\t9\t9766\t9766\t1.000\n",
      "\t10\t472\t472\t1.000\n",
      "\tAssembling sequence file...\n",
      "\tNormalizing lower order oligos using homer2\n",
      "\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t21.39%\t21.53%\t0.994\n",
      "\t\tC\t28.61%\t28.47%\t1.005\n",
      "\t\tG\t28.61%\t28.47%\t1.005\n",
      "\t\tT\t21.39%\t21.53%\t0.994\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t5.05%\t5.31%\t0.950\n",
      "\t\tCA\t7.30%\t7.05%\t1.036\n",
      "\t\tGA\t6.01%\t6.17%\t0.974\n",
      "\t\tTA\t3.01%\t2.98%\t1.011\n",
      "\t\tAC\t4.97%\t4.74%\t1.049\n",
      "\t\tCC\t9.37%\t9.67%\t0.968\n",
      "\t\tGC\t8.28%\t7.90%\t1.047\n",
      "\t\tTC\t6.01%\t6.17%\t0.974\n",
      "\t\tAG\t7.67%\t7.64%\t1.004\n",
      "\t\tCG\t4.29%\t4.13%\t1.038\n",
      "\t\tGG\t9.37%\t9.67%\t0.968\n",
      "\t\tTG\t7.30%\t7.05%\t1.036\n",
      "\t\tAT\t3.68%\t3.82%\t0.963\n",
      "\t\tCT\t7.67%\t7.64%\t1.004\n",
      "\t\tGT\t4.97%\t4.74%\t1.049\n",
      "\t\tTT\t5.05%\t5.31%\t0.950\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tNormalization weights can be found in file: SRR7475873_motif_macs2/seq.autonorm.tsv\n",
      "\tConverging on autonormalization solution:\n",
      "\t...............................................................................\n",
      "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t21.39%\t21.38%\t1.000\n",
      "\t\tC\t28.61%\t28.62%\t1.000\n",
      "\t\tG\t28.61%\t28.62%\t1.000\n",
      "\t\tT\t21.39%\t21.38%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t5.05%\t5.06%\t0.997\n",
      "\t\tCA\t7.30%\t7.28%\t1.003\n",
      "\t\tGA\t6.01%\t6.03%\t0.997\n",
      "\t\tTA\t3.01%\t3.00%\t1.005\n",
      "\t\tAC\t4.97%\t4.95%\t1.005\n",
      "\t\tCC\t9.37%\t9.41%\t0.996\n",
      "\t\tGC\t8.28%\t8.25%\t1.004\n",
      "\t\tTC\t6.01%\t6.03%\t0.997\n",
      "\t\tAG\t7.67%\t7.65%\t1.003\n",
      "\t\tCG\t4.29%\t4.28%\t1.000\n",
      "\t\tGG\t9.37%\t9.41%\t0.996\n",
      "\t\tTG\t7.30%\t7.28%\t1.003\n",
      "\t\tAT\t3.68%\t3.71%\t0.993\n",
      "\t\tCT\t7.67%\t7.65%\t1.003\n",
      "\t\tGT\t4.97%\t4.95%\t1.005\n",
      "\t\tTT\t5.05%\t5.06%\t0.997\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tFinished preparing sequence/group files\n",
      "\n",
      "\t----------------------------------------------------------\n",
      "\tKnown motif enrichment\n",
      "\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\t440 motifs loaded\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 440 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tPreparing HTML output with sequence logos...\n",
      "\t\t1 of 440 (1e-2782) CTCF(Zf)/CD4+-CTCF-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t2 of 440 (1e-1749) BORIS(Zf)/K562-CTCFL-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t3 of 440 (1e-1529) Fos(bZIP)/TSC-Fos-ChIP-Seq(GSE110950)/Homer\n",
      "\t\t4 of 440 (1e-1514) Atf3(bZIP)/GBM-ATF3-ChIP-Seq(GSE33912)/Homer\n",
      "\t\t5 of 440 (1e-1497) Fra1(bZIP)/BT549-Fra1-ChIP-Seq(GSE46166)/Homer\n",
      "\t\t6 of 440 (1e-1447) BATF(bZIP)/Th17-BATF-ChIP-Seq(GSE39756)/Homer\n",
      "\t\t7 of 440 (1e-1434) JunB(bZIP)/DendriticCells-Junb-ChIP-Seq(GSE36099)/Homer\n",
      "\t\t8 of 440 (1e-1426) Fra2(bZIP)/Striatum-Fra2-ChIP-Seq(GSE43429)/Homer\n",
      "\t\t9 of 440 (1e-1391) AP-1(bZIP)/ThioMac-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t10 of 440 (1e-1256) Fosl2(bZIP)/3T3L1-Fosl2-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t11 of 440 (1e-1105) Jun-AP1(bZIP)/K562-cJun-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t12 of 440 (1e-484) Bach2(bZIP)/OCILy7-Bach2-ChIP-Seq(GSE44420)/Homer\n",
      "\t\t13 of 440 (1e-442) Fli1(ETS)/CD8-FLI-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t14 of 440 (1e-385) ETS1(ETS)/Jurkat-ETS1-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t15 of 440 (1e-376) ETV4(ETS)/HepG2-ETV4-ChIP-Seq(ENCODE)/Homer\n",
      "\t\t16 of 440 (1e-358) Etv2(ETS)/ES-ER71-ChIP-Seq(GSE59402)/Homer\n",
      "\t\t17 of 440 (1e-352) ETV1(ETS)/GIST48-ETV1-ChIP-Seq(GSE22441)/Homer\n",
      "\t\t18 of 440 (1e-336) GABPA(ETS)/Jurkat-GABPa-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t19 of 440 (1e-329) ERG(ETS)/VCaP-ERG-ChIP-Seq(GSE14097)/Homer\n",
      "\t\t20 of 440 (1e-303) Sp5(Zf)/mES-Sp5.Flag-ChIP-Seq(GSE72989)/Homer\n",
      "\t\t21 of 440 (1e-291) Elk1(ETS)/Hela-Elk1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t22 of 440 (1e-277) EWS:FLI1-fusion(ETS)/SK_N_MC-EWS:FLI1-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t23 of 440 (1e-267) Elk4(ETS)/Hela-Elk4-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t24 of 440 (1e-266) Sp1(Zf)/Promoter/Homer\n",
      "\t\t25 of 440 (1e-251) Ets1-distal(ETS)/CD4+-PolII-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t26 of 440 (1e-249) KLF1(Zf)/HUDEP2-KLF1-CutnRun(GSE136251)/Homer\n",
      "\t\t27 of 440 (1e-246) IRF2(IRF)/Erythroblas-IRF2-ChIP-Seq(GSE36985)/Homer\n",
      "\t\t28 of 440 (1e-236) ETS(ETS)/Promoter/Homer\n",
      "\t\t29 of 440 (1e-233) ELF1(ETS)/Jurkat-ELF1-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t30 of 440 (1e-230) KLF14(Zf)/HEK293-KLF14.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t31 of 440 (1e-229) EWS:ERG-fusion(ETS)/CADO_ES1-EWS:ERG-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t32 of 440 (1e-228) Sp2(Zf)/HEK293-Sp2.eGFP-ChIP-Seq(Encode)/Homer\n",
      "\t\t33 of 440 (1e-222) Elf4(ETS)/BMDM-Elf4-ChIP-Seq(GSE88699)/Homer\n",
      "\t\t34 of 440 (1e-205) IRF1(IRF)/PBMC-IRF1-ChIP-Seq(GSE43036)/Homer\n",
      "\t\t35 of 440 (1e-199) ISRE(IRF)/ThioMac-LPS-Expression(GSE23622)/Homer\n",
      "\t\t36 of 440 (1e-198) KLF5(Zf)/LoVo-KLF5-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t37 of 440 (1e-192) KLF6(Zf)/PDAC-KLF6-ChIP-Seq(GSE64557)/Homer\n",
      "\t\t38 of 440 (1e-188) KLF3(Zf)/MEF-Klf3-ChIP-Seq(GSE44748)/Homer\n",
      "\t\t39 of 440 (1e-187) NFAT:AP1(RHD,bZIP)/Jurkat-NFATC1-ChIP-Seq(Jolma_et_al.)/Homer\n",
      "\t\t40 of 440 (1e-184) EHF(ETS)/LoVo-EHF-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t41 of 440 (1e-180) NF-E2(bZIP)/K562-NFE2-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t42 of 440 (1e-172) IRF3(IRF)/BMDM-Irf3-ChIP-Seq(GSE67343)/Homer\n",
      "\t\t43 of 440 (1e-168) IRF8(IRF)/BMDM-IRF8-ChIP-Seq(GSE77884)/Homer\n",
      "\t\t44 of 440 (1e-151) NFY(CCAAT)/Promoter/Homer\n",
      "\t\t45 of 440 (1e-151) Bach1(bZIP)/K562-Bach1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t46 of 440 (1e-147) RUNX1(Runt)/Jurkat-RUNX1-ChIP-Seq(GSE29180)/Homer\n",
      "\t\t47 of 440 (1e-146) MafK(bZIP)/C2C12-MafK-ChIP-Seq(GSE36030)/Homer\n",
      "\t\t48 of 440 (1e-143) Klf9(Zf)/GBM-Klf9-ChIP-Seq(GSE62211)/Homer\n",
      "\t\t49 of 440 (1e-139) NFE2L2(bZIP)/HepG2-NFE2L2-ChIP-Seq(Encode)/Homer\n",
      "\t\t50 of 440 (1e-139) RUNX-AML(Runt)/CD4+-PolII-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t51 of 440 (1e-136) Nrf2(bZIP)/Lymphoblast-Nrf2-ChIP-Seq(GSE37589)/Homer\n",
      "\t\t52 of 440 (1e-130) RUNX2(Runt)/PCa-RUNX2-ChIP-Seq(GSE33889)/Homer\n",
      "\t\t53 of 440 (1e-126) RUNX(Runt)/HPC7-Runx1-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t54 of 440 (1e-120) ETS:RUNX(ETS,Runt)/Jurkat-RUNX1-ChIP-Seq(GSE17954)/Homer\n",
      "\t\t55 of 440 (1e-118) ELF3(ETS)/PDAC-ELF3-ChIP-Seq(GSE64557)/Homer\n",
      "\t\t56 of 440 (1e-115) Tgif2(Homeobox)/mES-Tgif2-ChIP-Seq(GSE55404)/Homer\n",
      "\t\t57 of 440 (1e-111) PU.1(ETS)/ThioMac-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t58 of 440 (1e-104) ELF5(ETS)/T47D-ELF5-ChIP-Seq(GSE30407)/Homer\n",
      "\t\t59 of 440 (1e-102) Ronin(THAP)/ES-Thap11-ChIP-Seq(GSE51522)/Homer\n",
      "\t\t60 of 440 (1e-97) PU.1-IRF(ETS:IRF)/Bcell-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t61 of 440 (1e-96) bZIP:IRF(bZIP,IRF)/Th17-BatF-ChIP-Seq(GSE39756)/Homer\n",
      "\t\t62 of 440 (1e-93) SPDEF(ETS)/VCaP-SPDEF-ChIP-Seq(SRA014231)/Homer\n",
      "\t\t63 of 440 (1e-93) Maz(Zf)/HepG2-Maz-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t64 of 440 (1e-85) PU.1:IRF8(ETS:IRF)/pDC-Irf8-ChIP-Seq(GSE66899)/Homer\n",
      "\t\t65 of 440 (1e-78) Klf4(Zf)/mES-Klf4-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t66 of 440 (1e-78) IRF4(IRF)/GM12878-IRF4-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t67 of 440 (1e-77) GFY-Staf(?,Zf)/Promoter/Homer\n",
      "\t\t68 of 440 (1e-75) MafB(bZIP)/BMM-Mafb-ChIP-Seq(GSE75722)/Homer\n",
      "\t\t69 of 440 (1e-74) Atf1(bZIP)/K562-ATF1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t70 of 440 (1e-69) Atf7(bZIP)/3T3L1-Atf7-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t71 of 440 (1e-65) CTCF-SatelliteElement(Zf?)/CD4+-CTCF-ChIP-Seq(Barski_et_al.)/Homer\n",
      "\t\t72 of 440 (1e-62) CRE(bZIP)/Promoter/Homer\n",
      "\t\t73 of 440 (1e-61) IRF:BATF(IRF:bZIP)/pDC-Irf8-ChIP-Seq(GSE66899)/Homer\n",
      "\t\t74 of 440 (1e-59) GFY(?)/Promoter/Homer\n",
      "\t\t75 of 440 (1e-59) LEF1(HMG)/H1-LEF1-ChIP-Seq(GSE64758)/Homer\n",
      "\t\t76 of 440 (1e-57) Egr1(Zf)/K562-Egr1-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t77 of 440 (1e-57) Tgif1(Homeobox)/mES-Tgif1-ChIP-Seq(GSE55404)/Homer\n",
      "\t\t78 of 440 (1e-53) Tcf3(HMG)/mES-Tcf3-ChIP-Seq(GSE11724)/Homer\n",
      "\t\t79 of 440 (1e-53) WT1(Zf)/Kidney-WT1-ChIP-Seq(GSE90016)/Homer\n",
      "\t\t80 of 440 (1e-50) Atf2(bZIP)/3T3L1-Atf2-ChIP-Seq(GSE56872)/Homer\n",
      "\t\t81 of 440 (1e-48) Tbx5(T-box)/HL1-Tbx5.biotin-ChIP-Seq(GSE21529)/Homer\n",
      "\t\t82 of 440 (1e-47) Tcf7(HMG)/GM12878-TCF7-ChIP-Seq(Encode)/Homer\n",
      "\t\t83 of 440 (1e-47) c-Jun-CRE(bZIP)/K562-cJun-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t84 of 440 (1e-46) MafA(bZIP)/Islet-MafA-ChIP-Seq(GSE30298)/Homer\n",
      "\t\t85 of 440 (1e-45) KLF10(Zf)/HEK293-KLF10.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t86 of 440 (1e-44) MITF(bHLH)/MastCells-MITF-ChIP-Seq(GSE48085)/Homer\n",
      "\t\t87 of 440 (1e-43) SpiB(ETS)/OCILY3-SPIB-ChIP-Seq(GSE56857)/Homer\n",
      "\t\t88 of 440 (1e-42) CREB5(bZIP)/LNCaP-CREB5.V5-ChIP-Seq(GSE137775)/Homer\n",
      "\t\t89 of 440 (1e-42) JunD(bZIP)/K562-JunD-ChIP-Seq/Homer\n",
      "\t\t90 of 440 (1e-40) Unknown-ESC-element(?)/mES-Nanog-ChIP-Seq(GSE11724)/Homer\n",
      "\t\t91 of 440 (1e-39) Egr2(Zf)/Thymocytes-Egr2-ChIP-Seq(GSE34254)/Homer\n",
      "\t\t92 of 440 (1e-38) PRDM1(Zf)/Hela-PRDM1-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t93 of 440 (1e-36) NFAT(RHD)/Jurkat-NFATC1-ChIP-Seq(Jolma_et_al.)/Homer\n",
      "\t\t94 of 440 (1e-35) En1(Homeobox)/SUM149-EN1-ChIP-Seq(GSE120957)/Homer\n",
      "\t\t95 of 440 (1e-35) NFIL3(bZIP)/HepG2-NFIL3-ChIP-Seq(Encode)/Homer\n",
      "\t\t96 of 440 (1e-34) Tbet(T-box)/CD8-Tbet-ChIP-Seq(GSE33802)/Homer\n",
      "\t\t97 of 440 (1e-34) Pdx1(Homeobox)/Islet-Pdx1-ChIP-Seq(SRA008281)/Homer\n",
      "\t\t98 of 440 (1e-33) Meis1(Homeobox)/MastCells-Meis1-ChIP-Seq(GSE48085)/Homer\n",
      "\t\t99 of 440 (1e-33) Zic3(Zf)/mES-Zic3-ChIP-Seq(GSE37889)/Homer\n",
      "\t\t100 of 440 (1e-32) Smad3(MAD)/NPC-Smad3-ChIP-Seq(GSE36673)/Homer\n",
      "\t\t101 of 440 (1e-32) Rfx2(HTH)/LoVo-RFX2-ChIP-Seq(GSE49402)/Homer\n",
      "\t\t102 of 440 (1e-31) Usf2(bHLH)/C2C12-Usf2-ChIP-Seq(GSE36030)/Homer\n",
      "\t\t103 of 440 (1e-30) NFkB-p65-Rel(RHD)/ThioMac-LPS-Expression(GSE23622)/Homer\n",
      "\t\t104 of 440 (1e-27) PBX2(Homeobox)/K562-PBX2-ChIP-Seq(Encode)/Homer\n",
      "\t\t105 of 440 (1e-27) RFX(HTH)/K562-RFX3-ChIP-Seq(SRA012198)/Homer\n",
      "\t\t106 of 440 (1e-27) NFkB-p65(RHD)/GM12787-p65-ChIP-Seq(GSE19485)/Homer\n",
      "\t\t107 of 440 (1e-27) TCFL2(HMG)/K562-TCF7L2-ChIP-Seq(GSE29196)/Homer\n",
      "\t\t108 of 440 (1e-26) Rfx5(HTH)/GM12878-Rfx5-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t109 of 440 (1e-26) CLOCK(bHLH)/Liver-Clock-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t110 of 440 (1e-26) YY1(Zf)/Promoter/Homer\n",
      "\t\t111 of 440 (1e-25) USF1(bHLH)/GM12878-Usf1-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t112 of 440 (1e-25) Zic2(Zf)/ESC-Zic2-ChIP-Seq(SRP197560)/Homer\n",
      "\t\t113 of 440 (1e-24) NRF1(NRF)/MCF7-NRF1-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t114 of 440 (1e-23) BMAL1(bHLH)/Liver-Bmal1-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t115 of 440 (1e-22) Max(bHLH)/K562-Max-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t116 of 440 (1e-22) HIC1(Zf)/Treg-ZBTB29-ChIP-Seq(GSE99889)/Homer\n",
      "\t\t117 of 440 (1e-22) NFkB-p50,p52(RHD)/Monocyte-p50-ChIP-Chip(Schreiber_et_al.)/Homer\n",
      "\t\t118 of 440 (1e-21) GLIS3(Zf)/Thyroid-Glis3.GFP-ChIP-Seq(GSE103297)/Homer\n",
      "\t\t119 of 440 (1e-19) EKLF(Zf)/Erythrocyte-Klf1-ChIP-Seq(GSE20478)/Homer\n",
      "\t\t120 of 440 (1e-18) MNT(bHLH)/HepG2-MNT-ChIP-Seq(Encode)/Homer\n",
      "\t\t121 of 440 (1e-18) ZNF143|STAF(Zf)/CUTLL-ZNF143-ChIP-Seq(GSE29600)/Homer\n",
      "\t\t122 of 440 (1e-18) n-Myc(bHLH)/mES-nMyc-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t123 of 440 (1e-17) Ptf1a(bHLH)/Panc1-Ptf1a-ChIP-Seq(GSE47459)/Homer\n",
      "\t\t124 of 440 (1e-17) ZNF467(Zf)/HEK293-ZNF467.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t125 of 440 (1e-17) Atf4(bZIP)/MEF-Atf4-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t126 of 440 (1e-17) NRF(NRF)/Promoter/Homer\n",
      "\t\t127 of 440 (1e-16) SCL(bHLH)/HPC7-Scl-ChIP-Seq(GSE13511)/Homer\n",
      "\t\t128 of 440 (1e-16) NeuroD1(bHLH)/Islet-NeuroD1-ChIP-Seq(GSE30298)/Homer\n",
      "\t\t129 of 440 (1e-16) HLF(bZIP)/HSC-HLF.Flag-ChIP-Seq(GSE69817)/Homer\n",
      "\t\t130 of 440 (1e-15) GATA(Zf),IR3/iTreg-Gata3-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t131 of 440 (1e-15) Chop(bZIP)/MEF-Chop-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t132 of 440 (1e-15) NPAS(bHLH)/Liver-NPAS-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t133 of 440 (1e-15) TFE3(bHLH)/MEF-TFE3-ChIP-Seq(GSE75757)/Homer\n",
      "\t\t134 of 440 (1e-14) c-Myc(bHLH)/LNCAP-cMyc-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t135 of 440 (1e-14) Stat3+il21(Stat)/CD4-Stat3-ChIP-Seq(GSE19198)/Homer\n",
      "\t\t136 of 440 (1e-14) Nanog(Homeobox)/mES-Nanog-ChIP-Seq(GSE11724)/Homer\n",
      "\t\t137 of 440 (1e-14) Hoxb4(Homeobox)/ES-Hoxb4-ChIP-Seq(GSE34014)/Homer\n",
      "\t\t138 of 440 (1e-14) Brn1(POU,Homeobox)/NPC-Brn1-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t139 of 440 (1e-13) Ascl1(bHLH)/NeuralTubes-Ascl1-ChIP-Seq(GSE55840)/Homer\n",
      "\t\t140 of 440 (1e-13) Rfx1(HTH)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer\n",
      "\t\t141 of 440 (1e-13) THRb(NR)/HepG2-THRb.Flag-ChIP-Seq(Encode)/Homer\n",
      "\t\t142 of 440 (1e-13) E-box(bHLH)/Promoter/Homer\n",
      "\t\t143 of 440 (1e-13) X-box(HTH)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer\n",
      "\t\t144 of 440 (1e-13) E2F4(E2F)/K562-E2F4-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t145 of 440 (1e-12) CEBP:AP1(bZIP)/ThioMac-CEBPb-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t146 of 440 (1e-11) HEB(bHLH)/mES-Heb-ChIP-Seq(GSE53233)/Homer\n",
      "\t\t147 of 440 (1e-11) STAT4(Stat)/CD4-Stat4-ChIP-Seq(GSE22104)/Homer\n",
      "\t\t148 of 440 (1e-11) bHLHE40(bHLH)/HepG2-BHLHE40-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t149 of 440 (1e-11) Oct4(POU,Homeobox)/mES-Oct4-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t150 of 440 (1e-11) Gfi1b(Zf)/HPC7-Gfi1b-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t151 of 440 (1e-10) Zic(Zf)/Cerebellum-ZIC1.2-ChIP-Seq(GSE60731)/Homer\n",
      "\t\t152 of 440 (1e-10) GFX(?)/Promoter/Homer\n",
      "\t\t153 of 440 (1e-10) Oct11(POU,Homeobox)/NCIH1048-POU2F3-ChIP-seq(GSE115123)/Homer\n",
      "\t\t154 of 440 (1e-10) Rfx6(HTH)/Min6b1-Rfx6.HA-ChIP-Seq(GSE62844)/Homer\n",
      "\t\t155 of 440 (1e-10) Oct6(POU,Homeobox)/NPC-Pou3f1-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t156 of 440 (1e-10) STAT1(Stat)/HelaS3-STAT1-ChIP-Seq(GSE12782)/Homer\n",
      "\t\t157 of 440 (1e-9) STAT5(Stat)/mCD4+-Stat5-ChIP-Seq(GSE12346)/Homer\n",
      "\t\t158 of 440 (1e-9) bHLHE41(bHLH)/proB-Bhlhe41-ChIP-Seq(GSE93764)/Homer\n",
      "\t\t159 of 440 (1e-9) Nur77(NR)/K562-NR4A1-ChIP-Seq(GSE31363)/Homer\n",
      "\t\t160 of 440 (1e-9) Hoxa9(Homeobox)/ChickenMSG-Hoxa9.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t161 of 440 (1e-8) Pbx3(Homeobox)/GM12878-PBX3-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t162 of 440 (1e-8) c-Myc(bHLH)/mES-cMyc-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t163 of 440 (1e-8) Lhx3(Homeobox)/Neuron-Lhx3-ChIP-Seq(GSE31456)/Homer\n",
      "\t\t164 of 440 (1e-8) Foxo1(Forkhead)/RAW-Foxo1-ChIP-Seq(Fan_et_al.)/Homer\n",
      "\t\t165 of 440 (1e-8) Slug(Zf)/Mesoderm-Snai2-ChIP-Seq(GSE61475)/Homer\n",
      "\t\t166 of 440 (1e-8) ZBTB33(Zf)/GM12878-ZBTB33-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t167 of 440 (1e-8) E2A(bHLH),near_PU.1/Bcell-PU.1-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t168 of 440 (1e-8) T1ISRE(IRF)/ThioMac-Ifnb-Expression/Homer\n",
      "\t\t169 of 440 (1e-7) Tbx6(T-box)/ESC-Tbx6-ChIP-Seq(GSE93524)/Homer\n",
      "\t\t170 of 440 (1e-7) NPAS2(bHLH)/Liver-NPAS2-ChIP-Seq(GSE39860)/Homer\n",
      "\t\t171 of 440 (1e-7) Tbx21(T-box)/GM12878-TBX21-ChIP-Seq(Encode)/Homer\n",
      "\t\t172 of 440 (1e-7) E2F3(E2F)/MEF-E2F3-ChIP-Seq(GSE71376)/Homer\n",
      "\t\t173 of 440 (1e-7) Znf263(Zf)/K562-Znf263-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t174 of 440 (1e-7) Oct2(POU,Homeobox)/Bcell-Oct2-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t175 of 440 (1e-7) Pknox1(Homeobox)/ES-Prep1-ChIP-Seq(GSE63282)/Homer\n",
      "\t\t176 of 440 (1e-6) Mef2d(MADS)/Retina-Mef2d-ChIP-Seq(GSE61391)/Homer\n",
      "\t\t177 of 440 (1e-6) Tbx20(T-box)/Heart-Tbx20-ChIP-Seq(GSE29636)/Homer\n",
      "\t\t178 of 440 (1e-6) Tcf12(bHLH)/GM12878-Tcf12-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t179 of 440 (1e-6) Zfp281(Zf)/ES-Zfp281-ChIP-Seq(GSE81042)/Homer\n",
      "\t\t180 of 440 (1e-5) OCT:OCT-short(POU,Homeobox)/NPC-OCT6-ChIP-Seq(GSE43916)/Homer\n",
      "\t\t181 of 440 (1e-5) Stat3(Stat)/mES-Stat3-ChIP-Seq(GSE11431)/Homer\n",
      "\t\t182 of 440 (1e-5) Snail1(Zf)/LS174T-SNAIL1.HA-ChIP-Seq(GSE127183)/Homer\n",
      "\t\t183 of 440 (1e-5) GATA3(Zf),DR4/iTreg-Gata3-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t184 of 440 (1e-5) ZNF7(Zf)/HepG2-ZNF7.Flag-ChIP-Seq(Encode)/Homer\n",
      "\t\t185 of 440 (1e-5) E2F6(E2F)/Hela-E2F6-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t186 of 440 (1e-5) PAX3:FKHR-fusion(Paired,Homeobox)/Rh4-PAX3:FKHR-ChIP-Seq(GSE19063)/Homer\n",
      "\t\t187 of 440 (1e-5) Hoxa10(Homeobox)/ChickenMSG-Hoxa10.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t188 of 440 (1e-5) GATA:SCL(Zf,bHLH)/Ter119-SCL-ChIP-Seq(GSE18720)/Homer\n",
      "\t\t189 of 440 (1e-5) LHX9(Homeobox)/Hct116-LHX9.V5-ChIP-Seq(GSE116822)/Homer\n",
      "\t\t190 of 440 (1e-5) PAX5(Paired,Homeobox),condensed/GM12878-PAX5-ChIP-Seq(GSE32465)/Homer\n",
      "\t\t191 of 440 (1e-4) Tbr1(T-box)/Cortex-Tbr1-ChIP-Seq(GSE71384)/Homer\n",
      "\t\t192 of 440 (1e-4) Eomes(T-box)/H9-Eomes-ChIP-Seq(GSE26097)/Homer\n",
      "\t\t193 of 440 (1e-4) PBX1(Homeobox)/MCF7-PBX1-ChIP-Seq(GSE28007)/Homer\n",
      "\t\t194 of 440 (1e-4) Bcl11a(Zf)/HSPC-BCL11A-ChIP-Seq(GSE104676)/Homer\n",
      "\t\t195 of 440 (1e-4) Pit1+1bp(Homeobox)/GCrat-Pit1-ChIP-Seq(GSE58009)/Homer\n",
      "\t\t196 of 440 (1e-4) Nkx2.5(Homeobox)/HL1-Nkx2.5.biotin-ChIP-Seq(GSE21529)/Homer\n",
      "\t\t197 of 440 (1e-4) Sox3(HMG)/NPC-Sox3-ChIP-Seq(GSE33059)/Homer\n",
      "\t\t198 of 440 (1e-4) GATA(Zf),IR4/iTreg-Gata3-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t199 of 440 (1e-4) Ascl2(bHLH)/ESC-Ascl2-ChIP-Seq(GSE97712)/Homer\n",
      "\t\t200 of 440 (1e-4) Duxbl(Homeobox)/NIH3T3-Duxbl.HA-ChIP-Seq(GSE119782)/Homer\n",
      "\t\t201 of 440 (1e-4) HOXA1(Homeobox)/mES-Hoxa1-ChIP-Seq(SRP084292)/Homer\n",
      "\t\t202 of 440 (1e-4) Sox10(HMG)/SciaticNerve-Sox3-ChIP-Seq(GSE35132)/Homer\n",
      "\t\t203 of 440 (1e-4) Tlx?(NR)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer\n",
      "\t\t204 of 440 (1e-4) OCT:OCT(POU,Homeobox,IR1)/NPC-Brn2-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t205 of 440 (1e-4) Smad2(MAD)/ES-SMAD2-ChIP-Seq(GSE29422)/Homer\n",
      "\t\t206 of 440 (1e-4) ZNF341(Zf)/EBV-ZNF341-ChIP-Seq(GSE113194)/Homer\n",
      "\t\t207 of 440 (1e-3) GATA3(Zf),DR8/iTreg-Gata3-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t208 of 440 (1e-3) DLX2(Homeobox)/BasalGanglia-Dlx2-ChIP-seq(GSE124936)/Homer\n",
      "\t\t209 of 440 (1e-3) MafF(bZIP)/HepG2-MafF-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t210 of 440 (1e-3) Nkx2.2(Homeobox)/NPC-Nkx2.2-ChIP-Seq(GSE61673)/Homer\n",
      "\t\t211 of 440 (1e-3) Hoxc9(Homeobox)/Ainv15-Hoxc9-ChIP-Seq(GSE21812)/Homer\n",
      "\t\t212 of 440 (1e-3) Six1(Homeobox)/Myoblast-Six1-ChIP-Chip(GSE20150)/Homer\n",
      "\t\t213 of 440 (1e-3) CEBP(bZIP)/ThioMac-CEBPb-ChIP-Seq(GSE21512)/Homer\n",
      "\t\t214 of 440 (1e-3) ZSCAN22(Zf)/HEK293-ZSCAN22.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t215 of 440 (1e-3) Brn2(POU,Homeobox)/NPC-Brn2-ChIP-Seq(GSE35496)/Homer\n",
      "\t\t216 of 440 (1e-3) Srebp1a(bHLH)/HepG2-Srebp1a-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t217 of 440 (1e-3) Tbox:Smad(T-box,MAD)/ESCd5-Smad2_3-ChIP-Seq(GSE29422)/Homer\n",
      "\t\t218 of 440 (1e-3) E2F7(E2F)/Hela-E2F7-ChIP-Seq(GSE32673)/Homer\n",
      "\t\t219 of 440 (1e-3) RORgt(NR)/EL4-RORgt.Flag-ChIP-Seq(GSE56019)/Homer\n",
      "\t\t220 of 440 (1e-3) RORgt(NR)/EL4-RORgt.Flag-ChIP-Seq(GSE56019)/Homer\n",
      "\t\t221 of 440 (1e-3) DLX1(Homeobox)/BasalGanglia-Dlx1-ChIP-seq(GSE124936)/Homer\n",
      "\t\t222 of 440 (1e-3) PGR(NR)/EndoStromal-PGR-ChIP-Seq(GSE69539)/Homer\n",
      "\t\t223 of 440 (1e-3) Nkx2.1(Homeobox)/LungAC-Nkx2.1-ChIP-Seq(GSE43252)/Homer\n",
      "\t\t224 of 440 (1e-3) Zfp809(Zf)/ES-Zfp809-ChIP-Seq(GSE70799)/Homer\n",
      "\t\t225 of 440 (1e-3) TATA-Box(TBP)/Promoter/Homer\n",
      "\t\t226 of 440 (1e-3) Zfp57(Zf)/H1-ZFP57.HA-ChIP-Seq(GSE115387)/Homer\n",
      "\t\t227 of 440 (1e-3) Bapx1(Homeobox)/VertebralCol-Bapx1-ChIP-Seq(GSE36672)/Homer\n",
      "\t\t228 of 440 (1e-3) PSE(SNAPc)/K562-mStart-Seq/Homer\n",
      "\t\t229 of 440 (1e-2) ZNF317(Zf)/HEK293-ZNF317.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t230 of 440 (1e-2) Pit1(Homeobox)/GCrat-Pit1-ChIP-Seq(GSE58009)/Homer\n",
      "\t\t231 of 440 (1e-2) Nkx6.1(Homeobox)/Islet-Nkx6.1-ChIP-Seq(GSE40975)/Homer\n",
      "\t\t232 of 440 (1e-2) Six2(Homeobox)/NephronProgenitor-Six2-ChIP-Seq(GSE39837)/Homer\n",
      "\t\t233 of 440 (1e-2) Smad4(MAD)/ESC-SMAD4-ChIP-Seq(GSE29422)/Homer\n",
      "\t\t234 of 440 (1e-2) HOXA2(Homeobox)/mES-Hoxa2-ChIP-Seq(Donaldson_et_al.)/Homer\n",
      "\t\t235 of 440 (1e-2) CHR(?)/Hela-CellCycle-Expression/Homer\n",
      "\t\t236 of 440 (1e-2) Erra(NR)/HepG2-Erra-ChIP-Seq(GSE31477)/Homer\n",
      "\t\t237 of 440 (1e-2) GATA3(Zf)/iTreg-Gata3-ChIP-Seq(GSE20898)/Homer\n",
      "\t\t238 of 440 (1e-2) HIF-1b(HLH)/T47D-HIF1b-ChIP-Seq(GSE59937)/Homer\n",
      "\t\t239 of 440 (1e-2) PRDM9(Zf)/Testis-DMC1-ChIP-Seq(GSE35498)/Homer\n",
      "\t\t240 of 440 (1e-2) E2A(bHLH)/proBcell-E2A-ChIP-Seq(GSE21978)/Homer\n",
      "\t\t241 of 440 (1e-2) RORg(NR)/Liver-Rorc-ChIP-Seq(GSE101115)/Homer\n",
      "\t\t242 of 440 (1e-2) CEBP:CEBP(bZIP)/MEF-Chop-ChIP-Seq(GSE35681)/Homer\n",
      "\t\t243 of 440 (1e-2) Hoxd12(Homeobox)/ChickenMSG-Hoxd12.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t244 of 440 (1e-2) Npas4(bHLH)/Neuron-Npas4-ChIP-Seq(GSE127793)/Homer\n",
      "\t\t245 of 440 (1e-2) Nkx3.1(Homeobox)/LNCaP-Nkx3.1-ChIP-Seq(GSE28264)/Homer\n",
      "\t\t246 of 440 (1e-2) Arnt:Ahr(bHLH)/MCF7-Arnt-ChIP-Seq(Lo_et_al.)/Homer\n",
      "\t\t247 of 440 (1e-2) ZNF528(Zf)/HEK293-ZNF528.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t248 of 440 (1e-2) ETS:E-box(ETS,bHLH)/HPC7-Scl-ChIP-Seq(GSE22178)/Homer\n",
      "\t\t249 of 440 (1e-2) Sox9(HMG)/Limb-SOX9-ChIP-Seq(GSE73225)/Homer\n",
      "\t\t250 of 440 (1e-2) MYB(HTH)/ERMYB-Myb-ChIPSeq(GSE22095)/Homer\n",
      "\t\t251 of 440 (1e-2) NF1(CTF)/LNCAP-NF1-ChIP-Seq(Unpublished)/Homer\n",
      "\t\t252 of 440 (1e-2) Brachyury(T-box)/Mesoendoderm-Brachyury-ChIP-exo(GSE54963)/Homer\n",
      "\t\t253 of 440 (1e-2) Hoxd10(Homeobox)/ChickenMSG-Hoxd10.Flag-ChIP-Seq(GSE86088)/Homer\n",
      "\t\t254 of 440 (1e-2) Gata6(Zf)/HUG1N-GATA6-ChIP-Seq(GSE51936)/Homer\n",
      "\t\t255 of 440 (1e-2) DLX5(Homeobox)/BasalGanglia-Dlx5-ChIP-seq(GSE124936)/Homer\n",
      "\t\t256 of 440 (1e-2) Isl1(Homeobox)/Neuron-Isl1-ChIP-Seq(GSE31456)/Homer\n",
      "\t\t257 of 440 (1e-2) ZNF692(Zf)/HEK293-ZNF692.GFP-ChIP-Seq(GSE58341)/Homer\n",
      "\t\t258 of 440 (1e-2) SCRT1(Zf)/HEK293-SCRT1.eGFP-ChIP-Seq(Encode)/Homer\n",
      "\t\t259 of 440 (1e-2) Sox6(HMG)/Myotubes-Sox6-ChIP-Seq(GSE32627)/Homer\n",
      "\t----------------------------------------------------------\n",
      "\tDe novo motif finding (HOMER)\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 12 (from 10) to work well with 12 CPUs\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 32896\n",
      "\tAutoadjustment for sequence coverage in background: 0.97x\n",
      "\n",
      "\tOligos: 32896 of 34497 max\n",
      "\tTree  : 67060 of 172485 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 2 mismatches...\n",
      "\n",
      "\tScreening oligos 32896 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t0.00% skipped, 100.00% checked (32896 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32896 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t0.00% skipped, 100.00% checked (32896 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 51.91% increased p-value, 48.81% high p-value\n",
      "\n",
      "\tScreening oligos 32896 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t64.77% skipped, 35.23% checked (11588 of 32896), of those checked:\n",
      "\t\t0.00% not in target, 18.95% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: ATGAGTCA... (-2923.419)\n",
      "\t\tRound 1: -6615.36 ATGAGTCA T:20285.0(24.50%),B:7108.3(9.72%),P:1e-2872\n",
      "\t\tRound 2: -6615.36 ATGAGTCA T:20285.0(24.50%),B:7108.3(9.72%),P:1e-2872\n",
      "\t\t=Final=: -3165.50 ATGAGTCA T:7204.0(9.98%),B:2433.8(3.41%),P:1e-1374\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: GCCCCGCC... (-433.551)\n",
      "\t\tRound 1: -710.29 CCCCCCCC T:20308.0(24.53%),B:14534.7(18.86%),P:1e-308\n",
      "\t\tRound 2: -710.29 CCCCCCCC T:20308.0(24.53%),B:14534.7(18.86%),P:1e-308\n",
      "\t\t=Final=: -341.44 CCCCCCCC T:9797.0(13.58%),B:7488.3(10.49%),P:1e-148\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: CCACTAGA... (-337.644)\n",
      "\t\tRound 1: -486.28 CCACTAGA T:2212.0(3.02%),B:1009.3(1.44%),P:1e-211\n",
      "\t\tRound 2: -486.28 CCACTAGA T:2212.0(3.02%),B:1009.3(1.44%),P:1e-211\n",
      "\t\t=Final=: -526.99 CCACTAGA T:2185.0(3.03%),B:997.0(1.40%),P:1e-228\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: CTTCCGGT... (-290.025)\n",
      "\t\tRound 1: -527.59 CTTCCKGT T:17056.0(21.05%),B:12481.4(16.43%),P:1e-229\n",
      "\t\tRound 2: -527.59 CTTCCKGT T:17056.0(21.05%),B:12481.4(16.43%),P:1e-229\n",
      "\t\t=Final=: -578.04 CTTCCKGT T:14649.0(20.30%),B:11100.6(15.56%),P:1e-251\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: TGTGGTTT... (-237.079)\n",
      "\t\tRound 1: -413.72 TGTGGTTT T:4002.0(5.39%),B:2347.5(3.32%),P:1e-179\n",
      "\t\tRound 2: -413.72 TGTGGTTT T:4002.0(5.39%),B:2347.5(3.32%),P:1e-179\n",
      "\t\t=Final=: -413.31 TGTGGTTT T:3780.0(5.24%),B:2281.6(3.20%),P:1e-179\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: CGCCCTCT... (-229.657)\n",
      "\t\tRound 1: -718.97 CGCCCTCT T:10513.0(13.56%),B:6732.0(9.23%),P:1e-312\n",
      "\t\tRound 2: -718.97 CGCCCTCT T:10513.0(13.56%),B:6732.0(9.23%),P:1e-312\n",
      "\t\t=Final=: -870.88 CGCCCTCT T:9784.0(13.56%),B:6313.2(8.85%),P:1e-378\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: CCGATTGG... (-195.751)\n",
      "\t\tRound 1: -487.28 CYSATTGG T:7174.0(9.46%),B:4627.0(6.44%),P:1e-211\n",
      "\t\tRound 2: -487.28 CYSATTGG T:7174.0(9.46%),B:4627.0(6.44%),P:1e-211\n",
      "\t\t=Final=: -320.83 CYSATTGG T:4170.0(5.78%),B:2745.7(3.85%),P:1e-139\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: ATGACTAA... (-162.480)\n",
      "\t\tRound 1: -563.47 ATGACTMA T:9306.0(12.10%),B:6127.3(8.44%),P:1e-244\n",
      "\t\tRound 2: -743.88 RTGANTMA T:6617.0(8.76%),B:3767.9(5.27%),P:1e-323\n",
      "\t\tRound 3: -743.88 RTGANTMA T:6617.0(8.76%),B:3767.9(5.27%),P:1e-323\n",
      "\t\t=Final=: -489.07 RTGANTMA T:5034.0(6.98%),B:3142.8(4.40%),P:1e-212\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: TTTCACTT... (-151.509)\n",
      "\t\tRound 1: -217.30 TTTCRSTT T:2176.0(2.97%),B:1295.0(1.84%),P:1e-94\n",
      "\t\tRound 2: -247.02 TTTCSKTT T:7703.0(10.12%),B:5667.1(7.83%),P:1e-107\n",
      "\t\tRound 3: -269.50 TTTCAGTT T:6077.0(8.08%),B:4260.2(5.94%),P:1e-117\n",
      "\t\tRound 4: -269.50 TTTCAGTT T:6077.0(8.08%),B:4260.2(5.94%),P:1e-117\n",
      "\t\t=Final=: -228.34 TTTCAGTT T:5377.0(7.45%),B:3964.0(5.56%),P:1e-99\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: TCCGCCCA... (-120.283)\n",
      "\t\tRound 1: -280.73 TCCGCCCA T:20896.0(25.14%),B:16807.2(21.47%),P:1e-121\n",
      "\t\tRound 2: -296.50 HCCRCCYM T:16116.0(20.01%),B:12603.1(16.58%),P:1e-128\n",
      "\t\tRound 3: -296.50 HCCRCCYM T:16116.0(20.01%),B:12603.1(16.58%),P:1e-128\n",
      "\t\t=Final=: -291.18 HCCRCCYM T:13464.0(18.66%),B:10955.9(15.35%),P:1e-126\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: TGACGTCA... (-103.041)\n",
      "\t\tRound 1: -185.07 TGACGTCA T:3989.0(5.38%),B:2784.7(3.93%),P:1e-80\n",
      "\t\tRound 2: -185.07 TGACGTCA T:3989.0(5.38%),B:2784.7(3.93%),P:1e-80\n",
      "\t\t=Final=: -209.66 TGACGTCA T:3790.0(5.25%),B:2665.4(3.74%),P:1e-91\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: GTGGGAGG... (-69.013)\n",
      "\t\tRound 1: -106.20 GTGGGAGG T:11797.0(15.08%),B:9873.8(13.24%),P:1e-46\n",
      "\t\tRound 2: -106.20 GTGGGAGG T:11797.0(15.08%),B:9873.8(13.24%),P:1e-46\n",
      "\t\t=Final=: -136.13 GTGGGAGG T:6909.0(9.57%),B:5630.1(7.89%),P:1e-59\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: TTTTTTTT... (-65.705)\n",
      "\t\tRound 1: -65.70 TTTTTTTT T:3576.0(4.83%),B:2833.1(3.99%),P:1e-28\n",
      "\t\tRound 2: -65.70 TTTTTTTT T:3576.0(4.83%),B:2833.1(3.99%),P:1e-28\n",
      "\t\t=Final=: -13.35 TTTTTTTT T:1172.0(1.62%),B:1009.2(1.41%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: TGTCATAC... (-57.184)\n",
      "\t\tRound 1: -83.55 TGTCATAC T:18042.0(22.12%),B:15693.5(20.21%),P:1e-36\n",
      "\t\tRound 2: -109.42 TGTCATAC T:35804.0(39.11%),B:31563.9(36.49%),P:1e-47\n",
      "\t\tRound 3: -137.00 TGTCATAY T:33782.0(37.38%),B:29391.7(34.48%),P:1e-59\n",
      "\t\tRound 4: -137.00 TGTCATAY T:33782.0(37.38%),B:29391.7(34.48%),P:1e-59\n",
      "\t\t=Final=: -144.23 TGTCATAY T:27533.0(38.15%),B:25085.8(35.16%),P:1e-62\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: TCTCGCGA... (-54.053)\n",
      "\t\tRound 1: -69.11 TCTCGCGA T:751.0(1.04%),B:461.3(0.66%),P:1e-30\n",
      "\t\tRound 2: -69.11 TCTCGCGA T:751.0(1.04%),B:461.3(0.66%),P:1e-30\n",
      "\t\t=Final=: -43.51 TCTCGCGA T:246.0(0.34%),B:129.3(0.18%),P:1e-18\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: GCATGCGC... (-53.320)\n",
      "\t\tRound 1: -69.43 GCATGCGC T:2304.0(3.14%),B:1726.9(2.45%),P:1e-30\n",
      "\t\tRound 2: -79.01 GCTTCCGC T:5613.0(7.48%),B:4553.0(6.34%),P:1e-34\n",
      "\t\tRound 3: -79.01 GCTTCCGC T:5613.0(7.48%),B:4553.0(6.34%),P:1e-34\n",
      "\t\t=Final=: -66.19 GCTTCCGC T:5823.0(8.07%),B:4981.5(6.98%),P:1e-28\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: CAGCCAAT... (-44.808)\n",
      "\t\tRound 1: -140.45 CRGCCAAT T:3618.0(4.89%),B:2603.4(3.67%),P:1e-60\n",
      "\t\tRound 2: -140.87 CRACCAAT T:3088.0(4.19%),B:2166.5(3.07%),P:1e-61\n",
      "\t\tRound 3: -146.13 SARCCWAT T:2849.0(3.87%),B:1959.2(2.78%),P:1e-63\n",
      "\t\tRound 4: -146.13 SARCCWAT T:2849.0(3.87%),B:1959.2(2.78%),P:1e-63\n",
      "\t\t=Final=: -150.75 SARCCWAT T:2719.0(3.77%),B:1909.3(2.68%),P:1e-65\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: CCACGCCC... (-44.107)\n",
      "\t\tRound 1: -65.55 CCACGCCC T:2612.0(3.55%),B:2002.4(2.84%),P:1e-28\n",
      "\t\tRound 2: -65.55 CCACGCCC T:2612.0(3.55%),B:2002.4(2.84%),P:1e-28\n",
      "\t\t=Final=: -67.69 CCACGCCC T:2455.0(3.40%),B:1920.5(2.69%),P:1e-29\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: CACATCCT... (-42.214)\n",
      "\t\tRound 1: -97.19 CACATCCT T:2294.0(3.13%),B:1632.6(2.32%),P:1e-42\n",
      "\t\tRound 2: -102.22 TACATCCG T:5834.0(7.77%),B:4630.1(6.44%),P:1e-44\n",
      "\t\tRound 3: -122.81 TATTTCCG T:5019.0(6.72%),B:3841.3(5.37%),P:1e-53\n",
      "\t\tRound 4: -124.60 CATTTCCT T:3357.0(4.55%),B:2434.8(3.44%),P:1e-54\n",
      "\t\tRound 5: -124.60 CATTTCCT T:3357.0(4.55%),B:2434.8(3.44%),P:1e-54\n",
      "\t\t=Final=: -146.35 CATTTCCT T:3252.0(4.51%),B:2369.8(3.32%),P:1e-63\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: TGTAGTTC... (-41.499)\n",
      "\t\tRound 1: -51.29 TGTAGTTC T:893.0(1.23%),B:607.6(0.87%),P:1e-22\n",
      "\t\tRound 2: -51.29 TGTAGTTC T:893.0(1.23%),B:607.6(0.87%),P:1e-22\n",
      "\t\t=Final=: -48.43 TGTAGTTC T:844.0(1.17%),B:591.4(0.83%),P:1e-21\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: GGTTGCTA... (-36.629)\n",
      "\t\tRound 1: -64.77 GGTTGCTA T:4039.0(5.44%),B:3243.9(4.56%),P:1e-28\n",
      "\t\tRound 2: -64.77 GGTTGCTA T:4039.0(5.44%),B:3243.9(4.56%),P:1e-28\n",
      "\t\t=Final=: -80.76 GGTTGCTA T:3854.0(5.34%),B:3113.1(4.36%),P:1e-35\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: TTTGATCT... (-35.489)\n",
      "\t\tRound 1: -75.33 TTTGATCT T:1768.0(2.42%),B:1260.8(1.80%),P:1e-32\n",
      "\t\tRound 2: -78.43 KTTGATST T:3595.0(4.86%),B:2795.6(3.94%),P:1e-34\n",
      "\t\tRound 3: -78.43 KTTGATST T:3595.0(4.86%),B:2795.6(3.94%),P:1e-34\n",
      "\t\t=Final=: -87.30 KTTGATST T:3404.0(4.72%),B:2686.9(3.77%),P:1e-37\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: AAGCCACA... (-32.503)\n",
      "\t\tRound 1: -34.01 AAGCCACA T:1006.0(1.38%),B:747.7(1.07%),P:1e-14\n",
      "\t\tRound 2: -44.66 AAGCGACA T:1604.0(2.20%),B:1218.9(1.74%),P:1e-19\n",
      "\t\tRound 3: -44.66 AAGCGACA T:1604.0(2.20%),B:1218.9(1.74%),P:1e-19\n",
      "\t\t=Final=: -53.71 AAGCGACA T:1578.0(2.19%),B:1201.6(1.68%),P:1e-23\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: CCCGCCTT... (-31.659)\n",
      "\t\tRound 1: -35.97 CCCGCCTT T:1775.0(2.43%),B:1399.5(1.99%),P:1e-15\n",
      "\t\tRound 2: -40.92 NCCGCCTW T:1224.0(1.68%),B:908.2(1.30%),P:1e-17\n",
      "\t\tRound 3: -40.92 NCCGCCTW T:1224.0(1.68%),B:908.2(1.30%),P:1e-17\n",
      "\t\t=Final=: -50.10 NCCGCCTW T:1204.0(1.67%),B:890.2(1.25%),P:1e-21\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: AGTTTCAA... (-28.466)\n",
      "\t\tRound 1: -37.06 AGTTTCAA T:4252.0(5.72%),B:3593.5(5.04%),P:1e-16\n",
      "\t\tRound 2: -47.11 AGTTTCCA T:1859.0(2.54%),B:1427.2(2.03%),P:1e-20\n",
      "\t\tRound 3: -47.11 AGTTTCCA T:1859.0(2.54%),B:1427.2(2.03%),P:1e-20\n",
      "\t\t=Final=: -52.60 AGTTTCCA T:1811.0(2.51%),B:1408.5(1.97%),P:1e-22\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: SRR7475873_motif_macs2/homerMotifs.motifs8\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 12 (from 10) to work well with 12 CPUs\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 522943\n",
      "\tAutoadjustment for sequence coverage in background: 0.97x\n",
      "\n",
      "\tOligos: 522943 of 551887 max\n",
      "\tTree  : 1069424 of 2759435 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 2 mismatches...\n",
      "\n",
      "\tScreening oligos 522943 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t1.13% skipped, 98.87% checked (517031 of 522943), of those checked:\n",
      "\t\t1.13% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 522943 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t1.13% skipped, 98.87% checked (517031 of 522943), of those checked:\n",
      "\t\t0.00% not in target, 52.26% increased p-value, 47.19% high p-value\n",
      "\n",
      "\tScreening oligos 522943 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t71.15% skipped, 28.85% checked (150852 of 522943), of those checked:\n",
      "\t\t0.00% not in target, 12.20% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: ATGAGTCATA... (-2276.558)\n",
      "\t\tRound 1: -6853.26 ATGAGTCATA T:15754.0(19.61%),B:4679.5(6.51%),P:1e-2976\n",
      "\t\tRound 2: -7175.25 ATGASTCATH T:17585.0(21.63%),B:5410.7(7.49%),P:1e-3116\n",
      "\t\tRound 3: -7175.25 ATGASTCATH T:17585.0(21.63%),B:5410.7(7.49%),P:1e-3116\n",
      "\t\t=Final=: -3448.53 ATGASTCATH T:8253.0(11.44%),B:2892.8(4.05%),P:1e-1497\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: CCACTAGAGG... (-843.651)\n",
      "\t\tRound 1: -2384.59 CCACYAGRGG T:4627.0(6.21%),B:1286.2(1.83%),P:1e-1035\n",
      "\t\tRound 2: -2384.59 CCACYAGRGG T:4627.0(6.21%),B:1286.2(1.83%),P:1e-1035\n",
      "\t\t=Final=: -2532.71 CCACYAGRGG T:4529.0(6.28%),B:1274.8(1.79%),P:1e-1099\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: GGCCCCGCCC... (-556.440)\n",
      "\t\tRound 1: -964.88 GGCCCCGCCC T:17452.0(21.48%),B:11554.7(15.32%),P:1e-419\n",
      "\t\tRound 2: -1014.96 RGYHCCGCCC T:24744.0(29.03%),B:17149.9(21.87%),P:1e-440\n",
      "\t\tRound 3: -1035.69 RGYCCCGCCC T:19870.0(24.07%),B:13262.2(17.37%),P:1e-449\n",
      "\t\tRound 4: -1061.26 RGYYCCGCCC T:19893.0(24.09%),B:13216.5(17.32%),P:1e-460\n",
      "\t\tRound 5: -1063.49 GGCYCCGCCC T:15136.0(18.92%),B:9556.6(12.85%),P:1e-461\n",
      "\t\tRound 6: -1074.63 RGYHCCGCCC T:14646.0(18.37%),B:9163.8(12.35%),P:1e-466\n",
      "\t\tRound 7: -1074.63 RGYHCCGCCC T:14646.0(18.37%),B:9163.8(12.35%),P:1e-466\n",
      "\t\t=Final=: -711.54 RGYHCCGCCC T:10704.0(14.83%),B:7369.5(10.33%),P:1e-309\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: CACTTCCTGT... (-295.812)\n",
      "\t\tRound 1: -686.81 CACTTCCTGT T:6895.0(9.11%),B:4055.5(5.67%),P:1e-298\n",
      "\t\tRound 2: -906.65 CACTTCCKGY T:11853.0(15.15%),B:7380.0(10.07%),P:1e-393\n",
      "\t\tRound 3: -976.61 CAYTTCCKGT T:14788.0(18.53%),B:9472.8(12.74%),P:1e-424\n",
      "\t\tRound 4: -976.61 CAYTTCCKGT T:14788.0(18.53%),B:9472.8(12.74%),P:1e-424\n",
      "\t\t=Final=: -987.76 CAYTTCCKGT T:12766.0(17.69%),B:8564.9(12.00%),P:1e-428\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: GAAAGTGAAA... (-243.063)\n",
      "\t\tRound 1: -393.44 GAAAGTGAAA T:1258.0(1.73%),B:488.2(0.70%),P:1e-170\n",
      "\t\tRound 2: -393.44 GAAAGTGAAA T:1258.0(1.73%),B:488.2(0.70%),P:1e-170\n",
      "\t\t=Final=: -362.75 GAAAGTGAAA T:1174.0(1.63%),B:475.2(0.67%),P:1e-157\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: GATGACGTCA... (-143.672)\n",
      "\t\tRound 1: -279.14 GATGACGTCA T:2599.0(3.54%),B:1513.8(2.15%),P:1e-121\n",
      "\t\tRound 2: -290.35 GATGACGTCA T:2859.0(3.88%),B:1689.1(2.40%),P:1e-126\n",
      "\t\tRound 3: -302.00 GRTGACGTCA T:10373.0(13.39%),B:7707.2(10.50%),P:1e-131\n",
      "\t\tRound 4: -311.76 GRTGACGTCA T:5320.0(7.11%),B:3547.0(4.97%),P:1e-135\n",
      "\t\tRound 5: -313.25 GGTGACGTCA T:5919.0(7.87%),B:4020.7(5.62%),P:1e-136\n",
      "\t\tRound 6: -326.15 NRTGACGTCA T:5098.0(6.82%),B:3341.4(4.69%),P:1e-141\n",
      "\t\tRound 7: -326.15 NRTGACGTCA T:5098.0(6.82%),B:3341.4(4.69%),P:1e-141\n",
      "\t\t=Final=: -236.98 NRTGACGTCA T:3429.0(4.75%),B:2307.1(3.23%),P:1e-102\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: AACCAATCAG... (-139.635)\n",
      "\t\tRound 1: -295.30 AACCAATCAG T:1975.0(2.70%),B:1042.6(1.49%),P:1e-128\n",
      "\t\tRound 2: -525.49 ARCCAATCAG T:6194.0(8.23%),B:3801.8(5.32%),P:1e-228\n",
      "\t\tRound 3: -684.04 RGCCAATCRG T:4858.0(6.51%),B:2591.4(3.66%),P:1e-297\n",
      "\t\tRound 4: -684.04 RGCCAATCRG T:4858.0(6.51%),B:2591.4(3.66%),P:1e-297\n",
      "\t\t=Final=: -448.87 RGCCAATCRG T:3883.0(5.38%),B:2308.1(3.23%),P:1e-194\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: AGCGCCACCT... (-117.175)\n",
      "\t\tRound 1: -273.32 AGCGCCACCT T:2723.0(3.70%),B:1614.5(2.30%),P:1e-118\n",
      "\t\tRound 2: -357.02 AGYGCCAYCT T:6353.0(8.43%),B:4263.6(5.95%),P:1e-155\n",
      "\t\tRound 3: -357.02 AGYGCCAYCT T:6353.0(8.43%),B:4263.6(5.95%),P:1e-155\n",
      "\t\t=Final=: -399.84 AGYGCCAYCT T:6023.0(8.35%),B:4105.7(5.75%),P:1e-173\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: TGTGGTTTCG... (-110.358)\n",
      "\t\tRound 1: -233.96 TGTGGTTTCG T:2757.0(3.75%),B:1705.6(2.42%),P:1e-101\n",
      "\t\tRound 2: -358.11 TGTGGTTTTV T:2693.0(3.66%),B:1472.3(2.10%),P:1e-155\n",
      "\t\tRound 3: -358.11 TGTGGTTTTV T:2693.0(3.66%),B:1472.3(2.10%),P:1e-155\n",
      "\t\t=Final=: -371.08 TGTGGTTTTV T:2588.0(3.59%),B:1438.4(2.02%),P:1e-161\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: TGCGCATGCG... (-93.948)\n",
      "\t\tRound 1: -124.15 TGCGCATGCG T:3032.0(4.11%),B:2165.4(3.07%),P:1e-53\n",
      "\t\tRound 2: -132.39 TGCGCATGCG T:2765.0(3.76%),B:1924.1(2.73%),P:1e-57\n",
      "\t\tRound 3: -141.54 TDCGCATRCG T:2471.0(3.37%),B:1665.4(2.37%),P:1e-61\n",
      "\t\tRound 4: -141.54 TDCGCATRCG T:2471.0(3.37%),B:1665.4(2.37%),P:1e-61\n",
      "\t\t=Final=: -95.52 TDCGCATRCG T:1634.0(2.26%),B:1136.7(1.59%),P:1e-41\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: TTCCACCCAC... (-77.524)\n",
      "\t\tRound 1: -130.36 TTCCACCCAC T:6179.0(8.21%),B:4802.8(6.68%),P:1e-56\n",
      "\t\tRound 2: -204.28 TWCCACCCAC T:6239.0(8.28%),B:4582.2(6.38%),P:1e-88\n",
      "\t\tRound 3: -226.36 HHCCACCCAC T:6562.0(8.69%),B:4781.4(6.65%),P:1e-98\n",
      "\t\tRound 4: -226.36 HHCCACCCAC T:6562.0(8.69%),B:4781.4(6.65%),P:1e-98\n",
      "\t\t=Final=: -259.25 HHCCACCCAC T:6031.0(8.36%),B:4443.0(6.23%),P:1e-112\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: GTATGACTCA... (-74.357)\n",
      "\t\tRound 1: -167.63 GTATGACTCA T:10191.0(13.17%),B:8101.1(11.00%),P:1e-72\n",
      "\t\tRound 2: -183.26 GTATGACTCA T:9537.0(12.38%),B:7462.6(10.18%),P:1e-79\n",
      "\t\tRound 3: -234.31 GTATGACTCA T:8345.0(10.92%),B:6245.3(8.60%),P:1e-101\n",
      "\t\tRound 4: -267.05 GTATGACTCA T:16939.0(20.92%),B:13446.6(17.59%),P:1e-115\n",
      "\t\tRound 5: -294.62 GTATGACTCA T:13884.0(17.50%),B:10700.8(14.27%),P:1e-127\n",
      "\t\tRound 6: -318.40 GWVTGACTCA T:9309.0(12.10%),B:6768.2(9.28%),P:1e-138\n",
      "\t\tRound 7: -335.66 NNVTGASTCA T:12933.0(16.41%),B:9731.2(13.07%),P:1e-145\n",
      "\t\tRound 8: -337.33 NNVTGASTCA T:15434.0(19.25%),B:11836.8(15.66%),P:1e-146\n",
      "\t\tRound 9: -337.33 NNVTGASTCA T:15434.0(19.25%),B:11836.8(15.66%),P:1e-146\n",
      "\t\t=Final=: -238.80 NNVTGASTCA T:10062.0(13.94%),B:8068.3(11.31%),P:1e-103\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: ACTACAATTC... (-65.816)\n",
      "\t\tRound 1: -133.08 ACTACAATTC T:574.0(0.79%),B:259.6(0.37%),P:1e-57\n",
      "\t\tRound 2: -133.08 ACTACAATTC T:574.0(0.79%),B:259.6(0.37%),P:1e-57\n",
      "\t\t=Final=: -122.06 ACTACAATTC T:537.0(0.74%),B:252.5(0.35%),P:1e-53\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: TTTTTTTTTT... (-54.799)\n",
      "\t\tRound 1: -65.69 TTTTTTTTTT T:2714.0(3.69%),B:2088.4(2.96%),P:1e-28\n",
      "\t\tRound 2: -65.69 TTTTTTTTTT T:2714.0(3.69%),B:2088.4(2.96%),P:1e-28\n",
      "\t\t=Final=: -18.88 TTTTTTTTTT T:901.0(1.25%),B:733.6(1.03%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: GGGGATTTCC... (-53.635)\n",
      "\t\tRound 1: -95.55 GGGGATTTCC T:2863.0(3.89%),B:2109.6(2.99%),P:1e-41\n",
      "\t\tRound 2: -98.05 GGGGAWTTCC T:1746.0(2.39%),B:1185.5(1.69%),P:1e-42\n",
      "\t\tRound 3: -112.10 GGGRATTTCC T:842.0(1.16%),B:465.0(0.67%),P:1e-48\n",
      "\t\tRound 4: -118.84 GGGGAATTCC T:1019.0(1.40%),B:584.9(0.84%),P:1e-51\n",
      "\t\tRound 5: -135.49 GGGGAATTCC T:1043.0(1.43%),B:579.8(0.83%),P:1e-58\n",
      "\t\tRound 6: -135.49 GGGGAATTCC T:1043.0(1.43%),B:579.8(0.83%),P:1e-58\n",
      "\t\t=Final=: -105.67 GGGGAATTCC T:886.0(1.23%),B:521.5(0.73%),P:1e-45\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: CCCGCCCTCT... (-46.624)\n",
      "\t\tRound 1: -97.78 CCCGCCCTCT T:8653.0(11.30%),B:7128.3(9.75%),P:1e-42\n",
      "\t\tRound 2: -137.00 CCCGCCCTCT T:8289.0(10.85%),B:6597.8(9.06%),P:1e-59\n",
      "\t\tRound 3: -170.01 CCCGCCCYCT T:14227.0(17.89%),B:11614.3(15.39%),P:1e-73\n",
      "\t\tRound 4: -178.33 CCCGCCCYCT T:10093.0(13.05%),B:7965.0(10.83%),P:1e-77\n",
      "\t\tRound 5: -182.84 YCCGCCCHCT T:9621.0(12.48%),B:7538.0(10.28%),P:1e-79\n",
      "\t\tRound 6: -192.27 YCCGCCCHCT T:9820.0(12.72%),B:7666.4(10.45%),P:1e-83\n",
      "\t\tRound 7: -192.27 YCCGCCCHCT T:9820.0(12.72%),B:7666.4(10.45%),P:1e-83\n",
      "\t\t=Final=: -214.43 YCCGCCCHCT T:8542.0(11.84%),B:6793.0(9.52%),P:1e-93\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: GAGTGACACG... (-45.579)\n",
      "\t\tRound 1: -52.89 GAGTAACACS T:46.0(0.06%),B:8.0(0.01%),P:1e-22\n",
      "\t\tRound 2: -54.74 GATTGACACG T:31.0(0.04%),B:3.8(0.00%),P:1e-23\n",
      "\t\tRound 3: -54.74 GATTGACACG T:31.0(0.04%),B:3.8(0.00%),P:1e-23\n",
      "\t\t=Final=: -46.63 GATTGACACG T:31.0(0.04%),B:3.8(0.01%),P:1e-20\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: CGGCTGTGTA... (-45.579)\n",
      "\t\tRound 1: -57.84 CGGCTGTGTA T:37.0(0.05%),B:4.7(0.01%),P:1e-25\n",
      "\t\tRound 2: -57.84 CGGCTGTGTA T:37.0(0.05%),B:4.7(0.01%),P:1e-25\n",
      "\t\t=Final=: -51.56 CGGCTGTGTA T:37.0(0.05%),B:4.7(0.01%),P:1e-22\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: CTTCCGGTAT... (-42.687)\n",
      "\t\tRound 1: -81.38 CTTCCGGTAT T:490.0(0.68%),B:253.3(0.36%),P:1e-35\n",
      "\t\tRound 2: -98.17 CTTCCGGTDT T:1948.0(2.66%),B:1347.0(1.92%),P:1e-42\n",
      "\t\tRound 3: -101.88 CTTCCGGTWT T:4368.0(5.87%),B:3364.0(4.73%),P:1e-44\n",
      "\t\tRound 4: -111.67 CTTCCGGTWT T:2287.0(3.12%),B:1589.2(2.26%),P:1e-48\n",
      "\t\tRound 5: -127.68 CTTCCGGTTT T:2711.0(3.69%),B:1892.7(2.69%),P:1e-55\n",
      "\t\tRound 6: -127.68 CTTCCGGTTT T:2711.0(3.69%),B:1892.7(2.69%),P:1e-55\n",
      "\t\t=Final=: -142.39 CTTCCGGTTT T:2615.0(3.62%),B:1842.2(2.58%),P:1e-61\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: CCTTTGATCT... (-42.967)\n",
      "\t\tRound 1: -97.29 CCTTTGATCT T:2016.0(2.75%),B:1405.1(2.00%),P:1e-42\n",
      "\t\tRound 2: -112.86 CCTTTGATCT T:3416.0(4.62%),B:2518.0(3.56%),P:1e-49\n",
      "\t\tRound 3: -141.15 CCTTTGATCT T:3326.0(4.50%),B:2360.0(3.34%),P:1e-61\n",
      "\t\tRound 4: -154.94 YCTTTGATST T:4246.0(5.71%),B:3079.9(4.33%),P:1e-67\n",
      "\t\tRound 5: -166.63 YCTTTGATST T:3596.0(4.86%),B:2511.7(3.55%),P:1e-72\n",
      "\t\tRound 6: -174.56 YCTTTGATST T:5072.0(6.79%),B:3705.3(5.19%),P:1e-75\n",
      "\t\tRound 7: -174.56 YCTTTGATST T:5072.0(6.79%),B:3705.3(5.19%),P:1e-75\n",
      "\t\t=Final=: -179.05 YCTTTGATST T:4581.0(6.35%),B:3417.5(4.79%),P:1e-77\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: TGACTACACT... (-42.570)\n",
      "\t\tRound 1: -45.42 TGACTACACT T:48.0(0.07%),B:9.0(0.01%),P:1e-19\n",
      "\t\tRound 2: -64.54 CGACTACACT T:28.0(0.04%),B:2.1(0.00%),P:1e-28\n",
      "\t\tRound 3: -64.54 CGACTACACT T:28.0(0.04%),B:2.1(0.00%),P:1e-28\n",
      "\t\t=Final=: -50.12 CGACTACACT T:28.0(0.04%),B:2.1(0.00%),P:1e-21\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: ACTGGTAGTT... (-42.570)\n",
      "\t\tRound 1: -42.57 ACTGGTAGTT T:21.0(0.03%),B:2.4(0.00%),P:1e-18\n",
      "\t\tRound 2: -42.57 ACTGGTAGTT T:21.0(0.03%),B:2.4(0.00%),P:1e-18\n",
      "\t\t=Final=: -32.52 ACTGGTAGTT T:21.0(0.03%),B:2.4(0.00%),P:1e-14\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: TTAATGTTGC... (-42.570)\n",
      "\t\tRound 1: -42.57 TTAATGTTGC T:21.0(0.03%),B:2.9(0.00%),P:1e-18\n",
      "\t\tRound 2: -42.57 TTAATGTTGC T:21.0(0.03%),B:2.9(0.00%),P:1e-18\n",
      "\t\t=Final=: -32.52 TTAATGTTGC T:21.0(0.03%),B:2.9(0.00%),P:1e-14\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: TGTAAGTGTA... (-42.570)\n",
      "\t\tRound 1: -48.63 TGTAAGTGTA T:23.0(0.03%),B:2.5(0.00%),P:1e-21\n",
      "\t\tRound 2: -48.63 TGTAAGTGTA T:23.0(0.03%),B:2.5(0.00%),P:1e-21\n",
      "\t\t=Final=: -37.34 TGTAAGTGTA T:23.0(0.03%),B:2.5(0.00%),P:1e-16\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: CATACTGTAC... (-39.609)\n",
      "\t\tRound 1: -67.84 CATACTGTAC T:29.0(0.04%),B:2.8(0.00%),P:1e-29\n",
      "\t\tRound 2: -67.84 CATACTGTAC T:29.0(0.04%),B:2.8(0.00%),P:1e-29\n",
      "\t\t=Final=: -47.49 CATACTGTAC T:27.0(0.04%),B:2.8(0.00%),P:1e-20\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: SRR7475873_motif_macs2/homerMotifs.motifs10\n",
      "\n",
      "\n",
      "\tNumber of Trial motifs (-T) set to 12 (from 10) to work well with 12 CPUs\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 5852682\n",
      "\tAutoadjustment for sequence coverage in background: 0.97x\n",
      "\n",
      "\tOligos: 5852682 of 8830118 max\n",
      "\tTree  : 12663120 of 44150590 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 2 mismatches...\n",
      "\n",
      "\tScreening oligos 5852682 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t21.63% skipped, 78.37% checked (4586588 of 5852682), of those checked:\n",
      "\t\t21.63% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 5852682 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t21.63% skipped, 78.37% checked (4586588 of 5852682), of those checked:\n",
      "\t\t0.00% not in target, 34.07% increased p-value, 44.95% high p-value\n",
      "\n",
      "\tScreening oligos 5852682 (allowing 2 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t73.50% skipped, 26.50% checked (1551208 of 5852682), of those checked:\n",
      "\t\t0.00% not in target, 16.52% increased p-value, 0.00% high p-value\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: CCACTAGAGGGC... (-1910.479)\n",
      "\t\tRound 1: -4289.49 CCACYAGAGGGC T:4353.0(5.85%),B:654.8(0.94%),P:1e-1862\n",
      "\t\tRound 2: -4495.98 CCACTAGRKGGC T:4437.0(5.96%),B:643.0(0.92%),P:1e-1952\n",
      "\t\tRound 3: -4522.33 CCACCAGGKGGC T:5068.0(6.78%),B:849.3(1.21%),P:1e-1964\n",
      "\t\tRound 4: -4539.80 CCACYAGRKGGC T:5810.0(7.74%),B:1114.6(1.59%),P:1e-1971\n",
      "\t\tRound 5: -4539.80 CCACYAGRKGGC T:5810.0(7.74%),B:1114.6(1.59%),P:1e-1971\n",
      "\t\t=Final=: -4658.63 CCACYAGRKGGC T:5511.0(7.64%),B:1077.3(1.51%),P:1e-2023\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: AGTATGAGTCAT... (-772.617)\n",
      "\t\tRound 1: -4473.96 AGTATGAGTCAT T:17450.0(21.48%),B:7035.0(9.63%),P:1e-1943\n",
      "\t\tRound 2: -6604.32 NNDVTGASTCAB T:18538.0(22.65%),B:6198.0(8.54%),P:1e-2868\n",
      "\t\tRound 3: -6604.32 NNDVTGASTCAB T:18538.0(22.65%),B:6198.0(8.54%),P:1e-2868\n",
      "\t\t=Final=: -3234.39 NNDVTGASTCAB T:7928.0(10.99%),B:2815.7(3.95%),P:1e-1404\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: GGGGCGGGGCCT... (-446.807)\n",
      "\t\tRound 1: -834.70 GGGGCGGGGCCT T:11437.0(14.66%),B:7199.6(9.85%),P:1e-362\n",
      "\t\tRound 2: -1037.14 GGGGCGGGGCCT T:10429.0(13.46%),B:6089.4(8.39%),P:1e-450\n",
      "\t\tRound 3: -1125.57 GGGGCGGGGCCW T:14305.0(17.98%),B:8804.6(11.91%),P:1e-488\n",
      "\t\tRound 4: -1128.76 GGGGCGGGGCCW T:21436.0(25.70%),B:14239.1(18.54%),P:1e-490\n",
      "\t\tRound 5: -1128.76 GGGGCGGGGCCW T:21436.0(25.70%),B:14239.1(18.54%),P:1e-490\n",
      "\t\t=Final=: -717.35 GGGGCGGGGCCW T:7393.0(10.24%),B:4643.4(6.51%),P:1e-311\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: GAAAGTGAAAGT... (-341.936)\n",
      "\t\tRound 1: -609.02 GAAAGTGAAAGT T:1288.0(1.77%),B:389.7(0.56%),P:1e-264\n",
      "\t\tRound 2: -648.65 GAAAGTGAAAST T:1354.0(1.86%),B:405.2(0.58%),P:1e-281\n",
      "\t\tRound 3: -648.65 GAAAGTGAAAST T:1354.0(1.86%),B:405.2(0.58%),P:1e-281\n",
      "\t\t=Final=: -593.95 GAAAGTGAAAST T:1243.0(1.72%),B:387.8(0.54%),P:1e-257\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: AAACAGGAAGTG... (-225.982)\n",
      "\t\tRound 1: -408.87 AAACAGGAAGTG T:4240.0(5.71%),B:2528.4(3.57%),P:1e-177\n",
      "\t\tRound 2: -605.93 AAACAGGAAGTG T:3730.0(5.04%),B:1900.8(2.70%),P:1e-263\n",
      "\t\tRound 3: -639.24 ARACAGGAAGTG T:4450.0(5.98%),B:2359.1(3.34%),P:1e-277\n",
      "\t\tRound 4: -663.20 ARACAGGAAGTG T:4318.0(5.81%),B:2239.7(3.17%),P:1e-288\n",
      "\t\tRound 5: -663.20 ARACAGGAAGTG T:4318.0(5.81%),B:2239.7(3.17%),P:1e-288\n",
      "\t\t=Final=: -695.65 ARACAGGAAGTG T:4111.0(5.70%),B:2169.5(3.04%),P:1e-302\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: ACTACAATTCCC... (-174.158)\n",
      "\t\tRound 1: -228.08 ACTACAATTCCC T:894.0(1.23%),B:385.1(0.55%),P:1e-99\n",
      "\t\tRound 2: -240.65 ACTACAATTCCC T:1186.0(1.63%),B:562.4(0.81%),P:1e-104\n",
      "\t\tRound 3: -240.65 ACTACAATTCCC T:1186.0(1.63%),B:562.4(0.81%),P:1e-104\n",
      "\t\t=Final=: -188.28 ACTACAATTCCC T:782.0(1.08%),B:357.5(0.50%),P:1e-81\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: CGACCAATCAGA... (-134.123)\n",
      "\t\tRound 1: -342.88 CGACCAATCAGA T:2175.0(2.97%),B:1126.6(1.61%),P:1e-148\n",
      "\t\tRound 2: -482.34 CRGCCAATCAGA T:3176.0(4.31%),B:1659.5(2.36%),P:1e-209\n",
      "\t\tRound 3: -642.74 CRGCCAATCAGV T:4267.0(5.74%),B:2228.3(3.16%),P:1e-279\n",
      "\t\tRound 4: -700.42 CRGCCAATSAGV T:4906.0(6.57%),B:2603.5(3.68%),P:1e-304\n",
      "\t\tRound 5: -712.66 CRGCCAATCAGR T:5475.0(7.31%),B:2988.2(4.21%),P:1e-309\n",
      "\t\tRound 6: -712.66 CRGCCAATCAGR T:5475.0(7.31%),B:2988.2(4.21%),P:1e-309\n",
      "\t\t=Final=: -498.05 CRGCCAATCAGR T:3157.0(4.37%),B:1702.5(2.39%),P:1e-216\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: CATCTAGTGGTA... (-102.533)\n",
      "\t\tRound 1: -106.16 CATCTAGTGGTA T:40.0(0.06%),B:0.0(0.00%),P:1e-46\n",
      "\t\tRound 2: -106.16 CATCTAGTGGTA T:40.0(0.06%),B:0.0(0.00%),P:1e-46\n",
      "\t\t=Final=: -110.87 CATCTAGTGGTA T:40.0(0.06%),B:0.0(0.00%),P:1e-48\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: ATGCGCATGCGC... (-94.615)\n",
      "\t\tRound 1: -149.46 ATGCGCATGCGC T:3704.0(5.00%),B:2646.0(3.74%),P:1e-64\n",
      "\t\tRound 2: -149.46 ATGCGCATGCGC T:3704.0(5.00%),B:2646.0(3.74%),P:1e-64\n",
      "\t\t=Final=: -86.13 ATGCGCATGCGC T:1268.0(1.76%),B:856.4(1.20%),P:1e-37\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: GTGACGTCACTA... (-86.971)\n",
      "\t\tRound 1: -179.38 NNGACGTCACNN T:3121.0(4.23%),B:2097.5(2.97%),P:1e-77\n",
      "\t\tRound 2: -208.90 STGACGTCACYN T:2517.0(3.43%),B:1565.9(2.23%),P:1e-90\n",
      "\t\tRound 3: -213.01 GTGACGTCABSG T:2195.0(3.00%),B:1314.8(1.87%),P:1e-92\n",
      "\t\tRound 4: -221.10 GTGACGTCASCG T:1939.0(2.65%),B:1112.7(1.59%),P:1e-96\n",
      "\t\tRound 5: -221.10 GTGACGTCASCG T:1939.0(2.65%),B:1112.7(1.59%),P:1e-96\n",
      "\t\t=Final=: -119.90 GTGACGTCASCG T:1147.0(1.59%),B:699.6(0.98%),P:1e-52\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: TTGATGTGGTTT... (-85.176)\n",
      "\t\tRound 1: -155.45 TTGATGTGGTTT T:2720.0(3.70%),B:1831.8(2.60%),P:1e-67\n",
      "\t\tRound 2: -215.92 TTGATGTGGTTT T:3770.0(5.09%),B:2531.3(3.58%),P:1e-93\n",
      "\t\tRound 3: -228.37 TTGHTGTGGTTT T:1744.0(2.39%),B:961.8(1.37%),P:1e-99\n",
      "\t\tRound 4: -238.19 TKGHTGTGGTTT T:2904.0(3.94%),B:1810.0(2.57%),P:1e-103\n",
      "\t\tRound 5: -246.03 TKGYTGTGGTTT T:1565.0(2.15%),B:813.8(1.16%),P:1e-106\n",
      "\t\tRound 6: -246.03 TKGYTGTGGTTT T:1565.0(2.15%),B:813.8(1.16%),P:1e-106\n",
      "\t\t=Final=: -249.31 TKGYTGTGGTTT T:1514.0(2.10%),B:802.7(1.12%),P:1e-108\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: CACTAGAGGGCG... (-74.584)\n",
      "\t\tRound 1: -150.76 CACTAGAGGGCG T:8087.0(10.60%),B:6355.9(8.74%),P:1e-65\n",
      "\t\tRound 2: -164.51 CACYAGAGGGCG T:9542.0(12.39%),B:7549.7(10.30%),P:1e-71\n",
      "\t\tRound 3: -164.51 CACYAGAGGGCG T:9542.0(12.39%),B:7549.7(10.30%),P:1e-71\n",
      "\t\t=Final=: -200.21 CACYAGAGGGCG T:8653.0(11.99%),B:6945.1(9.73%),P:1e-86\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: GTGGGTGGAACC... (-71.602)\n",
      "\t\tRound 1: -95.07 GTGGGTGGAACC T:1721.0(2.36%),B:1170.4(1.67%),P:1e-41\n",
      "\t\tRound 2: -108.15 GTGGGTGGAACC T:3899.0(5.26%),B:2939.5(4.14%),P:1e-46\n",
      "\t\tRound 3: -118.45 GTGGGTGGAACC T:2079.0(2.84%),B:1404.7(2.00%),P:1e-51\n",
      "\t\tRound 4: -118.45 GTGGGTGGAACC T:2079.0(2.84%),B:1404.7(2.00%),P:1e-51\n",
      "\t\t=Final=: -126.55 GTGGGTGGAACC T:2003.0(2.78%),B:1371.2(1.92%),P:1e-54\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GGGGATTTCCCC... (-69.117)\n",
      "\t\tRound 1: -103.96 GGGGATTTCCCC T:2392.0(3.26%),B:1693.6(2.41%),P:1e-45\n",
      "\t\tRound 2: -103.96 GGGGATTTCCCC T:2392.0(3.26%),B:1693.6(2.41%),P:1e-45\n",
      "\t\t=Final=: -103.70 GGGGATTTCCCC T:2185.0(3.03%),B:1577.5(2.21%),P:1e-45\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: TCACTTCCGGTT... (-65.734)\n",
      "\t\tRound 1: -153.14 TNACTTCCGGBN T:1668.0(2.28%),B:1015.0(1.45%),P:1e-66\n",
      "\t\tRound 2: -249.65 KCACTTCCGGKN T:3369.0(4.56%),B:2145.6(3.04%),P:1e-108\n",
      "\t\tRound 3: -261.73 BCRCTTCCGGBN T:4470.0(6.01%),B:2984.6(4.21%),P:1e-113\n",
      "\t\tRound 4: -282.48 CCRCTTCCGSNN T:5418.0(7.23%),B:3691.4(5.18%),P:1e-122\n",
      "\t\tRound 5: -283.45 BSGTTTCCGGNB T:8211.0(10.75%),B:5967.5(8.23%),P:1e-123\n",
      "\t\tRound 6: -290.15 CNVCTTCCGSNN T:8725.0(11.39%),B:6371.2(8.76%),P:1e-126\n",
      "\t\tRound 7: -292.05 SNVCTTCCGSNC T:8891.0(11.59%),B:6503.2(8.94%),P:1e-126\n",
      "\t\tRound 8: -292.05 SNVCTTCCGSNC T:8891.0(11.59%),B:6503.2(8.94%),P:1e-126\n",
      "\t\t=Final=: -263.51 SNVCTTCCGSNC T:7388.0(10.24%),B:5607.2(7.86%),P:1e-114\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: ACGCATGCGTTG... (-64.529)\n",
      "\t\tRound 1: -77.92 ACGCATGCGTTG T:32.0(0.04%),B:2.6(0.00%),P:1e-33\n",
      "\t\tRound 2: -77.92 ACGCATGCGTTG T:32.0(0.04%),B:2.6(0.00%),P:1e-33\n",
      "\t\t=Final=: -58.70 ACGCATGCGTTG T:25.0(0.03%),B:2.0(0.00%),P:1e-25\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: TAGACTAGAGCG... (-61.269)\n",
      "\t\tRound 1: -68.56 TAGACTAGAGCG T:36.0(0.05%),B:3.1(0.00%),P:1e-29\n",
      "\t\tRound 2: -68.56 TAGACTAGAGCG T:36.0(0.05%),B:3.1(0.00%),P:1e-29\n",
      "\t\t=Final=: -58.72 TAGACTAGAGCG T:36.0(0.05%),B:3.1(0.00%),P:1e-25\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: GCTTAGTCACTA... (-59.865)\n",
      "\t\tRound 1: -128.10 NNTTAGTCACBA T:639.0(0.88%),B:306.6(0.44%),P:1e-55\n",
      "\t\tRound 2: -224.61 KYTTAGTCACTA T:1024.0(1.41%),B:470.7(0.67%),P:1e-97\n",
      "\t\tRound 3: -298.20 NCTTAGTCACWD T:642.0(0.89%),B:198.9(0.28%),P:1e-129\n",
      "\t\tRound 4: -298.20 NCTTAGTCACWD T:642.0(0.89%),B:198.9(0.28%),P:1e-129\n",
      "\t\t=Final=: -289.63 NCTTAGTCACWD T:591.0(0.82%),B:181.3(0.25%),P:1e-125\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: CCATATATGGCG... (-54.865)\n",
      "\t\tRound 1: -84.80 CCATATATGGCG T:34.0(0.05%),B:3.0(0.00%),P:1e-36\n",
      "\t\tRound 2: -84.80 CCATATATGGCG T:34.0(0.05%),B:3.0(0.00%),P:1e-36\n",
      "\t\t=Final=: -50.12 CCATATATGGCG T:28.0(0.04%),B:3.0(0.00%),P:1e-21\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: CAAGGAGCCTGC... (-54.865)\n",
      "\t\tRound 1: -71.15 CAAGGAGCCTGC T:30.0(0.04%),B:2.6(0.00%),P:1e-30\n",
      "\t\tRound 2: -84.80 CAAGGAGCYTGC T:34.0(0.05%),B:2.6(0.00%),P:1e-36\n",
      "\t\tRound 3: -84.80 CAAGGAGCYTGC T:34.0(0.05%),B:2.6(0.00%),P:1e-36\n",
      "\t\t=Final=: -47.49 CAAGGAGCYTGC T:27.0(0.04%),B:2.6(0.00%),P:1e-20\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: TTGGTCTCGTCA... (-54.865)\n",
      "\t\tRound 1: -77.92 TTGGTCTCGTCA T:32.0(0.04%),B:3.0(0.00%),P:1e-33\n",
      "\t\tRound 2: -77.92 TTGGTCTCGTCA T:32.0(0.04%),B:3.0(0.00%),P:1e-33\n",
      "\t\t=Final=: -60.98 TTGGTCTCGTCA T:32.0(0.04%),B:3.0(0.00%),P:1e-26\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: GGCTCATATTAC... (-54.728)\n",
      "\t\tRound 1: -64.53 GGCTCATATTAC T:28.0(0.04%),B:2.6(0.00%),P:1e-28\n",
      "\t\tRound 2: -98.93 GGCTCATATTAC T:38.0(0.05%),B:2.3(0.00%),P:1e-42\n",
      "\t\tRound 3: -106.16 GGYTCMTATTAC T:40.0(0.06%),B:2.8(0.00%),P:1e-46\n",
      "\t\tRound 4: -106.16 GGYTCMTATTAC T:40.0(0.06%),B:2.8(0.00%),P:1e-46\n",
      "\t\t=Final=: -107.19 GGYTCMTATTAC T:39.0(0.05%),B:1.4(0.00%),P:1e-46\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: GTCACGTGACGT... (-53.217)\n",
      "\t\tRound 1: -85.52 NHCACGTRACRN T:4980.0(6.67%),B:3961.7(5.54%),P:1e-37\n",
      "\t\tRound 2: -85.52 NHCACGTRACRN T:4980.0(6.67%),B:3961.7(5.54%),P:1e-37\n",
      "\t\t=Final=: -74.89 NHCACGTRACRN T:4018.0(5.57%),B:3286.7(4.61%),P:1e-32\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: ACTTCTCACGTA... (-61.269)\n",
      "\t\tRound 1: -61.27 ACTTCTCACGTA T:27.0(0.04%),B:2.4(0.00%),P:1e-26\n",
      "\t\tRound 2: -61.27 ACTTCTCACGTA T:27.0(0.04%),B:2.4(0.00%),P:1e-26\n",
      "\t\t=Final=: -47.49 ACTTCTCACGTA T:27.0(0.04%),B:2.4(0.00%),P:1e-20\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: TTACTTTCACTT... (-55.124)\n",
      "\t\tRound 1: -88.59 TTACTTTCACTT T:77.0(0.11%),B:11.8(0.02%),P:1e-38\n",
      "\t\tRound 2: -103.01 TTACTTTCACTT T:94.0(0.13%),B:14.5(0.02%),P:1e-44\n",
      "\t\tRound 3: -117.36 TTACTTTCAYTT T:79.0(0.11%),B:8.4(0.01%),P:1e-50\n",
      "\t\tRound 4: -117.36 TTACTTTCAYTT T:79.0(0.11%),B:8.4(0.01%),P:1e-50\n",
      "\t\t=Final=: -110.81 TTACTTTCAYTT T:99.0(0.14%),B:14.8(0.02%),P:1e-48\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t144332 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing binomial scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: SRR7475873_motif_macs2/homerMotifs.motifs12\n",
      "\n",
      "\t(Motifs in homer2 format)\n",
      "\tDetermining similar motifs... 75 reduced to 29 motifs\n",
      "\tOutputing HTML and sequence logos for motif comparison...\n",
      "\tChecking de novo motifs against known motifs...\n",
      "\tFormatting HTML page...\n",
      "\t\t1 of 29 (1e-2023) similar to BORIS(Zf)/K562-CTCFL-ChIP-Seq(GSE32465)/Homer(0.926)\n",
      "\t\t2 of 29 (1e-1497) similar to BATF(bZIP)/Th17-BATF-ChIP-Seq(GSE39756)/Homer(0.995)\n",
      "\t\t3 of 29 (1e-460) similar to ERG(ETS)/VCaP-ERG-ChIP-Seq(GSE14097)/Homer(0.966)\n",
      "\t\t4 of 29 (1e-317) similar to Sp2(Zf)/HEK293-Sp2.eGFP-ChIP-Seq(Encode)/Homer(0.901)\n",
      "\t\t5 of 29 (1e-259) similar to IRF1(IRF)/PBMC-IRF1-ChIP-Seq(GSE43036)/Homer(0.990)\n",
      "\t\t6 of 29 (1e-227) similar to NFY(CCAAT)/Promoter/Homer(0.971)\n",
      "\t\t7 of 29 (1e-197) similar to RUNX1(Runt)/Jurkat-RUNX1-ChIP-Seq(GSE29180)/Homer(0.981)\n",
      "\t\t8 of 29 (1e-173) similar to Tgif2(Homeobox)/mES-Tgif2-ChIP-Seq(GSE55404)/Homer(0.897)\n",
      "\t\t9 of 29 (1e-99) similar to Atf1(bZIP)/K562-ATF1-ChIP-Seq(GSE31477)/Homer(0.962)\n",
      "\t\t10 of 29 (1e-97) similar to TCF7/MA0769.2/Jaspar(0.954)\n",
      "\t\t11 of 29 (1e-83) similar to GFY(?)/Promoter/Homer(0.980)\n",
      "\t\t12 of 29 (1e-51) similar to NFkB-p65(RHD)/GM12787-p65-ChIP-Seq(GSE19485)/Homer(0.910)\n",
      "\t\t13 of 29 (1e-47) similar to RFX7/MA1554.1/Jaspar(0.890)\n",
      "\t\t14 of 29 (1e-41) similar to NRF(NRF)/Promoter/Homer(0.786)\n",
      "\t\t15 of 29 (1e-37) similar to Six1(Homeobox)/Myoblast-Six1-ChIP-Chip(GSE20150)/Homer(0.670)\n",
      "\t\t16 of 29 (1e-32) similar to NFAT(RHD)/Jurkat-NFATC1-ChIP-Seq(Jolma_et_al.)/Homer(0.860)\n",
      "\t\t17 of 29 (1e-29) similar to Smad3(MAD)/NPC-Smad3-ChIP-Seq(GSE36673)/Homer(0.679)\n",
      "\t\t18 of 29 (1e-23) similar to GATA1/MA0035.4/Jaspar(0.634)\n",
      "\t\t19 of 29 (1e-22) similar to CArG(MADS)/PUER-Srf-ChIP-Seq(Sullivan_et_al.)/Homer(0.860)\n",
      "\t\t20 of 29 (1e-21) similar to ZBTB32/MA1580.1/Jaspar(0.910)\n",
      "\t\t21 of 29 (1e-19) similar to ZBTB33/MA0527.1/Jaspar(0.902)\n",
      "\t\t22 of 29 (1e-18) similar to HIF2a(bHLH)/785_O-HIF2a-ChIP-Seq(GSE34871)/Homer(0.677)\n",
      "\t\t23 of 29 (1e-17) similar to PB0036.1_Irf6_1/Jaspar(0.628)\n",
      "\t\t24 of 29 (1e-15) similar to POL009.1_DCE_S_II/Jaspar(0.653)\n",
      "\t\t25 of 29 (1e-15) similar to PB0195.1_Zbtb3_2/Jaspar(0.687)\n",
      "\t\t26 of 29 (1e-14) similar to Arid5a/MA0602.1/Jaspar(0.651)\n",
      "\t\t27 of 29 (1e-13) similar to PB0154.1_Osr1_2/Jaspar(0.645)\n",
      "\t\t28 of 29 (1e-9) similar to Nkx3-2/MA0122.3/Jaspar(0.712)\n",
      "\t\t29 of 29 (1e-8) similar to ZNF384/MA1125.1/Jaspar(0.908)\n",
      "\tJob finished - if results look good, please send beer to ..\n",
      "\n",
      "\tCleaning up tmp files...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! findMotifsGenome.pl \\\n",
    "    /home/inou_doi/demo/3_peak_call/MACS2/SRR7475873_summits.bed \\\n",
    "    hg19 \\\n",
    "    SRR7475873_motif_macs2  \\\n",
    "    -size 200 \\\n",
    "    -mask \\\n",
    "    -p 12"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "5_homer",
   "language": "python",
   "name": "5_homer"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
